<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>References | Museum of Spatial Transcriptomics</title>
<meta name="author" content="Lambda Moses">
<meta name="author" content="Lior Pachter">
<meta name="description" content="“10X VISIUM SPATIAL TRANSCRIPTOMICS.” n.d. https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics.  “A Mouse for All Reasons.” 2007. Cell 128 (January): 9–13....">
<meta name="generator" content="bookdown 0.35 with bs4_book()">
<meta property="og:title" content="References | Museum of Spatial Transcriptomics">
<meta property="og:type" content="book">
<meta property="og:description" content="“10X VISIUM SPATIAL TRANSCRIPTOMICS.” n.d. https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics.  “A Mouse for All Reasons.” 2007. Cell 128 (January): 9–13....">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="References | Museum of Spatial Transcriptomics">
<meta name="twitter:description" content="“10X VISIUM SPATIAL TRANSCRIPTOMICS.” n.d. https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics.  “A Mouse for All Reasons.” 2007. Cell 128 (January): 9–13....">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.5.1/transition.js"></script><script src="libs/bs3compat-0.5.1/tabs.js"></script><script src="libs/bs3compat-0.5.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Museum of Spatial Transcriptomics</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Preface</a></li>
<li><a class="" href="intro.html"><span class="header-section-number">1</span> Introduction</a></li>
<li class="book-part">Prequel era</li>
<li><a class="" href="prequel.html"><span class="header-section-number">2</span> Prequel era</a></li>
<li><a class="" href="prequel-analysis.html"><span class="header-section-number">3</span> Data analysis in the prequel era</a></li>
<li class="book-part">Current era</li>
<li><a class="" href="current.html"><span class="header-section-number">4</span> From the past to the present</a></li>
<li><a class="" href="current-techs.html"><span class="header-section-number">5</span> Current era technologies</a></li>
<li><a class="" href="text-mining.html"><span class="header-section-number">6</span> Text mining LCM transcriptomics abstracts</a></li>
<li><a class="" href="current-analysis.html"><span class="header-section-number">7</span> Data analysis in the current era</a></li>
<li class="book-part">Future perspectives</li>
<li><a class="" href="future.html"><span class="header-section-number">8</span> From the past to the present to the future</a></li>
<li><a class="active" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/pachterlab/LP_2021">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="references" class="section level1 unnumbered">
<h1>References<a class="anchor" aria-label="anchor" href="#references"><i class="fas fa-link"></i></a>
</h1>

<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-illinois" class="csl-entry">
<span>“10X VISIUM SPATIAL TRANSCRIPTOMICS.”</span> n.d. <a href="https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics">https://biotech.illinois.edu/htdna/applications/10x-visium-spatial-transcriptomics</a>.
</div>
<div id="ref-AMouseForAllReasons" class="csl-entry">
<span>“A Mouse for All Reasons.”</span> 2007. <em>Cell</em> 128 (January): 9–13. <a href="https://doi.org/10.1016/j.cell.2006.12.018">https://doi.org/10.1016/j.cell.2006.12.018</a>.
</div>
<div id="ref-Abdelaal2020" class="csl-entry">
Abdelaal, Tamim, Soufiane Mourragui, Ahmed Mahfouz, and Marcel J T Reinders. 2020. <span>“SpaGE: Spatial Gene Enhancement Using scRNA-Seq.”</span> <em>Nucleic Acids Research</em> 48 (October): e107–7. <a href="https://doi.org/10.1093/nar/gkaa740">https://doi.org/10.1093/nar/gkaa740</a>.
</div>
<div id="ref-Abed-Esfahani2021" class="csl-entry">
Abed-Esfahani, Pegah, Benjamin C Darwin, Derek Howard, Nick Wang, Ethan Kim, Jason Lerch, and Leon French. 2021. <span>“Evaluation of Deep Convolutional Neural Networks for in Situ Hybridization Gene Expression Image Representation.”</span> <em>bioRxiv</em>, January, 2021.01.22.427860. <a href="https://doi.org/10.1101/2021.01.22.427860">https://doi.org/10.1101/2021.01.22.427860</a>.
</div>
<div id="ref-Aboobaker2005" class="csl-entry">
Aboobaker, A. Aziz, Pavel Tomancak, Nipam Patel, Gerald M. Rubin, and Eric C. Lai. 2005. <span>“Drosophila microRNAs Exhibit Diverse Spatial Expression Patterns During Embryonic Development.”</span> <em>Proceedings of the National Academy of Sciences</em> 102 (December): 18017–22. <a href="https://doi.org/10.1073/pnas.0508823102">https://doi.org/10.1073/pnas.0508823102</a>.
</div>
<div id="ref-Achim2015" class="csl-entry">
Achim, Kaia, Jean Baptiste Pettit, Luis R. Saraiva, Daria Gavriouchkina, Tomas Larsson, Detlev Arendt, and John C. Marioni. 2015. <span>“High-Throughput Spatial Mapping of Single-Cell RNA-Seq Data to Tissue of Origin.”</span> <em>Nature Biotechnology</em> 33 (May): 503–9. <a href="https://doi.org/10.1038/nbt.3209">https://doi.org/10.1038/nbt.3209</a>.
</div>
<div id="ref-umich" class="csl-entry">
<span>“ADVANCED GENOMICS CORE PRICING.”</span> n.d. <a href="https://brcf.medicine.umich.edu/cores/advanced-genomics/price-list/">https://brcf.medicine.umich.edu/cores/advanced-genomics/price-list/</a>.
</div>
<div id="ref-Agapite2020" class="csl-entry">
Agapite, Julie, Laurent-Philippe Albou, Suzi Aleksander, Joanna Argasinska, Valerio Arnaboldi, Helen Attrill, Susan M. Bello, et al. 2020. <span>“Alliance of Genome Resources Portal: Unified Model Organism Research Platform.”</span> <em>Nucleic Acids Research</em> 48 (January): D650–58. <a href="https://doi.org/10.1093/nar/gkz813">https://doi.org/10.1093/nar/gkz813</a>.
</div>
<div id="ref-Aguila2018" class="csl-entry">
Aguila, Julio, Shangli Cheng, Nigel Kee, Ming Cao, Qiaolin Deng, and Eva Hedlund. 2018. <span>“Spatial Transcriptomics and in Silico Random Pooling Identify Novel Markers of Vulnerable and Resistant Midbrain Dopamine Neurons.”</span> <em>bioRxiv</em>, October, 334417. <a href="https://doi.org/10.1101/334417">https://doi.org/10.1101/334417</a>.
</div>
<div id="ref-Aguila2021" class="csl-entry">
Aguila, Julio, Shangli Cheng, Nigel Kee, Ming Cao, Menghan Wang, Qiaolin Deng, and Eva Hedlund. 2021. <span>“Spatial RNA Sequencing Identifies Robust Markers of Vulnerable and Resistant Human Midbrain Dopamine Neurons and Their Expression in Parkinson’s Disease.”</span> <em>bioRxiv</em>, January, 334417. <a href="https://doi.org/10.1101/334417">https://doi.org/10.1101/334417</a>.
</div>
<div id="ref-Ahmed2015" class="csl-entry">
Ahmed, Ayisha, Nicole J Ward, Simon Moxon, Sara Lopez-Gomollon, Camille Viaut, Matthew L Tomlinson, Ilya Patrushev, et al. 2015. <span>“A Database of microRNA Expression Patterns in Xenopus Laevis.”</span> <em>PLOS ONE</em> 10 (October): e0138313. <a href="https://doi.org/10.1371/journal.pone.0138313">https://doi.org/10.1371/journal.pone.0138313</a>.
</div>
<div id="ref-Allen1988" class="csl-entry">
Allen, Nicholas D., David G. Cran, Sheila C. Barton, Simon Hettle, Wolf Reik, and M. Azim Surani. 1988. <span>“Transgenes as Probes for Active Chromosomal Domains in Mouse Development.”</span> <em>Nature</em> 333: 852–55. <a href="https://doi.org/10.1038/333852a0">https://doi.org/10.1038/333852a0</a>.
</div>
<div id="ref-Alon2021" class="csl-entry">
Alon, Shahar, Daniel R. Goodwin, Anubhav Sinha, Asmamaw T. Wassie, Fei Chen, Evan R. Daugharthy, Yosuke Bando, et al. 2021. <span>“Expansion Sequencing: Spatially Precise in Situ Transcriptomics in Intact Biological Systems.”</span> <em>Science</em> 371 (January). <a href="https://doi.org/10.1126/SCIENCE.AAX2656/SUPPL_FILE/AAX2656_TABLESS1-S6ANDS9-S14.XLSX">https://doi.org/10.1126/SCIENCE.AAX2656/SUPPL_FILE/AAX2656_TABLESS1-S6ANDS9-S14.XLSX</a>.
</div>
<div id="ref-Alonso2020" class="csl-entry">
Alonso, A M, A Carrea, and L Diambra. 2020. <span>“Prediction of Cell Position Using Single-Cell Transcriptomic Data: An Iterative Procedure [Version 2; Peer Review: 2 Approved].”</span> <em>F1000Research</em> 8. <a href="https://doi.org/10.12688/f1000research.20715.2">https://doi.org/10.12688/f1000research.20715.2</a>.
</div>
<div id="ref-Alsup2009" class="csl-entry">
Alsup, William H. 2009. <span>“Applera v. Illumina.”</span> <a href="https://www.leagle.com/decision/infco20100325236">https://www.leagle.com/decision/infco20100325236</a>.
</div>
<div id="ref-An2009" class="csl-entry">
An, Li, Hongbo Xie, Mark H Chin, Zoran Obradovic, Desmond J Smith, and Vasileios Megalooikonomou. 2009. <span>“Analysis of Multiplex Gene Expression Maps Obtained by Voxelation.”</span> <em>BMC Bioinformatics</em> 10 (April): S10. <a href="https://doi.org/10.1186/1471-2105-10-S4-S10">https://doi.org/10.1186/1471-2105-10-S4-S10</a>.
</div>
<div id="ref-Andersson2020" class="csl-entry">
Andersson, Alma, Joseph Bergenstråhle, Michaela Asp, Ludvig Bergenstråhle, Aleksandra Jurek, José Fernández Navarro, and Joakim Lundeberg. 2020. <span>“Single-Cell and Spatial Transcriptomics Enables Probabilistic Inference of Cell Type Topography.”</span> <em>Communications Biology</em> 3: 565. <a href="https://doi.org/10.1038/s42003-020-01247-y">https://doi.org/10.1038/s42003-020-01247-y</a>.
</div>
<div id="ref-Andersson2021" class="csl-entry">
Andersson, Axel, Ferran Diego, Fred A Hamprecht, and Carolina Wählby. 2021. <span>“ISTDECO: In Situ Transcriptomics Decoding by Deconvolution.”</span> <em>bioRxiv</em>, January, 2021.03.01.433040. <a href="https://doi.org/10.1101/2021.03.01.433040">https://doi.org/10.1101/2021.03.01.433040</a>.
</div>
<div id="ref-Andonian2019" class="csl-entry">
Andonian, Alexander, Daniel Paseltiner, Travis J. Gould, and Jason B. Castro. 2019. <span>“A Deep Learning Based Method for Large-Scale Classification, Registration, and Clustering of in-Situ Hybridization Experiments in the Mouse Olfactory Bulb.”</span> <em>Journal of Neuroscience Methods</em> 312 (January): 162–68. <a href="https://doi.org/10.1016/j.jneumeth.2018.12.003">https://doi.org/10.1016/j.jneumeth.2018.12.003</a>.
</div>
<div id="ref-ArcturusUS" class="csl-entry">
<span>“Arcturus XT Laser Capture Microdissection (LCM) Instrument - US.”</span> n.d. <a href="https:////www.thermofisher.com/us/en/home/life-science/gene-expression-analysis-genotyping/laser-capture-microdissection/arcturus-laser-capture-microdissection-lcm-instrument.html">//www.thermofisher.com/us/en/home/life-science/gene-expression-analysis-genotyping/laser-capture-microdissection/arcturus-laser-capture-microdissection-lcm-instrument.html</a>.
</div>
<div id="ref-Ardini-Poleske2017" class="csl-entry">
Ardini-Poleske, Maryanne E., Robert F. Clark, Charles Ansong, James P. Carson, Richard A. Corley, Gail H. Deutsch, James S. Hagood, et al. 2017. <span>“LungMAP: The Molecular Atlas of Lung Development Program.”</span> <em>American Journal of Physiology-Lung Cellular and Molecular Physiology</em> 313 (November): L733–40. <a href="https://doi.org/10.1152/ajplung.00139.2017">https://doi.org/10.1152/ajplung.00139.2017</a>.
</div>
<div id="ref-Armit2017" class="csl-entry">
Armit, Chris, Lorna Richardson, Shanmugasundaram Venkataraman, Liz Graham, Nicholas Burton, Bill Hill, Yiya Yang, and Richard A. Baldock. 2017. <span>“eMouseAtlas: An Atlas-Based Resource for Understanding Mammalian Embryogenesis.”</span> <em>Developmental Biology</em> 423 (March): 1–11. <a href="https://doi.org/10.1016/j.ydbio.2017.01.023">https://doi.org/10.1016/j.ydbio.2017.01.023</a>.
</div>
<div id="ref-Arnol2019a" class="csl-entry">
Arnol, Damien, Denis Schapiro, Bernd Bodenmiller, Julio Saez-Rodriguez, and Oliver Stegle. 2019. <span>“Modeling Cell-Cell Interactions from Spatial Molecular Data with Spatial Variance Component Analysis.”</span> <em>Cell Reports</em> 29 (October): 202–211.e6. <a href="https://doi.org/10.1016/j.celrep.2019.08.077">https://doi.org/10.1016/j.celrep.2019.08.077</a>.
</div>
<div id="ref-Asp2019" class="csl-entry">
Asp, Michaela, Stefania Giacomello, Ludvig Larsson, Chenglin Wu, Daniel Fürth, Xiaoyan Qian, Eva Wärdell, et al. 2019. <span>“A Spatiotemporal Organ-Wide Gene Expression and Cell Atlas of the Developing Human Heart.”</span> <em>Cell</em> 179 (December): 1647–1660.e19. <a href="https://doi.org/10.1016/j.cell.2019.11.025">https://doi.org/10.1016/j.cell.2019.11.025</a>.
</div>
<div id="ref-Asp2017" class="csl-entry">
Asp, Michaela, Fredrik Salmén, Patrik L Ståhl, Sanja Vickovic, Ulrika Felldin, Marie Löfling, José Fernandez Navarro, et al. 2017. <span>“Spatial Detection of Fetal Marker Genes Expressed at Low Level in Adult Human Heart Tissue.”</span> <em>Scientific Reports</em> 7: 12941. <a href="https://doi.org/10.1038/s41598-017-13462-5">https://doi.org/10.1038/s41598-017-13462-5</a>.
</div>
<div id="ref-Baccin2020" class="csl-entry">
Baccin, Chiara, Jude Al-Sabah, Lars Velten, Patrick M. Helbling, Florian Grünschläger, Pablo Hernández-Malmierca, César Nombela-Arrieta, Lars M. Steinmetz, Andreas Trumpp, and Simon Haas. 2020. <span>“Combined Single-Cell and Spatial Transcriptomics Reveal the Molecular, Cellular and Spatial Bone Marrow Niche Organization.”</span> <em>Nature Cell Biology</em> 22 (January): 38–48. <a href="https://doi.org/10.1038/s41556-019-0439-6">https://doi.org/10.1038/s41556-019-0439-6</a>.
</div>
<div id="ref-Baharlou2019" class="csl-entry">
Baharlou, Heeva, Nicolas P. Canete, Anthony L. Cunningham, Andrew N. Harman, and Ellis Patrick. 2019. <span>“Mass Cytometry Imaging for the Study of Human Diseases—Applications and Data Analysis Strategies.”</span> <em>Frontiers in Immunology</em> 10 (November): 2657. <a href="https://doi.org/10.3389/fimmu.2019.02657">https://doi.org/10.3389/fimmu.2019.02657</a>.
</div>
<div id="ref-Baker2021" class="csl-entry">
Baker, Dylan, Iman M Al-Naggar, Santhosh Sivajothi, William F Flynn, Anahita Amiri, Diane Luo, Cara C Hardy, George A Kuchel, Phillip P Smith, and Paul Robson. 2021. <span>“A Cellular Reference Resource for the Mouse Urinary Bladder.”</span> <em>bioRxiv</em>, January, 2021.09.20.461121. <a href="https://doi.org/10.1101/2021.09.20.461121">https://doi.org/10.1101/2021.09.20.461121</a>.
</div>
<div id="ref-Bakken2016" class="csl-entry">
Bakken, Trygve E., Jeremy A. Miller, Song Lin Ding, Susan M. Sunkin, Kimberly A. Smith, Lydia Ng, Aaron Szafer, et al. 2016. <span>“A Comprehensive Transcriptional Map of Primate Brain Development.”</span> <em>Nature</em> 535 (July): 367–75. <a href="https://doi.org/10.1038/nature18637">https://doi.org/10.1038/nature18637</a>.
</div>
<div id="ref-Baldock1992" class="csl-entry">
Baldock, Richard, Jonathan Bard, Matt Kaufman, and Duncan Davidson. 1992. <span>“What’s New? A Real Mouse for Your Computer.”</span> <em>BioEssays</em> 14 (July): 501–2. <a href="https://doi.org/10.1002/bies.950140713">https://doi.org/10.1002/bies.950140713</a>.
</div>
<div id="ref-Baner1998" class="csl-entry">
Baner, J., Mats Nilsson, Maritha Mendel-Hartvig, and Uif Landegren. 1998. <span>“Signal Amplification of Padlock Probes by Rolling Circle Replication.”</span> <em>Nucleic Acids Research</em> 26 (November): 5073–78. <a href="https://doi.org/10.1093/nar/26.22.5073">https://doi.org/10.1093/nar/26.22.5073</a>.
</div>
<div id="ref-Bassell1994" class="csl-entry">
Bassell, G J, C M Powers, K L Taneja, and R H Singer. 1994. <span>“Single mRNAs Visualized by Ultrastructural in Situ Hybridization Are Principally Localized at Actin Filament Intersections in Fibroblasts.”</span> <em>The Journal of Cell Biology</em> 126 (August): 863–76. <a href="https://doi.org/10.1083/jcb.126.4.863">https://doi.org/10.1083/jcb.126.4.863</a>.
</div>
<div id="ref-Bates2012" class="csl-entry">
Bates, Mark, Graham T. Dempsey, Kok Hao Chen, and Xiaowei Zhuang. 2012. <span>“Multicolor Super-Resolution Fluorescence Imaging via Multi-Parameter Fluorophore Detection.”</span> <em>ChemPhysChem</em> 13 (January): 99–107. <a href="https://doi.org/10.1002/cphc.201100735">https://doi.org/10.1002/cphc.201100735</a>.
</div>
<div id="ref-Battich2013" class="csl-entry">
Battich, Nico, Thomas Stoeger, and Lucas Pelkmans. 2013. <span>“Image-Based Transcriptomics in Thousands of Single Human Cells at Single-Molecule Resolution.”</span> <em>Nature Methods</em> 10 (November): 1127–36. <a href="https://doi.org/10.1038/nmeth.2657">https://doi.org/10.1038/nmeth.2657</a>.
</div>
<div id="ref-Bayraktar2020" class="csl-entry">
Bayraktar, Omer Ali, Theresa Bartels, Staffan Holmqvist, Vitalii Kleshchevnikov, Araks Martirosyan, Damon Polioudakis, Lucile Ben Haim, et al. 2020. <span>“Astrocyte Layers in the Mammalian Cerebral Cortex Revealed by a Single-Cell in Situ Transcriptomic Map.”</span> <em>Nature Neuroscience</em> 23 (April): 500–509. <a href="https://doi.org/10.1038/s41593-020-0602-1">https://doi.org/10.1038/s41593-020-0602-1</a>.
</div>
<div id="ref-Becker1996" class="csl-entry">
Becker, Ingrid, Karl Friedrich Becker, Michael H. Röhrl, Günter Minkus, Karin Schütze, and Heinz Höfler. 1996. <span>“Single-Cell Mutation Analysis of Tumors from Stained Histologic Slides.”</span> <em>Laboratory Investigation</em> 75 (December): 801–7. <a href="https://pubmed.ncbi.nlm.nih.gov/8973475/">https://pubmed.ncbi.nlm.nih.gov/8973475/</a>.
</div>
<div id="ref-Bell2004" class="csl-entry">
Bell, George W., Tatiana A. Yatskievych, and Parker B. Antin. 2004. <span>“GEISHA, a Whole-Mount in Situ Hybridization Gene Expression Screen in Chicken Embryos.”</span> <em>Developmental Dynamics</em> 229 (March): 677–87. <a href="https://doi.org/10.1002/dvdy.10503">https://doi.org/10.1002/dvdy.10503</a>.
</div>
<div id="ref-Bellen1989" class="csl-entry">
Bellen, H. J., C. J. O’Kane, C. Wilson, U. Grossniklaus, R. K. Pearson, and W. J. Gehring. 1989. <span>“P-Element-Mediated Enhancer Detection: A Versatile Method to Study Development in Drosophila.”</span> <em>Genes &amp; Development</em> 3 (September): 1288–1300. <a href="https://doi.org/10.1101/gad.3.9.1288">https://doi.org/10.1101/gad.3.9.1288</a>.
</div>
<div id="ref-Belmamoune2008" class="csl-entry">
Belmamoune, Mounia, and Fons J. Verbeek. 2008. <span>“Data Integration for Spatio-Temporal Patterns of Gene Expression of Zebrafish Development: The GEMS Database.”</span> <em>Journal of Integrative Bioinformatics</em> 5 (June): 35–48. <a href="https://doi.org/10.1515/jib-2008-92">https://doi.org/10.1515/jib-2008-92</a>.
</div>
<div id="ref-Belyavsky1989" class="csl-entry">
Belyavsky, A, T Vinogradova, and K Rajewsky. 1989. <span>“PCR-Based cDNA Library Construction: General cDNA Libraries at the Level of a Few Cells.”</span> <em>Nucleic Acids Research</em> 17 (April): 2919–32. <a href="https://doi.org/10.1093/nar/17.8.2919">https://doi.org/10.1093/nar/17.8.2919</a>.
</div>
<div id="ref-Bergenstrahle2020a" class="csl-entry">
Bergenstråhle, Joseph, Ludvig Bergenstråhle, and Joakim Lundeberg. 2020. <span>“SpatialCPie: An r/Bioconductor Package for Spatial Transcriptomics Cluster Evaluation.”</span> <em>BMC Bioinformatics</em> 21 (December): 161. <a href="https://doi.org/10.1186/s12859-020-3489-7">https://doi.org/10.1186/s12859-020-3489-7</a>.
</div>
<div id="ref-Bergenstrahle2020" class="csl-entry">
Bergenstråhle, Ludvig, Bryan He, Joseph Bergenstråhle, Alma Andersson, Joakim Lundeberg, James Zou, and Jonas Maaskola. 2020. <span>“Super-Resolved Spatial Transcriptomics by Deep Data Fusion.”</span> <em>bioRxiv</em>, March, 2020.02.28.963413. <a href="https://doi.org/10.1101/2020.02.28.963413">https://doi.org/10.1101/2020.02.28.963413</a>.
</div>
<div id="ref-Berglund2018" class="csl-entry">
Berglund, Emelie, Jonas Maaskola, Niklas Schultz, Stefanie Friedrich, Maja Marklund, Joseph Bergenstråhle, Firas Tarish, et al. 2018. <span>“Spatial Maps of Prostate Cancer Transcriptomes Reveal an Unexplored Landscape of Heterogeneity.”</span> <em>Nature Communications</em>. <a href="https://doi.org/10.1038/s41467-018-04724-5">https://doi.org/10.1038/s41467-018-04724-5</a>.
</div>
<div id="ref-BERNS1969" class="csl-entry">
Berns, Michael W., Roberts S. Olson, and Donald E. Rounds. 1969. <span>“In Vitro Production of Chromosomal Lesions with an Argon Laser Microbeam.”</span> <em>Nature</em> 221 (January): 74–75. <a href="https://doi.org/10.1038/221074a0">https://doi.org/10.1038/221074a0</a>.
</div>
<div id="ref-Bettenhausen1995" class="csl-entry">
Bettenhausen, Berthold, and Achim Gossler. 1995. <span>“Efficient Isolation of Novel Mouse Genes Differentially Expressed in Early Postimplantation Embryos.”</span> <em>Genomics</em> 28 (August): 436–41. <a href="https://doi.org/10.1006/geno.1995.1172">https://doi.org/10.1006/geno.1995.1172</a>.
</div>
<div id="ref-Bhaduri2021" class="csl-entry">
Bhaduri, Aparna, Carmen Sandoval-Espinosa, Marcos Otero-Garcia, Irene Oh, Raymund Yin, Ugomma C Eze, Tomasz J Nowakowski, and Arnold R Kriegstein. 2021. <span>“An Atlas of Cortical Arealization Identifies Dynamic Molecular Signatures.”</span> <em>bioRxiv</em>, January, 2021.05.17.444528. <a href="https://doi.org/10.1101/2021.05.17.444528">https://doi.org/10.1101/2021.05.17.444528</a>.
</div>
<div id="ref-Biancalani2020" class="csl-entry">
Biancalani, Tommaso, Gabriele Scalia, Lorenzo Buffoni, Raghav Avasthi, Ziqing Lu, Aman Sanger, Neriman Tokcan, et al. 2020. <span>“Deep Learning and Alignment of Spatially-Resolved Whole Transcriptomes of Single Cells in the Mouse Brain with Tangram.”</span> <em>bioRxiv</em>, January, 2020.08.29.272831. <a href="https://doi.org/10.1101/2020.08.29.272831">https://doi.org/10.1101/2020.08.29.272831</a>.
</div>
<div id="ref-Bidarimath2015" class="csl-entry">
Bidarimath, Mallikarjun, Andrew K. Edwards, and Chandrakant Tayade. 2015. <span>“Laser Capture Microdissection for Gene Expression Analysis.”</span> <em>Methods in Molecular Biology</em> 1219: 115–37. <a href="https://doi.org/10.1007/978-1-4939-1661-0_10">https://doi.org/10.1007/978-1-4939-1661-0_10</a>.
</div>
<div id="ref-Bier1989" class="csl-entry">
Bier, E., H. Vaessin, S. Shepherd, K. Lee, K. McCall, S. Barbel, L. Ackerman, R. Carretto, T. Uemura, and E. Grell. 1989. <span>“Searching for Pattern and Mutation in the Drosophila Genome with a p-lacZ Vector.”</span> <em>Genes &amp; Development</em> 3 (September): 1273–87. <a href="https://doi.org/10.1101/gad.3.9.1273">https://doi.org/10.1101/gad.3.9.1273</a>.
</div>
<div id="ref-BinTayyash2020" class="csl-entry">
BinTayyash, Nuha, Sokratia Georgaka, S T John, Sumon Ahmed, Alexis Boukouvalas, James Hensman, and Magnus Rattray. 2020. <span>“Non-Parametric Modelling of Temporal and Spatial Counts Data from RNA-Seq Experiments.”</span> <em>bioRxiv</em>, January, 2020.07.29.227207. <a href="https://doi.org/10.1101/2020.07.29.227207">https://doi.org/10.1101/2020.07.29.227207</a>.
</div>
<div id="ref-Birchall1995" class="csl-entry">
Birchall, Philip S, Rita M Fishpool, and Donna G Albertson. 1995. <span>“Expression Patterns of Predicted Genes from the c. Elegans Genome Sequence Visualized by FISH in Whole Organisms,”</span> 314–20. <a href="https://doi.org/10.1038/ng1195-314">https://doi.org/10.1038/ng1195-314</a>.
</div>
<div id="ref-Blackshaw2004" class="csl-entry">
Blackshaw, Seth, Sanjiv Harpavat, Jeff Trimarchi, Li Cai, Haiyan Huang, Winston P Kuo, Griffin Weber, et al. 2004. <span>“Genomic Analysis of Mouse Retinal Development.”</span> Edited by William Harris. <em>PLoS Biology</em> 2 (June): e247. <a href="https://doi.org/10.1371/journal.pbio.0020247">https://doi.org/10.1371/journal.pbio.0020247</a>.
</div>
<div id="ref-Blitz2017" class="csl-entry">
Blitz, Ira L., Kitt D. Paraiso, Ilya Patrushev, William T. Y. Chiu, Ken W. Y. Cho, and Michael J. Gilchrist. 2017. <span>“A Catalog of Xenopus Tropicalis Transcription Factors and Their Regional Expression in the Early Gastrula Stage Embryo.”</span> <em>Developmental Biology</em> 426 (June): 409–17. <a href="https://doi.org/10.1016/j.ydbio.2016.07.002">https://doi.org/10.1016/j.ydbio.2016.07.002</a>.
</div>
<div id="ref-DeBoer2009" class="csl-entry">
Boer, B. A. de, Jan M. Ruijter, F. P. J. M. Voorbraak, and A. F. M. Moorman. 2009. <span>“More Than a Decade of Developmental Gene Expression Atlases: Where Are We Now?”</span> <em>Nucleic Acids Research</em> 37 (December): 7349–59. <a href="https://doi.org/10.1093/nar/gkp819">https://doi.org/10.1093/nar/gkp819</a>.
</div>
<div id="ref-Bohland2010a" class="csl-entry">
Bohland, Jason W., Hemant Bokil, Sayan D. Pathak, Chang-Kyu Lee, Lydia Ng, Christopher Lau, Chihchau Kuan, Michael Hawrylycz, and Partha P. Mitra. 2010. <span>“Clustering of Spatial Gene Expression Patterns in the Mouse Brain and Comparison with Classical Neuroanatomy.”</span> <em>Methods</em> 50 (February): 105–12. <a href="https://doi.org/10.1016/j.ymeth.2009.09.001">https://doi.org/10.1016/j.ymeth.2009.09.001</a>.
</div>
<div id="ref-Boisset2018" class="csl-entry">
Boisset, Jean-Charles, Judith Vivié, Dominic Grün, Mauro J Muraro, Anna Lyubimova, and Alexander van Oudenaarden. 2018. <span>“Mapping the Physical Network of Cellular Interactions.”</span> <em>Nature Methods</em> 15: 547–53. <a href="https://doi.org/10.1038/s41592-018-0009-z">https://doi.org/10.1038/s41592-018-0009-z</a>.
</div>
<div id="ref-Boulgakov2020" class="csl-entry">
Boulgakov, Alexander A, Andrew D Ellington, and Edward M Marcotte. 2020. <span>“Bringing Microscopy-by-Sequencing into View.”</span> <em>Trends in Biotechnology</em> 38: 154–62. https://doi.org/<a href="https://doi.org/10.1016/j.tibtech.2019.06.001">https://doi.org/10.1016/j.tibtech.2019.06.001</a>.
</div>
<div id="ref-Bowes2009" class="csl-entry">
Bowes, Jeff B., Kevin A. Snyder, Erik Segerdell, Chris J. Jarabek, Kenan Azam, Aaron M. Zorn, and Peter D. Vize. 2009. <span>“Xenbase: Gene Expression and Improved Integration.”</span> <em>Nucleic Acids Research</em> 38 (October): D607–12. <a href="https://doi.org/10.1093/nar/gkp953">https://doi.org/10.1093/nar/gkp953</a>.
</div>
<div id="ref-Brady1990" class="csl-entry">
Brady, Gerard, Mary Barbara, and Norman N. Iscove. 1990. <span>“Representative in Vitro cDNA Amplification from Individual Hemopoietic Cells and Colonies.”</span> <em>Methods in Molecular and Cellular Biology</em> 2: 17–25.
</div>
<div id="ref-Brady2021" class="csl-entry">
Brady, Lauren, Michelle Kriner, Ilsa Coleman, Colm Morrissey, Martine Roudier, Lawrence D True, Roman Gulati, et al. 2021. <span>“Inter- and Intra-Tumor Heterogeneity of Metastatic Prostate Cancer Determined by Digital Spatial Gene Expression Profiling.”</span> <em>Nature Communications</em> 12: 1426. <a href="https://doi.org/10.1038/s41467-021-21615-4">https://doi.org/10.1038/s41467-021-21615-4</a>.
</div>
<div id="ref-VandenBrink2020" class="csl-entry">
Brink, Susanne C. van den, Anna Alemany, Vincent van Batenburg, Naomi Moris, Marloes Blotenburg, Judith Vivié, Peter Baillie-Johnson, et al. 2020. <span>“Single-Cell and Spatial Transcriptomics Reveal Somitogenesis in Gastruloids.”</span> <em>Nature</em> 582 (February): 405. <a href="https://doi.org/10.1038/s41586-020-2024-3">https://doi.org/10.1038/s41586-020-2024-3</a>.
</div>
<div id="ref-Brown2002" class="csl-entry">
Brown, V. M., A. Ossadtchi, A. H. Khan, S. Yee, G. Lacan, W. P. Melega, S. R. Cherry, R. M. Leahy, and D. J. Smith. 2002. <span>“Multiplex Three-Dimensional Brain Gene Expression Mapping in a Mouse Model of Parkinson’s Disease.”</span> <em>Genome Research</em> 12 (May): 868–84. <a href="https://doi.org/10.1101/gr.229002">https://doi.org/10.1101/gr.229002</a>.
</div>
<div id="ref-Brown2002a" class="csl-entry">
Brown, Vanessa M. 2002. <span>“High-Throughput Imaging of Brain Gene Expression.”</span> <em>Genome Research</em> 12 (February): 244–54. <a href="https://doi.org/10.1101/gr.204102">https://doi.org/10.1101/gr.204102</a>.
</div>
<div id="ref-Brown2002GET" class="csl-entry">
Brown, Vanessa M., Alex. Ossadtchi, Arshad H. Khan, Sanjiv S. Gambhir, Simon R. Cherry, Richard M. Leahy, and Desmond J. Smith. 2002. <span>“Gene Expression Tomography.”</span> <em>Physiological Genomics</em> 8 (February): 159–67. <a href="https://doi.org/10.1152/physiolgenomics.00090.2001">https://doi.org/10.1152/physiolgenomics.00090.2001</a>.
</div>
<div id="ref-VanBruggen2021" class="csl-entry">
Bruggen, David van, Fabio Pohl, Christoffer Mattsson Langseth, Petra Kukanja, Hower Lee, Mukund Kabbe, Mandy Meijer, et al. 2021. <span>“Developmental Landscape of Human Forebrain at a Single-Cell Level Unveils Early Waves of Oligodendrogenesis.”</span> <em>bioRxiv</em>, January, 2021.07.22.453317. <a href="https://doi.org/10.1101/2021.07.22.453317">https://doi.org/10.1101/2021.07.22.453317</a>.
</div>
<div id="ref-Buchberger2018" class="csl-entry">
Buchberger, Amanda Rae, Kellen DeLaney, Jillian Johnson, and Lingjun Li. 2018. <span>“Mass Spectrometry Imaging: A Review of Emerging Advancements and Future Insights.”</span> <em>Analytical Chemistry</em> 90 (January): 240–65. <a href="https://doi.org/10.1021/acs.analchem.7b04733">https://doi.org/10.1021/acs.analchem.7b04733</a>.
</div>
<div id="ref-Burgess2019" class="csl-entry">
Burgess, Darren J. 2019. <span>“Spatial Transcriptomics Coming of Age.”</span> <em>Nature Reviews Genetics</em> 20 (June): 317–17. <a href="https://doi.org/10.1038/s41576-019-0129-z">https://doi.org/10.1038/s41576-019-0129-z</a>.
</div>
<div id="ref-Burkhard2018" class="csl-entry">
Burkhard, Silja Barbara, and Jeroen Bakkers. 2018. <span>“Spatially Resolved RNA-Sequencing of the Embryonic Heart Identifies a Role for Wnt/β-Catenin Signaling in Autonomic Control of Heart Rate.”</span> <em>eLife</em> 7 (February). <a href="https://doi.org/10.7554/eLife.31515">https://doi.org/10.7554/eLife.31515</a>.
</div>
<div id="ref-Bustin2015" class="csl-entry">
Bustin, Stephen, Harvinder S Dhillon, Sara Kirvell, Christina Greenwood, Michael Parker, Gregory L Shipley, and Tania Nolan. 2015. <span>“Variability of the Reverse Transcription Step: Practical Implications.”</span> <em>Clinical Chemistry</em> 61 (January): 202–12. <a href="https://doi.org/10.1373/clinchem.2014.230615">https://doi.org/10.1373/clinchem.2014.230615</a>.
</div>
<div id="ref-Butler2021" class="csl-entry">
Butler, Daniel, Christopher Mozsary, Cem Meydan, Jonathan Foox, Joel Rosiene, Alon Shaiber, David Danko, et al. 2021. <span>“Shotgun Transcriptome, Spatial Omics, and Isothermal Profiling of SARS-CoV-2 Infection Reveals Unique Host Responses, Viral Diversification, and Drug Interactions.”</span> <em>Nature Communications</em> 12: 1660. <a href="https://doi.org/10.1038/s41467-021-21361-7">https://doi.org/10.1038/s41467-021-21361-7</a>.
</div>
<div id="ref-Cabili2015" class="csl-entry">
Cabili, Moran N., Margaret C. Dunagin, Patrick D. McClanahan, Andrew Biaesch, Olivia Padovan-Merhar, Aviv Regev, John L. Rinn, and Arjun Raj. 2015. <span>“Localization and Abundance Analysis of Human lncRNAs at Single-Cell and Single-Molecule Resolution.”</span> <em>Genome Biology</em> 16 (December): 20. <a href="https://doi.org/10.1186/s13059-015-0586-4">https://doi.org/10.1186/s13059-015-0586-4</a>.
</div>
<div id="ref-Cable2020" class="csl-entry">
Cable, Dylan M., Evan Murray, Luli S. Zou, Aleksandrina Goeva, Evan Z. Macosko, Fei Chen, and Rafael A. Irizarry. 2020. <span>“Robust Decomposition of Cell Type Mixtures in Spatial Transcriptomics.”</span> <em>bioRxiv</em>, May, 2020.05.07.082750. <a href="https://doi.org/10.1101/2020.05.07.082750">https://doi.org/10.1101/2020.05.07.082750</a>.
</div>
<div id="ref-Callaway2021" class="csl-entry">
Callaway, Edward M., Hong Wei Dong, Joseph R. Ecker, Michael J. Hawrylycz, Z. Josh Huang, Ed S. Lein, John Ngai, et al. 2021. <span>“A Multimodal Cell Census and Atlas of the Mammalian Primary Motor Cortex.”</span> <em>Nature 2021 598:7879</em> 598 (October): 86–102. <a href="https://doi.org/10.1038/s41586-021-03950-0">https://doi.org/10.1038/s41586-021-03950-0</a>.
</div>
<div id="ref-Campiteli2013" class="csl-entry">
Campiteli, Monica Guimarães, Cesar Henrique Comin, Luciano Da Fontoura Costa, M. Madan Babu, and Roberto Marcondes Cesar. 2013. <span>“A Methodology to Infer Gene Networks from Spatial Patterns of Expression – an Application to Fluorescence in Situ Hybridization Images.”</span> <em>Molecular BioSystems</em> 9 (June): 1926. <a href="https://doi.org/10.1039/c3mb25475e">https://doi.org/10.1039/c3mb25475e</a>.
</div>
<div id="ref-Canete2022" class="csl-entry">
Canete, Nicolas P, Sourish S Iyengar, John T Ormerod, Heeva Baharlou, Andrew N Harman, and Ellis Patrick. 2022. <span>“spicyR: Spatial Analysis of in Situ Cytometry Data in r.”</span> <em>Bioinformatics</em> 38 (May): 3099–3105.
</div>
<div id="ref-Cang2020" class="csl-entry">
Cang, Zixuan, and Qing Nie. 2020. <span>“Inferring Spatial and Signaling Relationships Between Cells from Single Cell Transcriptomic Data.”</span> <em>Nature Communications</em> 11: 2084. <a href="https://doi.org/10.1038/s41467-020-15968-5">https://doi.org/10.1038/s41467-020-15968-5</a>.
</div>
<div id="ref-Cankaya2007" class="csl-entry">
Cankaya, Murat, Ana Hernandez, Mehmet Ciftci, Sukru Beydemir, Hasan Ozdemir, Harun Budak, Ilhami Gulcin, et al. 2007. <span>“An Analysis of Expression Patterns of Genes Encoding Proteins with Catalytic Activities.”</span> <em>BMC Genomics</em> 8 (July): 232. <a href="https://doi.org/10.1186/1471-2164-8-232">https://doi.org/10.1186/1471-2164-8-232</a>.
</div>
<div id="ref-Cao2019" class="csl-entry">
Cao, Junyue, Malte Spielmann, Xiaojie Qiu, Xingfan Huang, Daniel M Ibrahim, Andrew J Hill, Fan Zhang, et al. 2019. <span>“The Single-Cell Transcriptional Landscape of Mammalian Organogenesis.”</span> <em>Nature</em> 566: 496–502. <a href="https://doi.org/10.1038/s41586-019-0969-x">https://doi.org/10.1038/s41586-019-0969-x</a>.
</div>
<div id="ref-Cao2021" class="csl-entry">
Cao, Zhaohui, Wenlong Zuo, Lanxiang Wang, Junyu Chen, Zepeng Qu, Fan Jin, and Lei Dai. 2021. <span>“Spatial Profiling of Microbial Communities by Sequential FISH with Error-Robust Encoding.”</span> <em>bioRxiv</em>, January, 2021.05.27.445923. <a href="https://doi.org/10.1101/2021.05.27.445923">https://doi.org/10.1101/2021.05.27.445923</a>.
</div>
<div id="ref-Carlberg2019" class="csl-entry">
Carlberg, Konstantin, Marina Korotkova, Ludvig Larsson, Anca I Catrina, Patrik L Ståhl, and Vivianne Malmström. 2019. <span>“Exploring Inflammatory Signatures in Arthritic Joint Biopsies with Spatial Transcriptomics.”</span> <em>Scientific Reports</em> 9: 18975. <a href="https://doi.org/10.1038/s41598-019-55441-y">https://doi.org/10.1038/s41598-019-55441-y</a>.
</div>
<div id="ref-Carson2002" class="csl-entry">
Carson, James P., Christina Thaller, and Gregor Eichele. 2002. <span>“A Transcriptome Atlas of the Mouse Brain at Cellular Resolution.”</span> <em>Current Opinion in Neurobiology</em> 12 (October): 562–65. <a href="https://doi.org/10.1016/S0959-4388(02)00356-2">https://doi.org/10.1016/S0959-4388(02)00356-2</a>.
</div>
<div id="ref-Carter2003" class="csl-entry">
Carter, Mark G. 2003. <span>“In Situ-Synthesized Novel Microarray Optimized for Mouse Stem Cell and Early Developmental Expression Profiling.”</span> <em>Genome Research</em> 13 (May): 1011–21. <a href="https://doi.org/10.1101/gr.878903">https://doi.org/10.1101/gr.878903</a>.
</div>
<div id="ref-Chalfoun2017" class="csl-entry">
Chalfoun, Joe, Michael Majurski, Tim Blattner, Kiran Bhadriraju, Walid Keyrouz, Peter Bajcsy, and Mary Brady. 2017. <span>“MIST: Accurate and Scalable Microscopy Image Stitching Tool with Stage Modeling and Error Minimization.”</span> <em>Scientific Reports</em> 7: 4988. <a href="https://doi.org/10.1038/s41598-017-04567-y">https://doi.org/10.1038/s41598-017-04567-y</a>.
</div>
<div id="ref-Chang2021" class="csl-entry">
Chang, Yuzhou, Fei He, Juexin Wang, Shuo Chen, Jingyi Li, Jixin Liu, Yang Yu, et al. 2021. <span>“Define and Visualize Pathological Architectures of Human Tissues from Spatially Resolved Transcriptomics Using Deep Learning.”</span> <em>bioRxiv</em>, January, 2021.07.08.451210. <a href="https://doi.org/10.1101/2021.07.08.451210">https://doi.org/10.1101/2021.07.08.451210</a>.
</div>
<div id="ref-Charitakis2022" class="csl-entry">
Charitakis, Natalie, Agus Salim, Adam T Piers, Kevin I Watt, Enzo R Porrello, David A Elliott, Mirana Ramialison, and David Elliott. 2022. <span>“Disparities in Spatially Variable Gene Calling Highlight the Need for Benchmarking Spatial Transcriptomics Methods.”</span> <em>bioRxiv</em>, November, 2022.10.31.514623. <a href="https://doi.org/10.1101/2022.10.31.514623">https://doi.org/10.1101/2022.10.31.514623</a>.
</div>
<div id="ref-Chavent2018" class="csl-entry">
Chavent, Marie, Vanessa Kuentz-Simonet, Amaury Labenne, and Jérôme Saracco. 2018. <span>“ClustGeo: An r Package for Hierarchical Clustering with Spatial Constraints.”</span> <em>Computational Statistics</em> 33 (January): 1799–1822. <a href="https://doi.org/10.1007/s00180-018-0791-1">https://doi.org/10.1007/s00180-018-0791-1</a>.
</div>
<div id="ref-Chen2021a" class="csl-entry">
Chen, Ao, Sha Liao, Mengnan Cheng, Kailong Ma, Liang Wu, Yiwei Lai, Jin Yang, et al. 2021. <span>“Large Field of View-Spatially Resolved Transcriptomics at Nanoscale Resolution.”</span> <em>bioRxiv</em>, January, 2021.01.17.427004. <a href="https://doi.org/10.1101/2021.01.17.427004">https://doi.org/10.1101/2021.01.17.427004</a>.
</div>
<div id="ref-Chen2022" class="csl-entry">
Chen, Carissa, Hani Jieun Kim, and Pengyi Yang. 2022. <span>“Evaluating Spatially Variable Gene Detection Methods for Spatial Transcriptomics Data.”</span> <em>bioRxiv</em>, November, 2022.11.23.517747. <a href="https://doi.org/10.1101/2022.11.23.517747">https://doi.org/10.1101/2022.11.23.517747</a>.
</div>
<div id="ref-Chen2015b" class="csl-entry">
Chen, Fei, Paul W. Tillberg, and Edward S. Boyden. 2015. <span>“Expansion Microscopy.”</span> <em>Science</em> 347 (January): 543–48. <a href="https://doi.org/10.1126/science.1260088">https://doi.org/10.1126/science.1260088</a>.
</div>
<div id="ref-Chen2016" class="csl-entry">
Chen, Fei, Asmamaw T. Wassie, Allison J. Cote, Anubhav Sinha, Shahar Alon, Shoh Asano, Evan R. Daugharthy, et al. 2016. <span>“Nanoscale Imaging of RNA with Expansion Microscopy.”</span> <em>Nature Methods</em> 13 (July): 679–84. <a href="https://doi.org/10.1038/nmeth.3899">https://doi.org/10.1038/nmeth.3899</a>.
</div>
<div id="ref-Chen2022a" class="csl-entry">
Chen, Jiawen, Weifang Liu, Tianyou Luo, Zhentao Yu, Minzhi Jiang, Jia Wen, Gaorav P. Gupta, et al. 2022. <span>“A Comprehensive Comparison on Cell-Type Composition Inference for Spatial Transcriptomics Data.”</span> <em>Briefings in Bioinformatics</em> 23 (July). <a href="https://doi.org/10.1093/BIB/BBAC245">https://doi.org/10.1093/BIB/BBAC245</a>.
</div>
<div id="ref-Chen2015" class="csl-entry">
Chen, Kok Hao, Alistair N. Boettiger, Jeffrey R. Moffitt, Siyuan Wang, and Xiaowei Zhuang. 2015. <span>“Spatially Resolved, Highly Multiplexed RNA Profiling in Single Cells.”</span> <em>Science</em>. <a href="https://doi.org/10.1126/science.aaa6090">https://doi.org/10.1126/science.aaa6090</a>.
</div>
<div id="ref-Chen2021" class="csl-entry">
Chen, Shuonan, Jackson Loper, Xiaoyin Chen, Alex Vaughan, Anthony M Zador, and Liam Paninski. 2021. <span>“BARcode DEmixing Through Non-Negative Spatial Regression (BarDensr).”</span> <em>PLOS Computational Biology</em> 17 (March): e1008256. <a href="https://doi.org/10.1371/journal.pcbi.1008256">https://doi.org/10.1371/journal.pcbi.1008256</a>.
</div>
<div id="ref-Chen2020" class="csl-entry">
Chen, Wei-Ting, Ashley Lu, Katleen Craessaerts, Benjamin Pavie, Carlo Sala Frigerio, Nikky Corthout, Xiaoyan Qian, et al. 2020. <span>“Spatial Transcriptomics and in Situ Sequencing to Study Alzheimer’s Disease.”</span> <em>Cell</em> 0 (July). <a href="https://doi.org/10.1016/j.cell.2020.06.038">https://doi.org/10.1016/j.cell.2020.06.038</a>.
</div>
<div id="ref-Chen2019" class="csl-entry">
Chen, Xiaoyin, Yu-Chi Sun, Huiqing Zhan, Justus M Kebschull, Stephan Fischer, Katherine Matho, Z Josh Huang, Jesse Gillis, and Anthony M Zador. 2019. <span>“High-Throughput Mapping of Long-Range Neuronal Projection Using &lt;Em&gt;in Situ&lt;/Em&gt; Sequencing.”</span> <em>Cell</em> 179 (October): 772–786.e19. <a href="https://doi.org/10.1016/j.cell.2019.09.023">https://doi.org/10.1016/j.cell.2019.09.023</a>.
</div>
<div id="ref-Chin2007" class="csl-entry">
Chin, Mark H., Alex B. Geng, Arshad H. Khan, Wei-Jun Qian, Vladislav A. Petyuk, Jyl Boline, Shawn Levy, et al. 2007. <span>“A Genome-Scale Map of Expression for a Mouse Brain Section Obtained Using Voxelation.”</span> <em>Physiological Genomics</em> 30 (August): 313–21. <a href="https://doi.org/10.1152/physiolgenomics.00287.2006">https://doi.org/10.1152/physiolgenomics.00287.2006</a>.
</div>
<div id="ref-Cho2021" class="csl-entry">
Cho, Chun-Seok, Jingyue Xi, Sung-Rye Park, Jer-En Hsu, Myungjin Kim, Goo Jun, Hyun-Min Kang, and Jun Hee Lee. 2021. <span>“Seq-Scope: Submicrometer-Resolution Spatial Transcriptomics for Single Cell and Subcellular Studies.”</span> <em>bioRxiv</em>, January, 2021.01.25.427807. <a href="https://doi.org/10.1101/2021.01.25.427807">https://doi.org/10.1101/2021.01.25.427807</a>.
</div>
<div id="ref-Choi2022" class="csl-entry">
Choi, Jongsu, Jin Li, Salma Ferdous, Qingnan Liang, Jeffrey R. Moffitt, and Rui Chen. 2022. <span>“Spatial Organization of the Mouse Retina at Single Cell Resolution.”</span> <em>bioRxiv</em>, December, 2022.12.04.518972. <a href="https://doi.org/10.1101/2022.12.04.518972">https://doi.org/10.1101/2022.12.04.518972</a>.
</div>
<div id="ref-Clarkson2016" class="csl-entry">
Clarkson, Melissa D. 2016. <span>“Representation of Anatomy in Online Atlases and Databases: A Survey and Collection of Patterns for Interface Design.”</span> <em>BMC Developmental Biology</em> 16 (December): 18. <a href="https://doi.org/10.1186/s12861-016-0116-y">https://doi.org/10.1186/s12861-016-0116-y</a>.
</div>
<div id="ref-Cleary2021" class="csl-entry">
Cleary, Brian, Brooke Simonton, Jon Bezney, Evan Murray, Shahul Alam, Anubhav Sinha, Ehsan Habibi, et al. 2021. <span>“Compressed Sensing for Highly Efficient Imaging Transcriptomics.”</span> <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/s41587-021-00883-x">https://doi.org/10.1038/s41587-021-00883-x</a>.
</div>
<div id="ref-Codeluppi2018" class="csl-entry">
Codeluppi, Simone, Lars E. Borm, Amit Zeisel, Gioele La Manno, Josina A. van Lunteren, Camilla I. Svensson, and Sten Linnarsson. 2018. <span>“Spatial Organization of the Somatosensory Cortex Revealed by osmFISH.”</span> <em>Nature Methods</em>. <a href="https://doi.org/10.1038/s41592-018-0175-z">https://doi.org/10.1038/s41592-018-0175-z</a>.
</div>
<div id="ref-Combs2013" class="csl-entry">
Combs, Peter A., and Michael B. Eisen. 2013. <span>“Sequencing mRNA from Cryo-Sliced Drosophila Embryos to Determine Genome-Wide Spatial Patterns of Gene Expression.”</span> Edited by Barbara Jennings. <em>PLoS ONE</em> 8 (August): e71820. <a href="https://doi.org/10.1371/journal.pone.0071820">https://doi.org/10.1371/journal.pone.0071820</a>.
</div>
<div id="ref-Combs2018" class="csl-entry">
Combs, Peter A., and Hunter B. Fraser. 2018. <span>“Spatially Varying Cis-Regulatory Divergence in Drosophila Embryos Elucidates Cis-Regulatory Logic.”</span> Edited by Claude Desplan. <em>PLOS Genetics</em> 14 (November): e1007631. <a href="https://doi.org/10.1371/journal.pgen.1007631">https://doi.org/10.1371/journal.pgen.1007631</a>.
</div>
<div id="ref-Consortium1998" class="csl-entry">
Consortium, The C.&amp;nbsp;elegans Sequencing. 1998. <span>“Genome Sequence of the Nematode c.&amp;amp;nbsp;elegans: A Platform for Investigating Biology.”</span> <em>Science</em> 282 (December): 2012–18. <a href="https://doi.org/10.1126/science.282.5396.2012">https://doi.org/10.1126/science.282.5396.2012</a>.
</div>
<div id="ref-Coskun2016a" class="csl-entry">
Coskun, Ahmet F., and Long Cai. 2016. <span>“Dense Transcript Profiling in Single Cells by Image Correlation Decoding.”</span> <em>Nature Methods</em> 13 (July): 657–60. <a href="https://doi.org/10.1038/nmeth.3895">https://doi.org/10.1038/nmeth.3895</a>.
</div>
<div id="ref-Coudry2007" class="csl-entry">
Coudry, Renata A., Sibele I. Meireles, Radka Stoyanova, Harry S. Cooper, Alan Carpino, Xianqun Wang, Paul F. Engstrom, and Margie L. Clapper. 2007. <span>“Successful Application of Microarray Technology to Microdissected Formalin-Fixed, Paraffin-Embedded Tissue.”</span> <em>The Journal of Molecular Diagnostics</em> 9 (February): 70–79. <a href="https://doi.org/10.2353/jmoldx.2007.060004">https://doi.org/10.2353/jmoldx.2007.060004</a>.
</div>
<div id="ref-Crosetto2015" class="csl-entry">
Crosetto, Nicola, Magda Bienko, and Alexander Van Oudenaarden. 2015. <span>“Spatially Resolved Transcriptomics and Beyond.”</span> <em>Nature Reviews Genetics</em>. <a href="https://doi.org/10.1038/nrg3832">https://doi.org/10.1038/nrg3832</a>.
</div>
<div id="ref-Cunnea2010" class="csl-entry">
Cunnea, Paula, Jill McMahon, Enda O’Connell, Kaveh Mashayekhi, Una Fitzgerald, and Stephen McQuaid. 2010. <span>“Gene Expression Analysis of the Microvascular Compartment in Multiple Sclerosis Using Laser Microdissected Blood Vessels.”</span> <em>Acta Neuropathologica</em> 119: 601–15. <a href="https://doi.org/10.1007/s00401-009-0618-9">https://doi.org/10.1007/s00401-009-0618-9</a>.
</div>
<div id="ref-Dana-Farber" class="csl-entry">
<span>“Dana-Farber Core Facilities.”</span> n.d. <a href="https://www.dana-farber.org/research/core-facilities/">https://www.dana-farber.org/research/core-facilities/</a>.
</div>
<div id="ref-Danaher2020" class="csl-entry">
Danaher, Patrick, Youngmi Kim, Brenn Nelson, Maddy Griswold, Zhi Yang, Erin Piazza, and Joseph M Beechem. 2020. <span>“Advances in Mixed Cell Deconvolution Enable Quantification of Cell Types in Spatially-Resolved Gene Expression Data.”</span> <em>bioRxiv</em>, January, 2020.08.04.235168. <a href="https://doi.org/10.1101/2020.08.04.235168">https://doi.org/10.1101/2020.08.04.235168</a>.
</div>
<div id="ref-Dar2021" class="csl-entry">
Dar, Daniel, Nina Dar, Long Cai, and Dianne K Newman. 2021. <span>“In Situ Single-Cell Activities of Microbial Populations Revealed by Spatial Transcriptomics.”</span> <em>bioRxiv</em>, January, 2021.02.24.432792. <a href="https://doi.org/10.1101/2021.02.24.432792">https://doi.org/10.1101/2021.02.24.432792</a>.
</div>
<div id="ref-Darnell2006" class="csl-entry">
Darnell, Diana K., Simran Kaur, Stacey Stanislaw, Jay K. Konieczka, Tatiana A. Yatskievych, and Parker B. Antin. 2006. <span>“MicroRNA Expression During Chick Embryo Development.”</span> <em>Developmental Dynamics</em> 235 (November): 3156–65. <a href="https://doi.org/10.1002/dvdy.20956">https://doi.org/10.1002/dvdy.20956</a>.
</div>
<div id="ref-Delorey2021a" class="csl-entry">
Delorey, Toni M, Carly G K Ziegler, Graham Heimberg, Rachelly Normand, Yiming Yang, Åsa Segerstolpe, Domenic Abbondanza, et al. 2021. <span>“COVID-19 Tissue Atlases Reveal SARS-CoV-2 Pathology and Cellular Targets.”</span> <em>Nature</em> 595: 107–13. <a href="https://doi.org/10.1038/s41586-021-03570-8">https://doi.org/10.1038/s41586-021-03570-8</a>.
</div>
<div id="ref-Diez-Roux2011" class="csl-entry">
Diez-Roux, Graciana, Sandro Banfi, Marc Sultan, Lars Geffers, Santosh Anand, David Rozado, Alon Magen, et al. 2011. <span>“A High-Resolution Anatomical Atlas of the Transcriptome in the Mouse Embryo.”</span> Edited by Gregory S. Barsh. <em>PLoS Biology</em> 9 (January): e1000582. <a href="https://doi.org/10.1371/journal.pbio.1000582">https://doi.org/10.1371/journal.pbio.1000582</a>.
</div>
<div id="ref-Dirks2004" class="csl-entry">
Dirks, Robert M., and Niles A. Pierce. 2004. <span>“From the Cover: Triggered Amplification by Hybridization Chain Reaction.”</span> <em>Proceedings of the National Academy of Sciences</em> 101 (October): 15275–78. <a href="https://doi.org/10.1073/pnas.0407024101">https://doi.org/10.1073/pnas.0407024101</a>.
</div>
<div id="ref-Dobin2012" class="csl-entry">
Dobin, Alexander, Carrie A Davis, Felix Schlesinger, Jorg Drenkow, Chris Zaleski, Sonali Jha, Philippe Batut, Mark Chaisson, and Thomas R Gingeras. 2012. <span>“STAR: Ultrafast Universal RNA-Seq Aligner.”</span> <em>Bioinformatics</em> 29 (October): 15–21. <a href="https://doi.org/10.1093/bioinformatics/bts635">https://doi.org/10.1093/bioinformatics/bts635</a>.
</div>
<div id="ref-Dong2021" class="csl-entry">
Dong, Kangning, and Shihua Zhang. 2021. <span>“Deciphering Spatial Domains from Spatially Resolved Transcriptomics with Adaptive Graph Attention Auto-Encoder.”</span> <em>bioRxiv</em>, January, 2021.08.21.457240. <a href="https://doi.org/10.1101/2021.08.21.457240">https://doi.org/10.1101/2021.08.21.457240</a>.
</div>
<div id="ref-Dou2020" class="csl-entry">
Dou, Jinzhuang, Shaoheng Liang, Vakul Mohanty, Xuesen Cheng, Sangbae Kim, Jongsu Choi, Yumei Li, Katayoun Rezvani, Rui Chen, and Ken Chen. 2020. <span>“Unbiased Integration of Single Cell Multi-Omics Data.”</span> <em>bioRxiv</em>, January, 2020.12.11.422014. <a href="https://doi.org/10.1101/2020.12.11.422014">https://doi.org/10.1101/2020.12.11.422014</a>.
</div>
<div id="ref-Dries2021" class="csl-entry">
Dries, Ruben, Qian Zhu, Rui Dong, Eng Chee-Huat Linus, Huipeng Li, Kan Liu, Yuntian Fu, et al. 2021. <span>“Giotto: A Toolbox for Integrative Analysis and Visualization of Spatial Expression Data.”</span> <em>Genome Biol.</em> 22 (March): 78.
</div>
<div id="ref-Drmanac2010" class="csl-entry">
Drmanac, R., Andrew B. Sparks, Matthew J. Callow, Aaron L. Halpern, Norman L. Burns, Bahram G. Kermani, Paolo Carnevali, et al. 2010. <span>“Human Genome Sequencing Using Unchained Base Reads on Self-Assembling DNA Nanoarrays.”</span> <em>Science</em> 327 (January): 78–81. <a href="https://doi.org/10.1126/science.1181498">https://doi.org/10.1126/science.1181498</a>.
</div>
<div id="ref-Droin2020" class="csl-entry">
Droin, Colas, Jakob El Kholtei, Keren Bahar Halpern, Clémence Hurni, Milena Rozenberg, Sapir Muvkadi, Shalev Itzkovitz, and Felix Naef. 2020. <span>“Space-Time Logic of Liver Gene Expression at Sublobular Scale.”</span> <em>bioRxiv</em>, January, 2020.03.05.976571. <a href="https://doi.org/10.1101/2020.03.05.976571">https://doi.org/10.1101/2020.03.05.976571</a>.
</div>
<div id="ref-nanostring_tap" class="csl-entry">
<span>“DSP Technology Access Program (TAP).”</span> n.d.
</div>
<div id="ref-Dulac1995" class="csl-entry">
Dulac, Catherine, and Richard Axel. 1995. <span>“A Novel Family of Genes Encoding Putative Pheromone Receptors in Mammals.”</span> <em>Cell</em> 83: 195–206. https://doi.org/<a href="https://doi.org/10.1016/0092-8674(95)90161-2">https://doi.org/10.1016/0092-8674(95)90161-2</a>.
</div>
<div id="ref-Durruthy-Durruthy2016" class="csl-entry">
Durruthy-Durruthy, Jens, Mark Wossidlo, Sunil Pai, Yusuke Takahashi, Gugene Kang, Larsson Omberg, Bertha Chen, Hiromitsu Nakauchi, Renee Reijo Pera, and Vittorio Sebastiano. 2016. <span>“Spatiotemporal Reconstruction of the Human Blastocyst by Single-Cell Gene-Expression Analysis Informs Induction of Naive Pluripotency.”</span> <em>Developmental Cell</em> 38 (July): 100–115. <a href="https://doi.org/10.1016/j.devcel.2016.06.014">https://doi.org/10.1016/j.devcel.2016.06.014</a>.
</div>
<div id="ref-Durruthy-Durruthy2014" class="csl-entry">
Durruthy-Durruthy, Robert, Assaf Gottlieb, Byron H. Hartman, Jörg Waldhaus, Roman D. Laske, Russ Altman, and Stefan Heller. 2014. <span>“Reconstruction of the Mouse Otocyst and Early Neuroblast Lineage at Single-Cell Resolution.”</span> <em>Cell</em> 157 (May): 964–78. <a href="https://doi.org/10.1016/j.cell.2014.03.036">https://doi.org/10.1016/j.cell.2014.03.036</a>.
</div>
<div id="ref-Ebbing2018" class="csl-entry">
Ebbing, Annabel, Abel Vertesy, Marco Betist, Bastiaan Spanjaard, Jan Philipp Junker, Eugene Berezikov, Alexander van Oudenaarden, and Hendrik Korswagen. 2018. <span>“Spatial Transcriptomics of c. Elegans Males and Hermaphrodites Identifies Novel Fertility Genes.”</span> <em>bioRxiv</em>, June, 348201. <a href="https://doi.org/10.1101/348201">https://doi.org/10.1101/348201</a>.
</div>
<div id="ref-Eberwine1992" class="csl-entry">
Eberwine, J, H Yeh, K Miyashiro, Y Cao, S Nair, R Finnell, M Zettel, and P Coleman. 1992. <span>“Analysis of Gene Expression in Single Live Neurons.”</span> <em>Proceedings of the National Academy of Sciences</em> 89 (April): 3010 LP–3014. <a href="https://doi.org/10.1073/pnas.89.7.3010">https://doi.org/10.1073/pnas.89.7.3010</a>.
</div>
<div id="ref-Edsgard2018" class="csl-entry">
Edsgärd, Daniel, Per Johnsson, and Rickard Sandberg. 2018. <span>“Identification of Spatial Expression Trends in Single-Cell Gene Expression Data.”</span> <em>Nature Methods</em> 15 (May): 339–42. <a href="https://doi.org/10.1038/nmeth.4634">https://doi.org/10.1038/nmeth.4634</a>.
</div>
<div id="ref-Efremova2020" class="csl-entry">
Efremova, Mirjana, Miquel Vento-Tormo, Sarah A Teichmann, and Roser Vento-Tormo. 2020. <span>“CellPhoneDB: Inferring Cell–Cell Communication from Combined Expression of Multi-Subunit Ligand–Receptor Complexes.”</span> <em>Nature Protocols</em> 15: 1484–1506. <a href="https://doi.org/10.1038/s41596-020-0292-x">https://doi.org/10.1038/s41596-020-0292-x</a>.
</div>
<div id="ref-Eichenberger2020" class="csl-entry">
Eichenberger, Bastian Th., YinXiu Zhan, Markus Rempfler, Luca Giorgetti, and Jeffrey A Chao. 2020. <span>“deepBlink: Threshold-Independent Detection and Localization of Diffraction-Limited Spots.”</span> <em>bioRxiv</em>, January, 2020.12.14.422631. <a href="https://doi.org/10.1101/2020.12.14.422631">https://doi.org/10.1101/2020.12.14.422631</a>.
</div>
<div id="ref-Elosua2020" class="csl-entry">
Elosua, Marc, Paula Nieto, Elisabetta Mereu, Ivo Gut, and Holger Heyn. 2020. <span>“SPOTlight: Seeded NMF Regression to Deconvolute Spatial Transcriptomics Spots with Single-Cell Transcriptomes.”</span> <em>bioRxiv</em>, June, 2020.06.03.131334. <a href="https://doi.org/10.1101/2020.06.03.131334">https://doi.org/10.1101/2020.06.03.131334</a>.
</div>
<div id="ref-Emmert-Buck1996" class="csl-entry">
Emmert-Buck, Michael R., Robert F. Bonner, Paul D. Smith, Rodrigo F. Chuaqui, Zhengping Zhuang, Seth R. Goldstein, Rhonda A. Weiss, and Lance A. Liotta. 1996. <span>“Laser Capture Microdissection.”</span> <em>Science</em> 274 (November): 998–1001. <a href="https://doi.org/10.1126/science.274.5289.998">https://doi.org/10.1126/science.274.5289.998</a>.
</div>
<div id="ref-Eng2019a" class="csl-entry">
Eng, Chee Huat Linus, Michael Lawson, Qian Zhu, Ruben Dries, Noushin Koulena, Yodai Takei, Jina Yun, et al. 2019. <span>“Transcriptome-Scale Super-Resolved Imaging in Tissues by RNA seqFISH+.”</span> <em>Nature</em> 568 (April): 235–39. <a href="https://doi.org/10.1038/s41586-019-1049-y">https://doi.org/10.1038/s41586-019-1049-y</a>.
</div>
<div id="ref-Eng2017" class="csl-entry">
Eng, Chee-Huat Linus, Sheel Shah, Julian Thomassie, and Long Cai. 2017. <span>“Profiling the Transcriptome with RNA SPOTs.”</span> <em>Nature Methods</em> 14: 1153–55. <a href="https://doi.org/10.1038/nmeth.4500">https://doi.org/10.1038/nmeth.4500</a>.
</div>
<div id="ref-Fan2020" class="csl-entry">
Fan, Zhen, Runsheng Chen, and Xiaowei Chen. 2020. <span>“SpatialDB: A Database for Spatially Resolved Transcriptomes.”</span> <em>Nucleic Acids Research</em> 48 (January): D233–37. <a href="https://doi.org/10.1093/nar/gkz934">https://doi.org/10.1093/nar/gkz934</a>.
</div>
<div id="ref-Farris2019" class="csl-entry">
Farris, Shannon, James M Ward, Kelly E Carstens, Mahsa Samadi, Yu Wang, and Serena M Dudek. 2019. <span>“Hippocampal Subregions Express Distinct Dendritic Transcriptomes That Reveal Differences in Mitochondrial Function in CA2.”</span> <em>Cell Reports</em> 29 (October): 522–539.e6. <a href="https://doi.org/10.1016/j.celrep.2019.08.093">https://doi.org/10.1016/j.celrep.2019.08.093</a>.
</div>
<div id="ref-Fazal2019a" class="csl-entry">
Fazal, Furqan M., Shuo Han, Kevin R. Parker, Pornchai Kaewsapsak, Jin Xu, Alistair N. Boettiger, Howard Y. Chang, and Alice Y. Ting. 2019. <span>“Atlas of Subcellular RNA Localization Revealed by APEX-Seq.”</span> <em>Cell</em> 178 (July): 473–490.e26. <a href="https://doi.org/10.1016/j.cell.2019.05.027">https://doi.org/10.1016/j.cell.2019.05.027</a>.
</div>
<div id="ref-Femino1998" class="csl-entry">
Femino, Andrea M, Fredric S Fay, Kevin Fogarty, and Robert H Singer. 1998. <span>“Visualization of Single RNA Transcripts in Situ.”</span> <em>Science</em> 280 (April): 585 LP–590. <a href="https://doi.org/10.1126/science.280.5363.585">https://doi.org/10.1126/science.280.5363.585</a>.
</div>
<div id="ref-Fire1995" class="csl-entry">
Fire, Andrew, and Si Qun Xu. 1995. <span>“Rolling Replication of Short DNA Circles.”</span> <em>Proceedings of the National Academy of Sciences</em> 92 (May): 4641–45. <a href="https://doi.org/10.1073/pnas.92.10.4641">https://doi.org/10.1073/pnas.92.10.4641</a>.
</div>
<div id="ref-Fleck2021" class="csl-entry">
Fleck, Jonas Simon, Fátima Sanchís-Calleja, Zhisong He, Malgorzata Santel, Michael James Boyle, J Gray Camp, and Barbara Treutlein. 2021. <span>“Resolving Organoid Brain Region Identities by Mapping Single-Cell Genomic Data to Reference Atlases.”</span> <em>Cell Stem Cell</em> 28: 1148–1159.e8. https://doi.org/<a href="https://doi.org/10.1016/j.stem.2021.02.015">https://doi.org/10.1016/j.stem.2021.02.015</a>.
</div>
<div id="ref-Foley2019" class="csl-entry">
Foley, Joseph W., Chunfang Zhu, Philippe Jolivet, Shirley X. Zhu, Peipei Lu, Michael J. Meaney, and Robert B. West. 2019. <span>“Gene Expression Profiling of Single Cells from Archival Tissue with Laser-Capture Microdissection and Smart-3SEQ.”</span> <em>Genome Research</em> 29 (November): 1816–25. <a href="https://doi.org/10.1101/gr.234807.118">https://doi.org/10.1101/gr.234807.118</a>.
</div>
<div id="ref-Foreman2019" class="csl-entry">
Foreman, Robert, and Roy Wollman. 2019. <span>“Mammalian Gene Expression Variability Is Explained by Underlying Cell State.”</span> <em>bioRxiv</em>. <a href="https://doi.org/10.1101/626424">https://doi.org/10.1101/626424</a>.
</div>
<div id="ref-Forrester1996" class="csl-entry">
Forrester, Lesley M., Andras Nagy, Mehran Sam, Alistair Watt, Lois Stevenson, Alan Bernstein, Alexandra L. Joyner, and Wolfgang Wurst. 1996. <span>“An Induction Gene Trap Screen in Embryonic Stem Cells: Identification of Genes That Respond to Retinoic Acid in Vitro.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 93 (February): 1677–82. <a href="https://doi.org/10.1073/pnas.93.4.1677">https://doi.org/10.1073/pnas.93.4.1677</a>.
</div>
<div id="ref-Fowlkes2008" class="csl-entry">
Fowlkes, Charless C., Cris L. Luengo Hendriks, Soile V. E. Keränen, Gunther H. Weber, Oliver Rübel, Min-Yu Huang, Sohail Chatoor, et al. 2008. <span>“A Quantitative Spatiotemporal Atlas of Gene Expression in the Drosophila Blastoderm.”</span> <em>Cell</em> 133 (April): 364–74. <a href="https://doi.org/10.1016/j.cell.2008.01.053">https://doi.org/10.1016/j.cell.2008.01.053</a>.
</div>
<div id="ref-Friedrich1991" class="csl-entry">
Friedrich, G., and P. Soriano. 1991. <span>“Promoter Traps in Embryonic Stem Cells: A Genetic Screen to Identify and Mutate Developmental Genes in Mice.”</span> <em>Genes &amp; Development</em> 5 (September): 1513–23. <a href="https://doi.org/10.1101/gad.5.9.1513">https://doi.org/10.1101/gad.5.9.1513</a>.
</div>
<div id="ref-Frise2010" class="csl-entry">
Frise, Erwin, Ann S Hammonds, and Susan E Celniker. 2010. <span>“Systematic Image‐driven Analysis of the Spatial Drosophila Embryonic Expression Landscape.”</span> <em>Molecular Systems Biology</em> 6 (January): 345. <a href="https://doi.org/10.1038/msb.2009.102">https://doi.org/10.1038/msb.2009.102</a>.
</div>
<div id="ref-Frohman1988" class="csl-entry">
Frohman, M. A., M. K. Dush, and G. R. Martin. 1988. <span>“Rapid Production of Full-Length cDNAs from Rare Transcripts: Amplification Using a Single Gene-Specific Oligonucleotide Primer.”</span> <em>Proceedings of the National Academy of Sciences</em> 85 (December): 8998–9002. <a href="https://doi.org/10.1073/pnas.85.23.8998">https://doi.org/10.1073/pnas.85.23.8998</a>.
</div>
<div id="ref-Fu2021" class="csl-entry">
Fu, Xiaonan, Li Sun, Jane Y Chen, Runze Dong, Yiing Lin, Richard D Palmiter, Shin Lin, and Liangcai Gu. 2021. <span>“Continuous Polony Gels for Tissue Mapping with High Resolution and RNA Capture Efficiency.”</span> <em>bioRxiv</em>, January, 2021.03.17.435795. <a href="https://doi.org/10.1101/2021.03.17.435795">https://doi.org/10.1101/2021.03.17.435795</a>.
</div>
<div id="ref-Furth2019" class="csl-entry">
Fürth, Daniel, Victor Hatini, and Je H Lee. 2019. <span>“In Situ Transcriptome Accessibility Sequencing (INSTA-Seq).”</span> <em>bioRxiv</em>, January, 722819. <a href="https://doi.org/10.1101/722819">https://doi.org/10.1101/722819</a>.
</div>
<div id="ref-Gall1969" class="csl-entry">
Gall, Joseph G, Mary Lou, Pardue Kline, and Norman H Giles. 1969. <span>“Formation and Detecction of RNA-DNA Hybrid Molecules in Cytological Preparations.”</span> <em>PNAS</em> 63: 378–83. <a href="https://doi.org/10.1073/pnas.63.2.378">https://doi.org/10.1073/pnas.63.2.378</a>.
</div>
<div id="ref-Gawantka1998" class="csl-entry">
Gawantka, Volker, Nicolas Pollet, Hajo Delius, Martin Vingron, Ralf Pfister, Rebecca Nitsch, Claudia Blumenstock, and Christof Niehrs. 1998. <span>“Gene Expression Screening in Xenopus Identifies Molecular Pathways, Predicts Gene Function and Provides a Global View of Embryonic Patterning.”</span> <em>Mechanisms of Development</em> 77 (September): 95–141. <a href="https://doi.org/10.1016/S0925-4773(98)00115-4">https://doi.org/10.1016/S0925-4773(98)00115-4</a>.
</div>
<div id="ref-Gebhardt2014" class="csl-entry">
Gebhardt, Rolf. 2014. <span>“Liver Zonation: Novel Aspects of Its Regulation and Its Impact on Homeostasis.”</span> <em>World Journal of Gastroenterology</em> 20 (July): 8491. <a href="https://doi.org/10.3748/wjg.v20.i26.8491">https://doi.org/10.3748/wjg.v20.i26.8491</a>.
</div>
<div id="ref-Geffers2015" class="csl-entry">
Geffers, Lars, and Gregor Eichele. 2015. <span>“High-Throughput in Situ Hybridization: Systematical Production of Gene Expression Data and Beyond BT - in Situ Hybridization Methods.”</span> Edited by Giselbert Hauptmann. <em>In Situ Hybridization Methods</em>, 221–45. <a href="https://doi.org/10.1007/978-1-4939-2303-8_11">https://doi.org/10.1007/978-1-4939-2303-8_11</a>.
</div>
<div id="ref-Geffers2013" class="csl-entry">
Geffers, Lars, Benjamin Tetzlaff, Xiao Cui, Jun Yan, and Gregor Eichele. 2012. <span>“METscout: A Pathfinder Exploring the Landscape of Metabolites, Enzymes and Transporters.”</span> <em>Nucleic Acids Research</em> 41 (September): D1047–54. <a href="https://doi.org/10.1093/nar/gks886">https://doi.org/10.1093/nar/gks886</a>.
</div>
<div id="ref-VanGelder1990" class="csl-entry">
Gelder, R N Van, M E von Zastrow, A Yool, W C Dement, J D Barchas, and J H Eberwine. 1990. <span>“Amplified RNA Synthesized from Limited Quantities of Heterogeneous cDNA.”</span> <em>Proceedings of the National Academy of Sciences</em> 87 (March): 1663 LP–1667. <a href="http://www.pnas.org/content/87/5/1663.abstract">http://www.pnas.org/content/87/5/1663.abstract</a>.
</div>
<div id="ref-DSPPanels" class="csl-entry">
<span>“Gene Expression Panels | NanoString Technologies.”</span> n.d. <a href="https://www.nanostring.com/products/gene-expression-panels/gene-expression-panels-overview">https://www.nanostring.com/products/gene-expression-panels/gene-expression-panels-overview</a>.
</div>
<div id="ref-Ghazanfar2020" class="csl-entry">
Ghazanfar, Shila, Yingxin Lin, Xianbin Su, David Ming Lin, Ellis Patrick, Ze-Guang Han, John C Marioni, and Jean Yee Hwa Yang. 2020. <span>“Investigating Higher-Order Interactions in Single-Cell Data with scHOT.”</span> <em>Nature Methods</em> 17: 799–806. <a href="https://doi.org/10.1038/s41592-020-0885-x">https://doi.org/10.1038/s41592-020-0885-x</a>.
</div>
<div id="ref-Giani2020" class="csl-entry">
Giani, Alice Maria, Guido Roberto Gallo, Luca Gianfranceschi, and Giulio Formenti. 2020. <span>“Long Walk to Genomics: History and Current Approaches to Genome Sequencing and Assembly.”</span> <em>Computational and Structural Biotechnology Journal</em> 18 (January): 9–19. <a href="https://doi.org/10.1016/j.csbj.2019.11.002">https://doi.org/10.1016/j.csbj.2019.11.002</a>.
</div>
<div id="ref-Gilchrist2009" class="csl-entry">
Gilchrist, Michael J., Mikkel B. Christensen, Odile Bronchain, Frédéric Brunet, Albert Chesneau, Ursula Fenger, Timothy J. Geach, et al. 2009. <span>“Database of Queryable Gene Expression Patterns for Xenopus.”</span> <em>Developmental Dynamics</em> 238 (June): 1379–88. <a href="https://doi.org/10.1002/dvdy.21940">https://doi.org/10.1002/dvdy.21940</a>.
</div>
<div id="ref-Giolai2019" class="csl-entry">
Giolai, Michael, Walter Verweij, Ashleigh Lister, Darren Heavens, Iain Macaulay, and Matthew D. Clark. 2019. <span>“Spatially Resolved Transcriptomics Reveals Plant Host Responses to Pathogens.”</span> <em>Plant Methods</em> 15 (December): 114. <a href="https://doi.org/10.1186/s13007-019-0498-5">https://doi.org/10.1186/s13007-019-0498-5</a>.
</div>
<div id="ref-DeGiovanni2020" class="csl-entry">
Giovanni, Marco De, Valeria Cutillo, Amir Giladi, Eleonora Sala, Carmela G Maganuco, Chiara Medaglia, Pietro Di Lucia, et al. 2020. <span>“Spatiotemporal Regulation of Type i Interferon Expression Determines the Antiviral Polarization of CD4+ t Cells.”</span> <em>Nature Immunology</em> 21: 321–30. <a href="https://doi.org/10.1038/s41590-020-0596-6">https://doi.org/10.1038/s41590-020-0596-6</a>.
</div>
<div id="ref-Glaser2015" class="csl-entry">
Glaser, Joshua I., Bradley M. Zamft, George M. Church, and Konrad P. Kording. 2015. <span>“Puzzle Imaging: Using Large-Scale Dimensionality Reduction Algorithms for Localization.”</span> Edited by Joseph Najbauer. <em>PLOS ONE</em> 10 (July): e0131593. <a href="https://doi.org/10.1371/journal.pone.0131593">https://doi.org/10.1371/journal.pone.0131593</a>.
</div>
<div id="ref-Goh2020" class="csl-entry">
Goh, Jolene Jie Lin, Nigel Chou, Wan Yi Seow, Norbert Ha, Chung Pui Paul Cheng, Yun Ching Chang, Ziqing Winston Zhao, and Kok Hao Chen. 2020. <span>“Highly Specific Multiplexed RNA Imaging in Tissues with Split-FISH.”</span> <em>Nature Methods</em> 17 (July): 689–93. <a href="https://doi.org/10.1038/s41592-020-0858-0">https://doi.org/10.1038/s41592-020-0858-0</a>.
</div>
<div id="ref-Gong2003" class="csl-entry">
Gong, Shiaoching, Chen Zheng, Martin L. Doughty, Kasia Losos, Nicholas Didkovsky, Uta B. Schambra, Norma J. Nowak, et al. 2003. <span>“A Gene Expression Atlas of the Central Nervous System Based on Bacterial Artificial Chromosomes.”</span> <em>Nature</em> 425 (October): 917–25. <a href="https://doi.org/10.1038/nature02033">https://doi.org/10.1038/nature02033</a>.
</div>
<div id="ref-BravoGonzalez-Blas2020" class="csl-entry">
González-Blas, Carmen Bravo, Xiao-Jiang Quan, Ramon Duran-Romaña, Ibrahim Ihsan Taskiran, Duygu Koldere, Kristofer Davie, Valerie Christiaens, et al. 2020. <span>“Identification of Genomic Enhancers Through Spatial Integration of Single-Cell Transcriptomics and Epigenomics.”</span> <em>Molecular Systems Biology</em> 16 (May): e9438. <a href="https://doi.org/10.15252/msb.20209438">https://doi.org/10.15252/msb.20209438</a>.
</div>
<div id="ref-Gossler1989" class="csl-entry">
Gossler, Achim, A. Joyner, Janet Rossant, and W. Skarnes. 1989. <span>“Mouse Embryonic Stem Cells and Reporter Constructs to Detect Developmentally Regulated Genes.”</span> <em>Science</em> 244 (April): 463–65. <a href="https://doi.org/10.1126/science.2497519">https://doi.org/10.1126/science.2497519</a>.
</div>
<div id="ref-Govek2019" class="csl-entry">
Govek, Kiya W, Venkata S Yamajala, and Pablo G Camara. 2019. <span>“Clustering-Independent Analysis of Genomic Data Using Spectral Simplicial Theory.”</span> <em>PLOS Computational Biology</em> 15 (November): e1007509. <a href="https://doi.org/10.1371/journal.pcbi.1007509">https://doi.org/10.1371/journal.pcbi.1007509</a>.
</div>
<div id="ref-Grange2014" class="csl-entry">
Grange, Pascal, Jason W. Bohland, Benjamin W. Okaty, Ken Sugino, Hemant Bokil, Sacha B. Nelson, Lydia Ng, Michael Hawrylycz, and Partha P. Mitra. 2014. <span>“Cell-Type-Based Model Explaining Coexpression Patterns of Genes in the Brain.”</span> <em>Proceedings of the National Academy of Sciences</em> 111 (April): 5397–5402. <a href="https://doi.org/10.1073/pnas.1312098111">https://doi.org/10.1073/pnas.1312098111</a>.
</div>
<div id="ref-Gregory2020" class="csl-entry">
Gregory, J M, K McDade, M R Livesey, I Croy, S Marion de Proce, T Aitman, S Chandran, and C Smith. 2020. <span>“Spatial Transcriptomics Identifies Spatially Dysregulated Expression of GRM3 and USP47 in Amyotrophic Lateral Sclerosis.”</span> <em>Neuropathology and Applied Neurobiology</em> 46 (August): 441–57. <a href="https://doi.org/10.1111/nan.12597">https://doi.org/10.1111/nan.12597</a>.
</div>
<div id="ref-Greulich1999" class="csl-entry">
Greulich, Karl Otto. 1999. <span>“Introduction: The History of Using Light as a Working Tool.”</span> <em>Micromanipulation by Light in Biology and Medicine</em>, 1–6. <a href="https://doi.org/10.1007/978-1-4612-4110-2_1">https://doi.org/10.1007/978-1-4612-4110-2_1</a>.
</div>
<div id="ref-Grun2014" class="csl-entry">
Grün, Dominic, Lennart Kester, and Alexander van Oudenaarden. 2014. <span>“Validation of Noise Models for Single-Cell Transcriptomics.”</span> <em>Nature Methods</em> 11 (June): 637–40. <a href="https://doi.org/10.1038/nmeth.2930">https://doi.org/10.1038/nmeth.2930</a>.
</div>
<div id="ref-Guroff2016" class="csl-entry">
Guroff, Margaret. 2016. <span>“American Drivers Have Bicyclists to Thank for a Smooth Ride to Work.”</span> <a href="https://www.smithsonianmag.com/travel/american-drivers-thank-bicyclists-180960399/">https://www.smithsonianmag.com/travel/american-drivers-thank-bicyclists-180960399/</a>.
</div>
<div id="ref-Gurunathan2004" class="csl-entry">
Gurunathan, Rajalakshmi, Bernard Van Emden, Sethuraman Panchanathan, and Sudhir Kumar. 2004. <span>“Identifying Spatially Similar Gene Expression Patterns in Early Stage Fruit Fly Embryo Images: Binary Feature Versus Invariant Moment Digital Representations.”</span> <em>BMC Bioinformatics</em> 5 (December): 202. <a href="https://doi.org/10.1186/1471-2105-5-202">https://doi.org/10.1186/1471-2105-5-202</a>.
</div>
<div id="ref-Gyllborg2020" class="csl-entry">
Gyllborg, Daniel, Christoffer Mattsson Langseth, Xiaoyan Qian, Sergio Marco Salas, Markus Hilscher, Ed Lein, and Mats Nilsson. 2020. <span>“Hybridization-Based in Situ Sequencing (HybISS): Spatial Transcriptomic Detection in Human and Mouse Brain Tissue.”</span> <em>bioRxiv</em>, February, 2020.02.03.931618. <a href="https://doi.org/10.1101/2020.02.03.931618">https://doi.org/10.1101/2020.02.03.931618</a>.
</div>
<div id="ref-Halpern2018" class="csl-entry">
Halpern, Keren Bahar, Rom Shenhav, Hassan Massalha, Beata Toth, Adi Egozi, Efi E. Massasa, Chiara Medgalia, et al. 2018. <span>“Paired-Cell Sequencing Enables Spatial Gene Expression Mapping of Liver Endothelial Cells.”</span> <em>Nature Biotechnology</em> 36 (November): 962. <a href="https://doi.org/10.1038/nbt.4231">https://doi.org/10.1038/nbt.4231</a>.
</div>
<div id="ref-Halpern2017" class="csl-entry">
Halpern, Keren Bahar, Rom Shenhav, Orit Matcovitch-Natan, Beáta Tóth, Doron Lemze, Matan Golan, Efi E. Massasa, et al. 2017. <span>“Single-Cell Spatial Reconstruction Reveals Global Division of Labour in the Mammalian Liver.”</span> <em>Nature</em> 542 (February): 1–5. <a href="https://doi.org/10.1038/nature21065">https://doi.org/10.1038/nature21065</a>.
</div>
<div id="ref-Hammonds2013" class="csl-entry">
Hammonds, Ann S., Christopher A. Bristow, William W. Fisher, Richard Weiszmann, Siqi Wu, Volker Hartenstein, Manolis Kellis, Bin Yu, Erwin Frise, and Susan E. Celniker. 2013. <span>“Spatial Expression of Transcription Factors in Drosophila Embryonic Organ Development.”</span> <em>Genome Biology</em> 14 (December): R140. <a href="https://doi.org/10.1186/gb-2013-14-12-r140">https://doi.org/10.1186/gb-2013-14-12-r140</a>.
</div>
<div id="ref-Hao2021" class="csl-entry">
Hao, Yuhan, Stephanie Hao, Erica Andersen-Nissen, William M Mauck 3rd, Shiwei Zheng, Andrew Butler, Maddie J Lee, et al. 2021. <span>“Integrated Analysis of Multimodal Single-Cell Data.”</span> <em>Cell</em> 184 (June): 3573–3587.e29.
</div>
<div id="ref-Harding2011" class="csl-entry">
Harding, Simon D., Chris Armit, Jane Armstrong, Jane Brennan, Ying Cheng, Bernard Haggarty, Derek Houghton, et al. 2011. <span>“The GUDMAP Database - an Online Resource for Genitourinary Research.”</span> <em>Development</em> 138 (July): 2845–53. <a href="https://doi.org/10.1242/dev.063594">https://doi.org/10.1242/dev.063594</a>.
</div>
<div id="ref-Harrison1973" class="csl-entry">
Harrison, P. R., D. Conkie, J. Paul, and K. Jones. 1973. <span>“Localisation of Cellular Globin Messenger RNA by in Situ Hybridisation to Complementary DNA.”</span> <em>FEBS Letters</em> 32 (May): 109–12. <a href="https://doi.org/10.1016/0014-5793(73)80749-5">https://doi.org/10.1016/0014-5793(73)80749-5</a>.
</div>
<div id="ref-Hashimshony2016" class="csl-entry">
Hashimshony, Tamar, Naftalie Senderovich, Gal Avital, Agnes Klochendler, Yaron de Leeuw, Leon Anavy, Dave Gennert, et al. 2016. <span>“CEL-Seq2: Sensitive Highly-Multiplexed Single-Cell RNA-Seq.”</span> <em>Genome Biology</em> 17 (December): 77. <a href="https://doi.org/10.1186/s13059-016-0938-8">https://doi.org/10.1186/s13059-016-0938-8</a>.
</div>
<div id="ref-Hashimshony2012" class="csl-entry">
Hashimshony, Tamar, Florian Wagner, Noa Sher, and Itai Yanai. 2012. <span>“CEL-Seq: Single-Cell RNA-Seq by Multiplexed Linear Amplification.”</span> <em>Cell Reports</em> 2: 666–73. https://doi.org/<a href="https://doi.org/10.1016/j.celrep.2012.08.003">https://doi.org/10.1016/j.celrep.2012.08.003</a>.
</div>
<div id="ref-Haudry2007" class="csl-entry">
Haudry, Yannick, Hugo Berube, Ivica Letunic, P.-D. Weeber, Julien Gagneur, Charles Girardot, Misha Kapushesky, et al. 2007. <span>“4DXpress: A Database for Cross-Species Expression Pattern Comparisons.”</span> <em>Nucleic Acids Research</em> 36 (December): D847–53. <a href="https://doi.org/10.1093/nar/gkm797">https://doi.org/10.1093/nar/gkm797</a>.
</div>
<div id="ref-Hawrylycz2012" class="csl-entry">
Hawrylycz, Michael J., Ed S. Lein, Angela L. Guillozet-Bongaarts, Elaine H. Shen, Lydia Ng, Jeremy A. Miller, Louie N. van de Lagemaat, et al. 2012. <span>“An Anatomically Comprehensive Atlas of the Adult Human Brain Transcriptome.”</span> <em>Nature</em> 489 (September): 391–99. <a href="https://doi.org/10.1038/nature11405">https://doi.org/10.1038/nature11405</a>.
</div>
<div id="ref-He2020" class="csl-entry">
He, Bryan, Ludvig Bergenstråhle, Linnea Stenbeck, Abubakar Abid, Alma Andersson, Åke Borg, Jonas Maaskola, Joakim Lundeberg, and James Zou. 2020. <span>“Integrating Spatial Gene Expression and Breast Tumour Morphology via Deep Learning.”</span> <em>Nature Biomedical Engineering</em>, June, 1–8. <a href="https://doi.org/10.1038/s41551-020-0578-x">https://doi.org/10.1038/s41551-020-0578-x</a>.
</div>
<div id="ref-Heffel2008a" class="csl-entry">
Heffel, Andreas, Peter F. Stadler, Sonja J. Prohaska, Gerhard Kauer, and Jens-Peer Kuska. 2008. <span>“Process Flow for Classification and Clustering of Fruit Fly Gene Expression Patterns.”</span> In <em>2008 15th IEEE International Conference on Image Processing</em>, 721–24. IEEE. <a href="https://doi.org/10.1109/ICIP.2008.4711856">https://doi.org/10.1109/ICIP.2008.4711856</a>.
</div>
<div id="ref-Hemby2002" class="csl-entry">
Hemby, Scott E, Stephen D Ginsberg, Brian Brunk, Steven E Arnold, John Q Trojanowski, and James H Eberwine. 2002. <span>“Gene Expression Profile for Schizophrenia: Discrete Neuron Transcription Patterns in the Entorhinal Cortex.”</span> <em>Archives of General Psychiatry</em> 59 (July): 631–40. <a href="https://doi.org/10.1001/archpsyc.59.7.631">https://doi.org/10.1001/archpsyc.59.7.631</a>.
</div>
<div id="ref-LuengoHendriks2006a" class="csl-entry">
Hendriks, Cris L. Luengo, Soile V. E. Keränen, Charless C. Fowlkes, Lisa Simirenko, Gunther H. Weber, Angela H. DePace, Clara Henriquez, et al. 2006. <span>“Three-Dimensional Morphology and Gene Expression in the Drosophila Blastoderm at Cellular Resolution i: Data Acquisition Pipeline.”</span> <em>Genome Biology</em> 7 (December): R123. <a href="https://doi.org/10.1186/gb-2006-7-12-r123">https://doi.org/10.1186/gb-2006-7-12-r123</a>.
</div>
<div id="ref-Henrich2003" class="csl-entry">
Henrich, Thorsten. 2003. <span>“MEPD: A Medaka Gene Expression Pattern Database.”</span> <em>Nucleic Acids Research</em> 31 (January): 72–74. <a href="https://doi.org/10.1093/nar/gkg017">https://doi.org/10.1093/nar/gkg017</a>.
</div>
<div id="ref-Hiwatashi2001" class="csl-entry">
Hiwatashi, Yuji, Tomoaki Nishiyama, Tomomichi Fujita, and Mitsuyasu Hasebe. 2001. <span>“Establishment of Gene-Trap and Enhancer-Trap Systems in the Moss Physcomitrella Patens.”</span> <em>The Plant Journal</em> 28 (December): 105–16. <a href="https://doi.org/10.1046/j.1365-313X.2001.01121.x">https://doi.org/10.1046/j.1365-313X.2001.01121.x</a>.
</div>
<div id="ref-Hoffecker2019" class="csl-entry">
Hoffecker, Ian T., Yunshi Yang, Giulio Bernardinelli, Pekka Orponen, and Björn Högberg. 2019. <span>“A Computational Framework for DNA Sequencing Microscopy.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 116 (September): 19282–87. <a href="https://doi.org/10.1073/pnas.1821178116">https://doi.org/10.1073/pnas.1821178116</a>.
</div>
<div id="ref-Hope1991" class="csl-entry">
Hope, I A. 1991. <span>“’Promoter Trapping’ in Caenorhabditis Elegans.”</span> <em>Development</em>. Vol. 113. <a href="https://dev.biologists.org/content/develop/113/2/399.full.pdf">https://dev.biologists.org/content/develop/113/2/399.full.pdf</a>.
</div>
<div id="ref-Horl2019" class="csl-entry">
Hörl, David, Fabio Rojas Rusak, Friedrich Preusser, Paul Tillberg, Nadine Randel, Raghav K Chhetri, Albert Cardona, et al. 2019. <span>“BigStitcher: Reconstructing High-Resolution Image Datasets of Cleared and Expanded Samples.”</span> <em>Nature Methods</em> 16: 870–74. <a href="https://doi.org/10.1038/s41592-019-0501-0">https://doi.org/10.1038/s41592-019-0501-0</a>.
</div>
<div id="ref-Howe2017" class="csl-entry">
Howe, Douglas G., Yvonne M. Bradford, Anne Eagle, David Fashena, Ken Frazer, Patrick Kalita, Prita Mani, et al. 2017. <span>“The Zebrafish Model Organism Database: New Support for Human Disease Models, Mutation Details, Gene Expression Phenotypes and Searching.”</span> <em>Nucleic Acids Research</em> 45 (January): D758–68. <a href="https://doi.org/10.1093/nar/gkw1116">https://doi.org/10.1093/nar/gkw1116</a>.
</div>
<div id="ref-Hu2020a" class="csl-entry">
Hu, Jian, Xiangjie Li, Kyle Coleman, Amelia Schroeder, David J Irwin, Edward B Lee, Russell T Shinohara, and Mingyao Li. 2020. <span>“Integrating Gene Expression, Spatial Location and Histology to Identify Spatial Domains and Spatially Variable Genes by Graph Convolutional Network.”</span> <em>bioRxiv</em>, January, 2020.11.30.405118. <a href="https://doi.org/10.1101/2020.11.30.405118">https://doi.org/10.1101/2020.11.30.405118</a>.
</div>
<div id="ref-Huber2018" class="csl-entry">
Huber, D., L. Voith von Voithenberg, and G. V. Kaigala. 2018. <span>“Fluorescence in Situ Hybridization (FISH): History, Limitations and What to Expect from Micro-Scale FISH?”</span> <em>Micro and Nano Engineering</em> 1 (November): 15–24. <a href="https://doi.org/10.1016/j.mne.2018.10.006">https://doi.org/10.1016/j.mne.2018.10.006</a>.
</div>
<div id="ref-Hunt-Newbury2007" class="csl-entry">
Hunt-Newbury, Rebecca, Ryan Viveiros, Robert Johnsen, Allan Mah, Dina Anastas, Lily Fang, Erin Halfnight, et al. 2007. <span>“High-Throughput in Vivo Analysis of Gene Expression in Caenorhabditis Elegans.”</span> Edited by John Sulston. <em>PLoS Biology</em> 5 (September): e237. <a href="https://doi.org/10.1371/journal.pbio.0050237">https://doi.org/10.1371/journal.pbio.0050237</a>.
</div>
<div id="ref-Hwang2020" class="csl-entry">
Hwang, William L, Karthik A Jagadeesh, Jimmy A Guo, Hannah I Hoffman, Payman Yadollahpour, Rahul Mohan, Eugene Drokhlyansky, et al. 2020. <span>“Single-Nucleus and Spatial Transcriptomics of Archival Pancreatic Cancer Reveals Multi-Compartment Reprogramming After Neoadjuvant Treatment.”</span> <em>bioRxiv</em>, August, 2020.08.25.267336. <a href="https://doi.org/10.1101/2020.08.25.267336">https://doi.org/10.1101/2020.08.25.267336</a>.
</div>
<div id="ref-Islam2011" class="csl-entry">
Islam, Saiful, Una Kjällquist, Annalena Moliner, Pawel Zajac, Jian-Bing Fan, Peter Lönnerberg, and Sten Linnarsson. 2011. <span>“Characterization of the Single-Cell Transcriptional Landscape by Highly Multiplex RNA-Seq.”</span> <em>Genome Research</em> 21 (July): 1160–67. <a href="https://doi.org/10.1101/gr.110882.110">https://doi.org/10.1101/gr.110882.110</a>.
</div>
<div id="ref-Jagalur2007" class="csl-entry">
Jagalur, Manjunatha, Chris Pal, Erik Learned-Miller, R Thomas Zoeller, and David Kulp. 2007. <span>“Analyzing in Situ Gene Expression in the Mouse Brain with Image Registration, Feature Extraction and Block Clustering.”</span> <em>BMC Bioinformatics</em> 8 (December): S5. <a href="https://doi.org/10.1186/1471-2105-8-S10-S5">https://doi.org/10.1186/1471-2105-8-S10-S5</a>.
</div>
<div id="ref-Janning1997" class="csl-entry">
Janning, Wilfried. 1997. <span>“FlyView, aDrosophilaimage Database, and otherDrosophiladatabases.”</span> <em>Seminars in Cell &amp; Developmental Biology</em> 8: 469–75. https://doi.org/<a href="https://doi.org/10.1006/scdb.1997.0172">https://doi.org/10.1006/scdb.1997.0172</a>.
</div>
<div id="ref-Jayaraman2001" class="csl-entry">
Jayaraman, Karthik, Sethuraman Panchanathan, and Sudhir Kumar. 2001. <span>“Classification and Indexing of Gene Expression Images.”</span>
</div>
<div id="ref-Jemt2016" class="csl-entry">
Jemt, Anders, Fredrik Salmén, Anna Lundmark, Annelie Mollbrink, José Fernández Navarro, Patrik L. Ståhl, Tülay Yucel-Lindberg, and Joakim Lundeberg. 2016. <span>“An Automated Approach to Prepare Tissue-Derived Spatially Barcoded RNA-Sequencing Libraries.”</span> <em>Scientific Reports</em>. <a href="https://doi.org/10.1038/srep37137">https://doi.org/10.1038/srep37137</a>.
</div>
<div id="ref-Jenett2012" class="csl-entry">
Jenett, Arnim, Gerald M. Rubin, Teri T. B. Ngo, David Shepherd, Christine Murphy, Heather Dionne, Barret D. Pfeiffer, et al. 2012. <span>“A GAL4-Driver Line Resource for Drosophila Neurobiology.”</span> <em>Cell Reports</em> 2 (October): 991–1001. <a href="https://doi.org/10.1016/j.celrep.2012.09.011">https://doi.org/10.1016/j.celrep.2012.09.011</a>.
</div>
<div id="ref-Jerby-Arnon2020" class="csl-entry">
Jerby-Arnon, Livnat, and Aviv Regev. 2020. <span>“Mapping Multicellular Programs from Single-Cell Profiles.”</span> <em>bioRxiv</em>, January, 2020.08.11.245472. <a href="https://doi.org/10.1101/2020.08.11.245472">https://doi.org/10.1101/2020.08.11.245472</a>.
</div>
<div id="ref-Ji2020" class="csl-entry">
Ji, Andrew L, Adam J Rubin, Kim Thrane, Sizun Jiang, David L Reynolds, Robin M Meyers, Margaret G Guo, et al. 2020. <span>“Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma.”</span> <em>Cell</em> 182: 497–514.e22. https://doi.org/<a href="https://doi.org/10.1016/j.cell.2020.05.039">https://doi.org/10.1016/j.cell.2020.05.039</a>.
</div>
<div id="ref-Ji2009" class="csl-entry">
Ji, Shuiwang, Ying Xin Li, Zhi Hua Zhou, Sudhir Kumar, and Jieping Ye. 2009. <span>“A Bag-of-Words Approach for Drosophila Gene Expression Pattern Annotation.”</span> <em>BMC Bioinformatics</em> 10 (April): 119. <a href="https://doi.org/10.1186/1471-2105-10-119">https://doi.org/10.1186/1471-2105-10-119</a>.
</div>
<div id="ref-Ji2008" class="csl-entry">
Ji, Shuiwang, Liang Sun, Rong Jin, Sudhir Kumar, and Jieping Ye. 2008. <span>“Automated Annotation of Drosophila Gene Expression Patterns Using a Controlled Vocabulary.”</span> <em>Bioinformatics</em> 24 (September): 1881–88. <a href="https://doi.org/10.1093/bioinformatics/btn347">https://doi.org/10.1093/bioinformatics/btn347</a>.
</div>
<div id="ref-Ji2009a" class="csl-entry">
Ji, Shuiwang, Lei Yuan, Ying-Xin Li, Zhi-Hua Zhou, Sudhir Kumar, and Jieping Ye. 2009. <span>“Drosophila Gene Expression Pattern Annotation Using Sparse Features and Term-Term Interactions.”</span> In <em>Proceedings of the 15th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining - KDD ’09</em>, 407. ACM Press. <a href="https://doi.org/10.1145/1557019.1557068">https://doi.org/10.1145/1557019.1557068</a>.
</div>
<div id="ref-Joglekar2020" class="csl-entry">
Joglekar, Anoushka, Andrey Prjibelski, Ahmed Mahfouz, Paul Collier, Susan Lin, Anna Katharina Schlusche, Jordan Marrocco, et al. 2020. <span>“Cell-Type, Single-Cell, and Spatial Signatures of Brain-Region Specific Splicing in Postnatal Development.”</span> <em>bioRxiv</em>, January, 2020.08.27.268730. <a href="https://doi.org/10.1101/2020.08.27.268730">https://doi.org/10.1101/2020.08.27.268730</a>.
</div>
<div id="ref-John1969" class="csl-entry">
John, H. A., M. L. Birnstiel, and K. W. Jones. 1969. <span>“RNA-DNA Hybrids at the Cytological Level.”</span> <em>Nature</em> 223: 582–87. <a href="https://doi.org/10.1038/223582a0">https://doi.org/10.1038/223582a0</a>.
</div>
<div id="ref-JHU" class="csl-entry">
<span>“Johns Hopkins Cell Imaging Core Facility.”</span> n.d. <a href="https://www.hopkinsmedicine.org/kimmel_cancer_center/research/shared_resources/cell_imaging.html">https://www.hopkinsmedicine.org/kimmel_cancer_center/research/shared_resources/cell_imaging.html</a>.
</div>
<div id="ref-Johnson2005" class="csl-entry">
Johnson, Alexander A. T., Julian M. Hibberd, Céline Gay, Pauline A. Essah, Jim Haseloff, Mark Tester, and Emmanuel Guiderdoni. 2005. <span>“Spatial Control of Transgene Expression in Rice (Oryza Sativa l.) Using the GAL4 Enhancer Trapping System.”</span> <em>The Plant Journal</em> 41 (February): 779–89. <a href="https://doi.org/10.1111/j.1365-313X.2005.02339.x">https://doi.org/10.1111/j.1365-313X.2005.02339.x</a>.
</div>
<div id="ref-Junker2014" class="csl-entry">
Junker, Jan Philipp, Emily S. Noël, Victor Guryev, Kevin A. Peterson, Gopi Shah, Jan Huisken, Andrew P. McMahon, Eugene Berezikov, Jeroen Bakkers, and Alexander Van Oudenaarden. 2014. <span>“Genome-Wide RNA Tomography in the Zebrafish Embryo.”</span> <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2014.09.038">https://doi.org/10.1016/j.cell.2014.09.038</a>.
</div>
<div id="ref-Kacharmina1999" class="csl-entry">
Kacharmina, Janet Estee, Peter B Crino, and James B T -Methods in Enzymology Eberwine. 1999. <span>“Preparation of cDNA from Single Cells and Subcellular Regions.”</span> <em>cDNA Preparation and Characterization</em> 303: 3–18. https://doi.org/<a href="https://doi.org/10.1016/S0076-6879(99)03003-7">https://doi.org/10.1016/S0076-6879(99)03003-7</a>.
</div>
<div id="ref-Kaewsapsak2017" class="csl-entry">
Kaewsapsak, Pornchai, David Michael Shechner, William Mallard, John L. Rinn, and Alice Y. Ting. 2017. <span>“Live-Cell Mapping of Organelle-Associated RNAs via Proximity Biotinylation Combined with Protein-RNA Crosslinking.”</span> <em>eLife</em> 6 (December). <a href="https://doi.org/10.7554/eLife.29224">https://doi.org/10.7554/eLife.29224</a>.
</div>
<div id="ref-Kamme2003" class="csl-entry">
Kamme, Fredrik, Ranelle Salunga, Jingxue Yu, Da-Thao Tran, Jessica Zhu, Lin Luo, Anton Bittner, et al. 2003. <span>“Single-Cell Microarray Analysis in Hippocampus CA1: Demonstration and Validation of Cellular Heterogeneity.”</span> <em>The Journal of Neuroscience</em> 23 (May): 3607 LP–3615. <a href="https://doi.org/10.1523/JNEUROSCI.23-09-03607.2003">https://doi.org/10.1523/JNEUROSCI.23-09-03607.2003</a>.
</div>
<div id="ref-Karaiskos2017" class="csl-entry">
Karaiskos, Nikos, Philipp Wahle, Jonathan Alles, Anastasiya Boltengagen, Salah Ayoub, Claudia Kipar, Christine Kocks, Nikolaus Rajewsky, and Robert P. Zinzen. 2017. <span>“The Drosophila Embryo at Single-Cell Transcriptome Resolution.”</span> <em>Science</em> 358 (October): 194–99. <a href="https://doi.org/10.1126/science.aan3235">https://doi.org/10.1126/science.aan3235</a>.
</div>
<div id="ref-Karali2010" class="csl-entry">
Karali, Marianthi, Ivana Peluso, Vincenzo A. Gennarino, Marchesa Bilio, Roberta Verde, Giampiero Lago, Pascal Dollé, and Sandro Banfi. 2010. <span>“miRNeye: A microRNA Expression Atlas of the Mouse Eye.”</span> <em>BMC Genomics</em> 11 (December): 715. <a href="https://doi.org/10.1186/1471-2164-11-715">https://doi.org/10.1186/1471-2164-11-715</a>.
</div>
<div id="ref-Kawakami2010" class="csl-entry">
Kawakami, Koichi, Gembu Abe, Tokuko Asada, Kazuhide Asakawa, Ryuichi Fukuda, Aki Ito, Pradeep Lal, et al. 2010. <span>“ZTrap: Zebrafish Gene Trap and Enhancer Trap Database.”</span> <em>BMC Developmental Biology</em> 10 (October): 105. <a href="https://doi.org/10.1186/1471-213X-10-105">https://doi.org/10.1186/1471-213X-10-105</a>.
</div>
<div id="ref-Kawashima2000" class="csl-entry">
Kawashima, T. 2000. <span>“MAGEST: MAboya Gene Expression Patterns and Sequence Tags.”</span> <em>Nucleic Acids Research</em> 28 (January): 133–35. <a href="https://doi.org/10.1093/nar/28.1.133">https://doi.org/10.1093/nar/28.1.133</a>.
</div>
<div id="ref-Ke2013" class="csl-entry">
Ke, Rongqin, Marco Mignardi, Alexandra Pacureanu, Jessica Svedlund, Johan Botling, Carolina Wählby, and Mats Nilsson. 2013. <span>“In Situ Sequencing for RNA Analysis in Preserved Tissue and Cells.”</span> <em>Nature Methods</em> 10 (September): 857–60. <a href="https://doi.org/10.1038/nmeth.2563">https://doi.org/10.1038/nmeth.2563</a>.
</div>
<div id="ref-Kim2020a" class="csl-entry">
Kim, Mean-Hwan, Cristina Radaelli, Elliot R Thomsen, Joseph T Mahoney, Brian Long, Michael J Taormina, Sara Kebede, et al. 2020. <span>“Molecular and Genetic Approaches for Assaying Human Cell Type Synaptic Connectivity.”</span> <em>bioRxiv</em>, January, 2020.10.16.343343. <a href="https://doi.org/10.1101/2020.10.16.343343">https://doi.org/10.1101/2020.10.16.343343</a>.
</div>
<div id="ref-Kishi2019" class="csl-entry">
Kishi, Jocelyn Y, Sylvain W Lapan, Brian J Beliveau, Emma R West, Allen Zhu, Hiroshi M Sasaki, Sinem K Saka, Yu Wang, Constance L Cepko, and Peng Yin. 2019. <span>“SABER Amplifies FISH: Enhanced Multiplexed Imaging of RNA and DNA in Cells and Tissues.”</span> <em>Nature Methods</em> 16: 533–44. <a href="https://doi.org/10.1038/s41592-019-0404-0">https://doi.org/10.1038/s41592-019-0404-0</a>.
</div>
<div id="ref-Kishi2018" class="csl-entry">
Kishi, Jocelyn Y, Thomas E Schaus, Nikhil Gopalkrishnan, Feng Xuan, and Peng Yin. 2018. <span>“Programmable Autonomous Synthesis of Single-Stranded DNA.”</span> <em>Nature Chemistry</em> 10: 155–64. <a href="https://doi.org/10.1038/nchem.2872">https://doi.org/10.1038/nchem.2872</a>.
</div>
<div id="ref-Kitahara2001" class="csl-entry">
Kitahara, Osamu, Yoichi Furukawa, Toshihiro Tanaka, Chikashi Kihara, Kenji Ono, Renpei Yanagawa, Marcelo E Nita, Toshihisa Takagi, Yusuke Nakamura, and Tatsuhiko Tsunoda. 2001. <span>“Alterations of Gene Expression During Colorectal Carcinogenesis Revealed by cDNA Microarrays After Laser-Capture Microdissection of Tumor Tissues and Normal Epithelia.”</span> <em>Cancer Research</em> 61 (May): 3544 LP–3549. <a href="http://cancerres.aacrjournals.org/content/61/9/3544.abstract">http://cancerres.aacrjournals.org/content/61/9/3544.abstract</a>.
</div>
<div id="ref-Kitamura2017" class="csl-entry">
Kitamura, S, T Tanahashi, E Aoyagi, T Nakagawa, K Okamoto, T Kimura, H Miyamoto, et al. 2017. <span>“Response Predictors of s-1, Cisplatin, and Docetaxel Combination Chemotherapy for Metastatic Gastric Cancer: Microarray Analysis of Whole Human Genes.”</span> <em>Oncology</em> 93: 127–35. <a href="https://doi.org/10.1159/000464329">https://doi.org/10.1159/000464329</a>.
</div>
<div id="ref-Klein2015" class="csl-entry">
Klein, Allon M., Linas Mazutis, Ilke Akartuna, Naren Tallapragada, Adrian Veres, Victor Li, Leonid Peshkin, David A. Weitz, and Marc W. Kirschner. 2015. <span>“Droplet Barcoding for Single-Cell Transcriptomics Applied to Embryonic Stem Cells.”</span> <em>Cell</em> 161 (May): 1187–1201. <a href="https://doi.org/10.1016/j.cell.2015.04.044">https://doi.org/10.1016/j.cell.2015.04.044</a>.
</div>
<div id="ref-Klein2002" class="csl-entry">
Klein, Christoph A, Stefan Seidl, Karina Petat-Dutter, Sonja Offner, Jochen B Geigl, Oleg Schmidt-Kittler, Nicole Wendler, et al. 2002. <span>“Combined Transcriptome and Genome Analysis of Single Micrometastatic Cells.”</span> <em>Nature Biotechnology</em> 20: 387–92. <a href="https://doi.org/10.1038/nbt0402-387">https://doi.org/10.1038/nbt0402-387</a>.
</div>
<div id="ref-Kleshchevnikov2020" class="csl-entry">
Kleshchevnikov, Vitalii, Artem Shmatko, Emma Dann, Alexander Aivazidis, Hamish W King, Tong Li, Artem Lomakin, et al. 2020. <span>“Comprehensive Mapping of Tissue Cell Architecture via Integrated Single Cell and Spatial Transcriptomics.”</span> <em>bioRxiv</em>, January, 2020.11.15.378125. <a href="https://doi.org/10.1101/2020.11.15.378125">https://doi.org/10.1101/2020.11.15.378125</a>.
</div>
<div id="ref-Kloosterman2006" class="csl-entry">
Kloosterman, Wigard P., Erno Wienholds, Ewart de Bruijn, Sakari Kauppinen, and Ronald H A Plasterk. 2006. <span>“In Situ Detection of miRNAs in Animal Embryos Using LNA-Modified Oligonucleotide Probes.”</span> <em>Nature Methods</em> 3 (January): 27–29. <a href="https://doi.org/10.1038/nmeth843">https://doi.org/10.1038/nmeth843</a>.
</div>
<div id="ref-Ko1990" class="csl-entry">
Ko, M S H, S B H Ko, N Takahashi, K Nishiguchi, and K Abe. 1990. <span>“Unbiased Amplification of a Highly Complex Mixture of DNA Fragments by <span>‘Lone Linker’</span>-Tagged PCR.”</span> <em>Nucleic Acids Research</em> 18 (July): 4293. <a href="https://doi.org/10.1093/nar/18.14.4293">https://doi.org/10.1093/nar/18.14.4293</a>.
</div>
<div id="ref-Ko2013" class="csl-entry">
Ko, Younhee, Seth A. Ament, James A. Eddy, Juan Caballero, John C. Earls, Leroy Hood, and Nathan D. Price. 2013. <span>“Cell Type-Specific Genes Show Striking and Distinct Patterns of Spatial Expression in the Mouse Brain.”</span> <em>Proceedings of the National Academy of Sciences</em> 110 (February): 3095–3100. <a href="https://doi.org/10.1073/pnas.1222897110">https://doi.org/10.1073/pnas.1222897110</a>.
</div>
<div id="ref-Kohman2020" class="csl-entry">
Kohman, Richie E, and George M Church. 2020. <span>“Fluorescent in Situ Sequencing of DNA Barcoded Antibodies.”</span> <em>bioRxiv</em> 20 (April): 2020.04.27.060624. <a href="https://doi.org/10.1101/2020.04.27.060624">https://doi.org/10.1101/2020.04.27.060624</a>.
</div>
<div id="ref-Kopczynski1998" class="csl-entry">
Kopczynski, Casey C., Jasprina N. Noordermeer, Thomas L. Serano, W.-Y. Chen, John D. Pendleton, Suzanna Lewis, Corey S. Goodman, and Gerald M. Rubin. 1998. <span>“A High Throughput Screen to Identify Secreted and Transmembrane Proteins Involved in Drosophila Embryogenesis.”</span> <em>Proceedings of the National Academy of Sciences</em> 95 (August): 9973–78. <a href="https://doi.org/10.1073/pnas.95.17.9973">https://doi.org/10.1073/pnas.95.17.9973</a>.
</div>
<div id="ref-Korsunsky2019" class="csl-entry">
Korsunsky, Ilya, Nghia Millard, Jean Fan, Kamil Slowikowski, Fan Zhang, Kevin Wei, Yuriy Baglaenko, Michael Brenner, Po-ru Loh, and Soumya Raychaudhuri. 2019. <span>“Fast, Sensitive and Accurate Integration of Single-Cell Data with Harmony.”</span> <em>Nature Methods</em> 16: 1289–96. <a href="https://doi.org/10.1038/s41592-019-0619-0">https://doi.org/10.1038/s41592-019-0619-0</a>.
</div>
<div id="ref-Koster2019" class="csl-entry">
Köster, Johannes, Myles Brown, and X. Shirley Liu. 2019. <span>“A Bayesian Model for Single Cell Transcript Expression Analysis on MERFISH Data.”</span> <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/bty718">https://doi.org/10.1093/bioinformatics/bty718</a>.
</div>
<div id="ref-Kudo2012" class="csl-entry">
Kudo, Lili C., Nancy Vi, Zhongcai Ma, Tony Fields, Nuraly K. Avliyakulov, Michael J. Haykinson, Anatol Bragin, and Stanislav L. Karsten. 2012. <span>“Novel Cell and Tissue Acquisition System (CTAS): Microdissection of Live and Frozen Brain Tissues.”</span> Edited by Rafael Aldabe. <em>PLoS ONE</em> 7 (July): e41564. <a href="https://doi.org/10.1371/journal.pone.0041564">https://doi.org/10.1371/journal.pone.0041564</a>.
</div>
<div id="ref-Kueckelhaus2020" class="csl-entry">
Kueckelhaus, Jan, Jasmin von Ehr, Vidhya M Ravi, Paulina Will, Kevin Joseph, Juergen Beck, Ulrich G Hofmann, Daniel Delev, Oliver Schnell, and Dieter Henrik Heiland. 2020. <span>“Inferring Spatially Transient Gene Expression Pattern from Spatial Transcriptomic Studies.”</span> <em>bioRxiv</em>, January, 2020.10.20.346544. <a href="https://doi.org/10.1101/2020.10.20.346544">https://doi.org/10.1101/2020.10.20.346544</a>.
</div>
<div id="ref-Kumar2002" class="csl-entry">
Kumar, Sudhir, Karthik Jayaraman, Sethuraman Panchanathan, Rajalakshmi Gurunathan, Ana Marti-Subirana, and Stuart J Newfeld. 2002. <span>“BEST: A Novel Computational Approach for Comparing Gene Expression Patterns from Early Stages of &amp;Lt;em&amp;gt;drosophila Melanogaster&amp;lt;/Em&amp;gt; Development.”</span> <em>Genetics</em> 162 (December): 2037 LP–2047. <a href="http://www.genetics.org/content/162/4/2037.abstract">http://www.genetics.org/content/162/4/2037.abstract</a>.
</div>
<div id="ref-Kumar2017" class="csl-entry">
Kumar, Sudhir, Charlotte Konikoff, Maxwell Sanderford, Li Liu, Stuart Newfeld, Jieping Ye, and Rob J. Kulathinal. 2017. <span>“FlyExpress 7: An Integrated Discovery Platform to Study Coexpressed Genes Using in Situ Hybridization Images in Drosophila.”</span> <em>G3; Genes|Genomes|Genetics</em> 7 (August): 2791–97. <a href="https://doi.org/10.1534/g3.117.040345">https://doi.org/10.1534/g3.117.040345</a>.
</div>
<div id="ref-Kvon2014" class="csl-entry">
Kvon, Evgeny Z., Tomas Kazmar, Gerald Stampfel, J. Omar Yáñez-Cuna, Michaela Pagani, Katharina Schernhuber, Barry J. Dickson, and Alexander Stark. 2014. <span>“Genome-Scale Functional Characterization of Drosophila Developmental Enhancers in Vivo.”</span> <em>Nature</em> 512 (August): 91–95. <a href="https://doi.org/10.1038/nature13395">https://doi.org/10.1038/nature13395</a>.
</div>
<div id="ref-Lacraz2017" class="csl-entry">
Lacraz, Grégory P. A., Jan Philipp Junker, Monika M. Gladka, Bas Molenaar, Koen T. Scholman, Marta Vigil-Garcia, Danielle Versteeg, et al. 2017. <span>“Tomo-Seq Identifies SOX9 as a Key Regulator of Cardiac Fibrosis During Ischemic Injury.”</span> <em>Circulation</em> 136 (October): 1396–1409. <a href="https://doi.org/10.1161/CIRCULATIONAHA.117.027832">https://doi.org/10.1161/CIRCULATIONAHA.117.027832</a>.
</div>
<div id="ref-Lake2021" class="csl-entry">
Lake, Blue B, Rajasree Menon, Seth Winfree, Qiwen Hu, Ricardo Melo Ferreira, Kian Kalhor, Daria Barwinska, et al. 2021. <span>“An Atlas of Healthy and Injured Cell States and Niches in the Human Kidney.”</span> <em>bioRxiv</em>, January, 2021.07.28.454201. <a href="https://doi.org/10.1101/2021.07.28.454201">https://doi.org/10.1101/2021.07.28.454201</a>.
</div>
<div id="ref-Landegren1988" class="csl-entry">
Landegren, U, R Kaiser, J Sanders, and L Hood. 1988. <span>“A Ligase-Mediated Gene Detection Technique.”</span> <em>Science</em> 241 (August): 1077 LP–1080. <a href="https://doi.org/10.1126/science.3413476">https://doi.org/10.1126/science.3413476</a>.
</div>
<div id="ref-Lander2001" class="csl-entry">
Lander, Eric S., Lauren M. Linton, Bruce Birren, Chad Nusbaum, Michael C. Zody, Jennifer Baldwin, Keri Devon, et al. 2001. <span>“Initial Sequencing and Analysis of the Human Genome.”</span> <em>Nature</em> 409 (February): 860–921. <a href="https://doi.org/10.1038/35057062">https://doi.org/10.1038/35057062</a>.
</div>
<div id="ref-Langer-Safer1982" class="csl-entry">
Langer-Safer, P. R., M. Levine, and D. C. Ward. 1982. <span>“Immunological Method for Mapping Genes on Drosophila Polytene Chromosomes.”</span> <em>Proceedings of the National Academy of Sciences</em> 79 (July): 4381–85. <a href="https://doi.org/10.1073/pnas.79.14.4381">https://doi.org/10.1073/pnas.79.14.4381</a>.
</div>
<div id="ref-Langseth2021" class="csl-entry">
Langseth, Christoffer Mattsson, Daniel Gyllborg, Jeremy A Miller, Jennie L Close, Brian Long, Ed S Lein, Markus M Hilscher, and Mats Nilsson. 2021. <span>“Comprehensive in Situ Mapping of Human Cortical Transcriptomic Cell Types.”</span> <em>Communications Biology</em> 4: 998. <a href="https://doi.org/10.1038/s42003-021-02517-z">https://doi.org/10.1038/s42003-021-02517-z</a>.
</div>
<div id="ref-Larsson2010" class="csl-entry">
Larsson, Chatarina, Ida Grundberg, Ola Söderberg, and Mats Nilsson. 2010. <span>“In Situ Detection and Genotyping of Individual mRNA Molecules.”</span> <em>Nature Methods</em> 7 (May): 395–97. <a href="https://doi.org/10.1038/nmeth.1448">https://doi.org/10.1038/nmeth.1448</a>.
</div>
<div id="ref-Laskow2017" class="csl-entry">
Laskow, Sarah. 2017. <span>“Resurrecting the Forgotten Bike Highways of 1930s Britain.”</span>
</div>
<div id="ref-Lebrigand2020" class="csl-entry">
Lebrigand, Kevin, Joseph Bergenstråhle, Kim Thrane, Annelie Mollbrink, Pascal Barbry, Rainer Waldmann, and Joakim Lundeberg. 2020. <span>“The Spatial Landscape of Gene Expression Isoforms in Tissue Sections.”</span> <em>bioRxiv</em>, August, 2020.08.24.252296. <a href="https://doi.org/10.1101/2020.08.24.252296">https://doi.org/10.1101/2020.08.24.252296</a>.
</div>
<div id="ref-Lecuyer2007" class="csl-entry">
Lécuyer, Eric, Hideki Yoshida, Neela Parthasarathy, Christina Alm, Tomas Babak, Tanja Cerovina, Timothy R. Hughes, Pavel Tomancak, and Henry M. Krause. 2007. <span>“Global Analysis of mRNA Localization Reveals a Prominent Role in Organizing Cellular Architecture and Function.”</span> <em>Cell</em> 131 (October): 174–87. <a href="https://doi.org/10.1016/j.cell.2007.08.003">https://doi.org/10.1016/j.cell.2007.08.003</a>.
</div>
<div id="ref-Lee2020" class="csl-entry">
Lee, Hower, Sergio Marco Salas, Daniel Gyllborg, and Mats Nilsson. 2020. <span>“Direct RNA Targeted Transcriptomic Profiling in Tissue Using Hybridization-Based RNA in Situ Sequencing (HybRISS).”</span> <em>bioRxiv</em>, January, 2020.12.02.408781. <a href="https://doi.org/10.1101/2020.12.02.408781">https://doi.org/10.1101/2020.12.02.408781</a>.
</div>
<div id="ref-Lee2015" class="csl-entry">
Lee, Je Hyuk, Evan R. Daugharthy, Jonathan Scheiman, Reza Kalhor, Thomas C. Ferrante, Richard Terry, Brian M. Turczyk, et al. 2015. <span>“Fluorescent in Situ Sequencing (FISSEQ) of RNA for Gene Expression Profiling in Intact Cells and Tissues.”</span> <em>Nature Protocols</em> 10 (March): 442–58. <a href="https://doi.org/10.1038/nprot.2014.191">https://doi.org/10.1038/nprot.2014.191</a>.
</div>
<div id="ref-Lee2014" class="csl-entry">
Lee, Je Hyuk, Evan R Daugharthy, Jonathan Scheiman, Reza Kalhor, Joyce L Yang, Thomas C Ferrante, Richard Terry, et al. 2014. <span>“Highly Multiplexed Subcellular RNA Sequencing in Situ.”</span> <em>Science</em> 343 (March): 1360 LP–1363. <a href="https://doi.org/10.1126/science.1250212">https://doi.org/10.1126/science.1250212</a>.
</div>
<div id="ref-Youjin2021" class="csl-entry">
Lee, Youjin, Derek Bogdanoff, Yutong Wang, George C. Hartoularos, Jonathan M. Woo, Mowery Cody T., Nisonoff Hunter M., et al. 2021. <span>“XYZeq: Spatially Resolved Single-Cell RNA Sequencing Reveals Expression Heterogeneity in the Tumor Microenvironment.”</span> <em>Science Advances</em> 7 (September): eabg4755. <a href="https://doi.org/10.1126/sciadv.abg4755">https://doi.org/10.1126/sciadv.abg4755</a>.
</div>
<div id="ref-Leighton2001" class="csl-entry">
Leighton, Philip A., Kevin J. Mitchell, Lisa V. Goodrich, Xiaowei Lu, Kathy Pinson, Paul Scherz, William C. Skarnes, and Marc Tessier-Lavigne. 2001. <span>“Defining Brain Wiring Patterns and Mechanisms Through Gene Trapping in Mice.”</span> <em>Nature</em> 410 (March): 174–79. <a href="https://doi.org/10.1038/35065539">https://doi.org/10.1038/35065539</a>.
</div>
<div id="ref-Lein2004" class="csl-entry">
Lein, Ed S. 2004. <span>“Defining a Molecular Atlas of the Hippocampus Using DNA Microarrays and High-Throughput in Situ Hybridization.”</span> <em>Journal of Neuroscience</em> 24 (April): 3879–89. <a href="https://doi.org/10.1523/JNEUROSCI.4710-03.2004">https://doi.org/10.1523/JNEUROSCI.4710-03.2004</a>.
</div>
<div id="ref-Lein2007" class="csl-entry">
Lein, Ed S., Michael J. Hawrylycz, Nancy Ao, Mikael Ayres, Amy Bensinger, Amy Bernard, Andrew F. Boe, et al. 2007. <span>“Genome-Wide Atlas of Gene Expression in the Adult Mouse Brain.”</span> <em>Nature</em> 445 (January): 168–76. <a href="https://doi.org/10.1038/nature05453">https://doi.org/10.1038/nature05453</a>.
</div>
<div id="ref-Lein2017" class="csl-entry">
Lein, Ed, Lars E. Borm, and Sten Linnarsson. 2017. <span>“The Promise of Spatial Transcriptomics for Neuroscience in the Era of Molecular Cell Typing.”</span> <em>Science</em>. <a href="https://doi.org/10.1126/science.aan6827">https://doi.org/10.1126/science.aan6827</a>.
</div>
<div id="ref-Levsky2002" class="csl-entry">
Levsky, Jeffrey M. 2002. <span>“Single-Cell Gene Expression Profiling.”</span> <em>Science</em> 297 (August): 836–40. <a href="https://doi.org/10.1126/science.1072241">https://doi.org/10.1126/science.1072241</a>.
</div>
<div id="ref-Levy-Jurgenson2020" class="csl-entry">
Levy-Jurgenson, Alona, Xavier Tekpli, Vessela N Kristensen, and Zohar Yakhini. 2020. <span>“Spatial Transcriptomics Inferred from Pathology Whole-Slide Images Links Tumor Heterogeneity to Survival in Breast and Lung Cancer.”</span> <em>Scientific Reports</em> 10: 18802. <a href="https://doi.org/10.1038/s41598-020-75708-z">https://doi.org/10.1038/s41598-020-75708-z</a>.
</div>
<div id="ref-Li2022" class="csl-entry">
Li, Bin, Wen Zhang, Chuang Guo, Hao Xu, Longfei Li, Minghao Fang, Yinlei Hu, et al. 2022. <span>“Benchmarking Spatial and Single-Cell Transcriptomics Integration Methods for Transcript Distribution Prediction and Cell Type Deconvolution.”</span> <em>Nature Methods 2022 19:6</em> 19 (May): 662–70. <a href="https://doi.org/10.1038/s41592-022-01480-9">https://doi.org/10.1038/s41592-022-01480-9</a>.
</div>
<div id="ref-Li2023" class="csl-entry">
Li, Haoyang, Juexiao Zhou, Zhongxiao Li, Siyuan Chen, Xingyu Liao, Bin Zhang, Ruochi Zhang, Yu Wang, Shiwei Sun, and Xin Gao. 2023. <span>“A Comprehensive Benchmarking with Practical Guidelines for Cellular Deconvolution of Spatial Transcriptomics.”</span> <em>Nature Communications 2023 14:1</em> 14 (March): 1–10. <a href="https://doi.org/10.1038/s41467-023-37168-7">https://doi.org/10.1038/s41467-023-37168-7</a>.
</div>
<div id="ref-Li2016" class="csl-entry">
Li, Junxiang, Haofei Luo, Rui Wang, Jidong Lang, Siyu Zhu, Zhenming Zhang, Jianhuo Fang, et al. 2016. <span>“Systematic Reconstruction of Molecular Cascades Regulating GP Development Using Single-Cell RNA-Seq.”</span> <em>Cell Reports</em> 15 (May): 1467–80. <a href="https://doi.org/10.1016/j.celrep.2016.04.043">https://doi.org/10.1016/j.celrep.2016.04.043</a>.
</div>
<div id="ref-Li2021" class="csl-entry">
Li, Qiang, Zuwan Lin, Ren Liu, Xin Tang, Jiahao Huang, Yichun He, Haowen Zhou, et al. 2021. <span>“&amp;Lt;em&amp;gt;in Situ&amp;lt;/Em&amp;gt; Electro-Sequencing in Three-Dimensional Tissues.”</span> <em>bioRxiv</em>, January, 2021.04.22.440941. <a href="https://doi.org/10.1101/2021.04.22.440941">https://doi.org/10.1101/2021.04.22.440941</a>.
</div>
<div id="ref-Li2021a" class="csl-entry">
Li, Qiwei, Minzhe Zhang, Yang Xie, and Guanghua Xiao. 2021. <span>“Bayesian Modeling of Spatial Molecular Profiling Data via Gaussian Process.”</span> <em>Bioinformatics</em>, June. <a href="https://doi.org/10.1093/bioinformatics/btab455">https://doi.org/10.1093/bioinformatics/btab455</a>.
</div>
<div id="ref-Li2022a" class="csl-entry">
Li, Yijun, Stefan Stanojevic, Bing He, Zheng Jing, Qianhui Huang, Jian Kang, and Lana X. Garmire. 2022. <span>“Benchmarking Computational Integration Methods for Spatial Transcriptomics Data.”</span> <em>bioRxiv</em>, January, 2021.08.27.457741. <a href="https://doi.org/10.1101/2021.08.27.457741">https://doi.org/10.1101/2021.08.27.457741</a>.
</div>
<div id="ref-Li2009" class="csl-entry">
Li, Ying-Xin, Shuiwang Ji, Sudhir Kumar, Jieping Ye, and Zhi-Hua Zhou. 2009. <span>“Drosophila Gene Expression Pattern Annotation Through Multi-Instance Multi-Label Learning.”</span> <em>IJCAI : Proceedings of the Conference</em> 2009 (January): 1445–50. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20824158%20http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC2932460">http://www.ncbi.nlm.nih.gov/pubmed/20824158 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC2932460</a>.
</div>
<div id="ref-Li2017" class="csl-entry">
Li, Yujie, Hanbo Chen, Xi Jiang, Xiang Li, Jinglei Lv, Hanchuan Peng, Joe Z. Tsien, and Tianming Liu. 2017. <span>“Discover Mouse Gene Coexpression Landscapes Using Dictionary Learning and Sparse Coding.”</span> <em>Brain Structure and Function</em> 222 (December): 4253–70. <a href="https://doi.org/10.1007/s00429-017-1460-9">https://doi.org/10.1007/s00429-017-1460-9</a>.
</div>
<div id="ref-Li2020" class="csl-entry">
Li, Zhuliu, Tianci Song, Jeongsik Yong, and Rui Kuang. 2020. <span>“Imputation of Spatially-Resolved Transcriptomes by Graph-Regularized Tensor Completion.”</span> <em>bioRxiv</em>, January, 2020.08.05.237560. <a href="https://doi.org/10.1101/2020.08.05.237560">https://doi.org/10.1101/2020.08.05.237560</a>.
</div>
<div id="ref-Liang1992" class="csl-entry">
Liang, P, and A B Pardee. 1992. <span>“Differential Display of Eukaryotic Messenger RNA by Means of the Polymerase Chain Reaction.”</span> <em>Science</em> 257 (August): 967 LP–971. <a href="https://doi.org/10.1126/science.1354393">https://doi.org/10.1126/science.1354393</a>.
</div>
<div id="ref-Liao2020" class="csl-entry">
Liao, Jie, Xiaoyan Lu, Xin Shao, Ling Zhu, and Xiaohui Fan. 2020. <span>“Uncovering an Organ’s Molecular Architecture at Single-Cell Resolution by Spatially Resolved Transcriptomics.”</span> <em>Trends in Biotechnology</em>, June. <a href="https://doi.org/10.1016/j.tibtech.2020.05.006">https://doi.org/10.1016/j.tibtech.2020.05.006</a>.
</div>
<div id="ref-Lignell2017" class="csl-entry">
Lignell, Antti, Laura Kerosuo, Sebastian J. Streichan, Long Cai, and Marianne E. Bronner. 2017. <span>“Identification of a Neural Crest Stem Cell Niche by Spatial Genomic Analysis.”</span> <em>Nature Communications</em> 8 (December): 1830. <a href="https://doi.org/10.1038/s41467-017-01561-w">https://doi.org/10.1038/s41467-017-01561-w</a>.
</div>
<div id="ref-Lis1983" class="csl-entry">
Lis, John T., Jeffrey A. Simon, and Claudia A. Sutton. 1983. <span>“New Heat Shock Puffs and β=galactosidase Activity Resulting from Transformation of Drosophila with an Hsp70-lacZ Hybrid Gene.”</span> <em>Cell</em> 35 (December): 403–10. <a href="https://doi.org/10.1016/0092-8674(83)90173-3">https://doi.org/10.1016/0092-8674(83)90173-3</a>.
</div>
<div id="ref-Liscovitch2013a" class="csl-entry">
Liscovitch, Noa, Uri Shalit, and Gal Chechik. 2013. <span>“FuncISH: Learning a Functional Representation of Neural ISH Images.”</span> <em>Bioinformatics</em> 29 (July): i36–43. <a href="https://doi.org/10.1093/bioinformatics/btt207">https://doi.org/10.1093/bioinformatics/btt207</a>.
</div>
<div id="ref-Littman2020" class="csl-entry">
Littman, Russell, Zachary Hemminger, Robert Foreman, Douglas Arneson, Guanglin Zhang, Fernando Gómez-Pinilla, Xia Yang, and Roy Wollman. 2020. <span>“JSTA: Joint Cell Segmentation and Cell Type Annotation for Spatial Transcriptomics.”</span> <em>bioRxiv</em>, January, 2020.09.18.304147. <a href="https://doi.org/10.1101/2020.09.18.304147">https://doi.org/10.1101/2020.09.18.304147</a>.
</div>
<div id="ref-Liu2012" class="csl-entry">
Liu, Lin, Yinhu Li, Siliang Li, Ni Hu, Yimin He, Ray Pong, Danni Lin, Lihua Lu, and Maggie Law. 2012. <span>“Comparison of Next-Generation Sequencing Systems.”</span> Edited by P J Oefner. <em>Journal of Biomedicine and Biotechnology</em> 2012: 251364. <a href="https://doi.org/10.1155/2012/251364">https://doi.org/10.1155/2012/251364</a>.
</div>
<div id="ref-Liu2020" class="csl-entry">
Liu, Miao, Yanfang Lu, Bing Yang, Yanbo Chen, Jonathan S. D. Radda, Mengwei Hu, Samuel G. Katz, and Siyuan Wang. 2020. <span>“Multiplexed Imaging of Nucleome Architectures in Single Cells of Mammalian Tissue.”</span> <em>Nature Communications</em> 11 (December): 1–14. <a href="https://doi.org/10.1038/s41467-020-16732-5">https://doi.org/10.1038/s41467-020-16732-5</a>.
</div>
<div id="ref-Liu2021" class="csl-entry">
Liu, Songlei, Sukanya Punthambaker, Eswar P R Iyer, Thomas Ferrante, Daniel Goodwin, Daniel Fürth, Andrew C Pawlowski, et al. 2021. <span>“Barcoded Oligonucleotides Ligated on RNA Amplified for Multiplexed and Parallel in Situ Analyses.”</span> <em>Nucleic Acids Research</em> 49 (June): e58–58. <a href="https://doi.org/10.1093/nar/gkab120">https://doi.org/10.1093/nar/gkab120</a>.
</div>
<div id="ref-Liu2020b" class="csl-entry">
Liu, Yang, Archibald Enninful, Yanxiang Deng, and Rong Fan. 2020. <span>“Spatial Transcriptome Sequencing of FFPE Tissues at Cellular Level.”</span> <em>bioRxiv</em>, January, 2020.10.13.338475. <a href="https://doi.org/10.1101/2020.10.13.338475">https://doi.org/10.1101/2020.10.13.338475</a>.
</div>
<div id="ref-Liu2020a" class="csl-entry">
Liu, Yang, Mingyu Yang, Yanxiang Deng, Graham Su, Archibald Enninful, Cindy C Guo, Toma Tebaldi, et al. 2020. <span>“High-Spatial-Resolution Multi-Omics Sequencing via Deterministic Barcoding in Tissue.”</span> <em>Cell</em> 183: 1665–1681.e18. https://doi.org/<a href="https://doi.org/10.1016/j.cell.2020.10.026">https://doi.org/10.1016/j.cell.2020.10.026</a>.
</div>
<div id="ref-Liu2022" class="csl-entry">
Liu, Zhaoyang, Dongqing Sun, and Chenfei Wang. 2022. <span>“Evaluation of Cell-Cell Interaction Methods by Integrating Single-Cell RNA Sequencing Data with Spatial Information.”</span> <em>Genome Biology</em> 23 (December): 1–38. <a href="https://doi.org/10.1186/S13059-022-02783-Y/FIGURES/9">https://doi.org/10.1186/S13059-022-02783-Y/FIGURES/9</a>.
</div>
<div id="ref-Liu2007" class="csl-entry">
Liu, Zheng, S. Frank Yan, John R. Walker, Theresa A. Zwingman, Tao Jiang, Jing Li, and Yingyao Zhou. 2007. <span>“Study of Gene Function Based on Spatial Co-Expression in a High-Resolution Mouse Brain Atlas.”</span> <em>BMC Systems Biology</em> 1 (April): 19. <a href="https://doi.org/10.1186/1752-0509-1-19">https://doi.org/10.1186/1752-0509-1-19</a>.
</div>
<div id="ref-Lizardi1998" class="csl-entry">
Lizardi, Paul M., Xiaohua Huang, Zhengrong Zhu, Patricia Bray-Ward, David C. Thomas, and David C. Ward. 1998. <span>“Mutation Detection and Single-Molecule Counting Using Isothermal Rolling-Circle Amplification.”</span> <em>Nature Genetics</em> 19 (July): 225–32. <a href="https://doi.org/10.1038/898">https://doi.org/10.1038/898</a>.
</div>
<div id="ref-Ljungberg2019" class="csl-entry">
Ljungberg, M Cecilia, Mayce Sadi, Yunguan Wang, Bruce J Aronow, Yan Xu, Rong J Kao, Ying Liu, et al. 2019. <span>“Spatial Distribution of Marker Gene Activity in the Mouse Lung During Alveolarization.”</span> <em>Data in Brief</em> 22: 365–72. https://doi.org/<a href="https://doi.org/10.1016/j.dib.2018.10.150">https://doi.org/10.1016/j.dib.2018.10.150</a>.
</div>
<div id="ref-Lohoff2021" class="csl-entry">
Lohoff, T., S. Ghazanfar, A. Missarova, N. Koulena, N. Pierson, J. A. Griffiths, E. S. Bardot, et al. 2021. <span>“Integration of Spatial and Single-Cell Transcriptomic Data Elucidates Mouse Organogenesis.”</span> <em>Nature Biotechnology 2021 40:1</em> 40 (September): 74–85. <a href="https://doi.org/10.1038/s41587-021-01006-2">https://doi.org/10.1038/s41587-021-01006-2</a>.
</div>
<div id="ref-Long2021" class="csl-entry">
Long, W, T Li, Y Yang, and H -B. Shen. 2021. <span>“FlyIT: Drosophila Embryogenesis Image Annotation Based on Image Tiling and Convolutional Neural Networks.”</span> <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em> 18: 194–204. <a href="https://doi.org/10.1109/TCBB.2019.2935723">https://doi.org/10.1109/TCBB.2019.2935723</a>.
</div>
<div id="ref-Lopez2021" class="csl-entry">
Lopez, Romain, Baoguo Li, Hadas Keren-Shaul, Pierre Boyeau, Merav Kedmi, David Pilzer, Adam Jelinski, et al. 2021. <span>“Multi-Resolution Deconvolution of Spatial Transcriptomics Data Reveals Continuous Patterns of Inflammation.”</span> <em>bioRxiv</em>, January, 2021.05.10.443517. <a href="https://doi.org/10.1101/2021.05.10.443517">https://doi.org/10.1101/2021.05.10.443517</a>.
</div>
<div id="ref-Lopez2019" class="csl-entry">
Lopez, Romain, Achille Nazaret, Maxime Langevin, Jules Samaran, Jeffrey Regier, Michael I. Jordan, and Nir Yosef. 2019. <span>“A Joint Model of Unpaired Data from scRNA-Seq and Spatial Transcriptomics for Imputing Missing Gene Expression Measurements,”</span> May. <a href="http://arxiv.org/abs/1905.02269">http://arxiv.org/abs/1905.02269</a>.
</div>
<div id="ref-Lovatt2014" class="csl-entry">
Lovatt, Ditte, Brittani K. Ruble, Jaehee Lee, Hannah Dueck, Tae Kyung Kim, Stephen Fisher, Chantal Francis, et al. 2014. <span>“Transcriptome in Vivo Analysis (TIVA) of Spatially Defined Single Cells in Live Tissue.”</span> <em>Nature Methods</em> 11 (February): 190–96. <a href="https://doi.org/10.1038/nmeth.2804">https://doi.org/10.1038/nmeth.2804</a>.
</div>
<div id="ref-Lovell2020" class="csl-entry">
Lovell, Peter V., Morgan Wirthlin, Taylor Kaser, Alexa A. Buckner, Julia B. Carleton, Brian R. Snider, Anne K. McHugh, Alexander Tolpygo, Partha P. Mitra, and Claudio V. Mello. 2020. <span>“ZEBrA: Zebra Finch Expression Brain Atlas—a Resource for Comparative Molecular Neuroanatomy and Brain Evolution Studies.”</span> <em>Journal of Comparative Neurology</em> 528 (August): 2099–2131. <a href="https://doi.org/10.1002/cne.24879">https://doi.org/10.1002/cne.24879</a>.
</div>
<div id="ref-Lowe2004" class="csl-entry">
Lowe, David G. 2004. <span>“Distinctive Image Features from Scale-Invariant Keypoints.”</span> <em>International Journal of Computer Vision</em> 60 (November): 91–110. <a href="https://doi.org/10.1023/B:VISI.0000029664.99615.94">https://doi.org/10.1023/B:VISI.0000029664.99615.94</a>.
</div>
<div id="ref-Lu2021" class="csl-entry">
Lu, Yanfang, Miao Liu, Jennifer Yang, Sherman M Weissman, Xinghua Pan, Samuel G Katz, and Siyuan Wang. 2021. <span>“Spatial Transcriptome Profiling by MERFISH Reveals Fetal Liver Hematopoietic Stem Cell Niche Architecture.”</span> <em>Cell Discovery</em> 7: 47. <a href="https://doi.org/10.1038/s41421-021-00266-1">https://doi.org/10.1038/s41421-021-00266-1</a>.
</div>
<div id="ref-Lubeck2012" class="csl-entry">
Lubeck, Eric, and Long Cai. 2012. <span>“Single-Cell Systems Biology by Super-Resolution Imaging and Combinatorial Labeling.”</span> <em>Nature Methods 2012 9:7</em> 9 (June): 743–48. <a href="https://doi.org/10.1038/nmeth.2069">https://doi.org/10.1038/nmeth.2069</a>.
</div>
<div id="ref-Lubeck2014" class="csl-entry">
Lubeck, Eric, Ahmet F. Coskun, Timur Zhiyentayev, Mubhij Ahmad, and Long Cai. 2014. <span>“Single-Cell in Situ RNA Profiling by Sequential Hybridization.”</span> <em>Nature Methods</em>. Nature Publishing Group. <a href="https://doi.org/10.1038/nmeth.2892">https://doi.org/10.1038/nmeth.2892</a>.
</div>
<div id="ref-Lukan2020" class="csl-entry">
Lukan, Tjaša, Maruša Pompe‐Novak, Špela Baebler, Magda Tušek‐Žnidarič, Aleš Kladnik, Maja Križnik, Andrej Blejec, et al. 2020. <span>“Precision Transcriptomics of Viral Foci Reveals the Spatial Regulation of Immune‐signaling Genes and Identifies RBOHD as an Important Player in the Incompatible Interaction Between Potato Virus y and Potato.”</span> <em>The Plant Journal</em> 104 (November): 645–61. <a href="https://doi.org/10.1111/tpj.14953">https://doi.org/10.1111/tpj.14953</a>.
</div>
<div id="ref-Lundberg2019" class="csl-entry">
Lundberg, Emma, and Georg H. H. Borner. 2019. <span>“Spatial Proteomics: A Powerful Discovery Tool for Cell Biology.”</span> <em>Nature Reviews Molecular Cell Biology</em> 20 (May): 285–302. <a href="https://doi.org/10.1038/s41580-018-0094-y">https://doi.org/10.1038/s41580-018-0094-y</a>.
</div>
<div id="ref-Lundmark2018" class="csl-entry">
Lundmark, Anna, Natalija Gerasimcik, Tove Båge, Anders Jemt, Annelie Mollbrink, Fredrik Salmén, Joakim Lundeberg, and Tülay Yucel-Lindberg. 2018. <span>“Gene Expression Profiling of Periodontitis-Affected Gingival Tissue by Spatial Transcriptomics.”</span> <em>Scientific Reports</em> 8: 9370. <a href="https://doi.org/10.1038/s41598-018-27627-3">https://doi.org/10.1038/s41598-018-27627-3</a>.
</div>
<div id="ref-Luo1999" class="csl-entry">
Luo, Lin, Ranelle C. Salunga, Hongqing Guo, Anton Bittner, K. C. Joy, Jose E. Galindo, Huinian Xiao, et al. 1999. <span>“Gene Expression Profiles of Laser-Captured Adjacent Neuronal Subtypes.”</span> <em>Nature Medicine</em> 5 (January): 117–22. <a href="https://doi.org/10.1038/4806">https://doi.org/10.1038/4806</a>.
</div>
<div id="ref-Lynch1995" class="csl-entry">
Lynch, Andrew S., David Briggs, and Ian A. Hope. 1995. <span>“Developmental Expression Pattern Screen for Genes Predicted in the c. Elegans Genome Sequencing Project.”</span> <em>Nature Genetics</em> 11 (November): 309–13. <a href="https://doi.org/10.1038/ng1195-309">https://doi.org/10.1038/ng1195-309</a>.
</div>
<div id="ref-Maaskola2018" class="csl-entry">
Maaskola, Jonas, Ludvig Bergenstråhle, Aleksandra Jurek, José Fernández Navarro, Jens Lagergren, and Joakim Lundeberg. 2018. <span>“Charting Tissue Expression Anatomy by Spatial Transcriptome Decomposition.”</span> <em>bioRxiv</em>, December, 362624. <a href="https://doi.org/10.1101/362624">https://doi.org/10.1101/362624</a>.
</div>
<div id="ref-Macevicz1995" class="csl-entry">
Macevicz, Stephen. 1995. DNA sequencing by parallel olgonucleotide extensions, issued 1995. <a href="https://patents.google.com/patent/US5969119A/en">https://patents.google.com/patent/US5969119A/en</a>.
</div>
<div id="ref-Macosko2015" class="csl-entry">
Macosko, Evan Z., Anindita Basu, Rahul Satija, James Nemesh, Karthik Shekhar, Melissa Goldman, Itay Tirosh, et al. 2015. <span>“Highly Parallel Genome-Wide Expression Profiling of Individual Cells Using Nanoliter Droplets.”</span> <em>Cell</em> 161 (May): 1202–14. <a href="https://doi.org/10.1016/j.cell.2015.05.002">https://doi.org/10.1016/j.cell.2015.05.002</a>.
</div>
<div id="ref-Manco2020" class="csl-entry">
Manco, Rita, Inna Averbukh, Ziv Porat, Keren Bahar Halpern, Ido Amit, and Shalev Itzkovitz. 2020. <span>“Clump Sequencing Exposes the Spatial Expression Programs of Intestinal Secretory Cells.”</span> <em>bioRxiv</em>, August, 2020.08.05.237917. <a href="https://doi.org/10.1101/2020.08.05.237917">https://doi.org/10.1101/2020.08.05.237917</a>.
</div>
<div id="ref-Maniatis2019" class="csl-entry">
Maniatis, Silas, Tarmo Äijö, Sanja Vickovic, Catherine Braine, Kristy Kang, Annelie Mollbrink, Delphine Fagegaltier, et al. 2019. <span>“Spatiotemporal Dynamics of Molecular Pathology in Amyotrophic Lateral Sclerosis.”</span> <em>Science</em> 364 (April): 89 LP–93. <a href="https://doi.org/10.1126/science.aav9776">https://doi.org/10.1126/science.aav9776</a>.
</div>
<div id="ref-LaManno2021" class="csl-entry">
Manno, Gioele La, Kimberly Siletti, Alessandro Furlan, Daniel Gyllborg, Elin Vinsland, Alejandro Mossi Albiach, Christoffer Mattsson Langseth, et al. 2021. <span>“Molecular Architecture of the Developing Mouse Brain.”</span> <em>Nature</em> 596: 92–96. <a href="https://doi.org/10.1038/s41586-021-03775-x">https://doi.org/10.1038/s41586-021-03775-x</a>.
</div>
<div id="ref-LaManno2018" class="csl-entry">
Manno, Gioele La, Ruslan Soldatov, Amit Zeisel, Emelie Braun, Hannah Hochgerner, Viktor Petukhov, Katja Lidschreiber, et al. 2018. <span>“RNA Velocity of Single Cells.”</span> <em>Nature</em> 560: 494–98. <a href="https://doi.org/10.1038/s41586-018-0414-6">https://doi.org/10.1038/s41586-018-0414-6</a>.
</div>
<div id="ref-Mantri2020" class="csl-entry">
Mantri, Madhav, Gaetano J Scuderi, Roozbeh Abedini Nassab, Michael F Z Wang, David McKellar, Jonathan T Butcher, and Iwijn De Vlaminck. 2020. <span>“Spatiotemporal Single-Cell RNA Sequencing of Developing Hearts Reveals Interplay Between Cellular Differentiation and Morphogenesis.”</span> <em>bioRxiv</em>, January, 2020.05.03.065102. <a href="https://doi.org/10.1101/2020.05.03.065102">https://doi.org/10.1101/2020.05.03.065102</a>.
</div>
<div id="ref-Margaroli2021" class="csl-entry">
Margaroli, C, P Benson, N S Sharma, M C Madison, S W Robison, N Arora, K Ton, et al. 2021. <span>“Spatial Mapping of SARS-CoV-2 and H1N1 Lung Injury Identifies Differential Transcriptional Signatures.”</span> <em>Cell Reports. Medicine</em>, March, 100242. <a href="https://doi.org/10.1016/j.xcrm.2021.100242">https://doi.org/10.1016/j.xcrm.2021.100242</a>.
</div>
<div id="ref-Marquart2015" class="csl-entry">
Marquart, Gregory D., Kathryn M. Tabor, Mary Brown, Jennifer L. Strykowski, Gaurav K. Varshney, Matthew C. LaFave, Thomas Mueller, Shawn M. Burgess, Shin-ichi Higashijima, and Harold A. Burgess. 2015. <span>“A 3D Searchable Database of Transgenic Zebrafish Gal4 and Cre Lines for Functional Neuroanatomy Studies.”</span> <em>Frontiers in Neural Circuits</em> 9 (November): 1–17. <a href="https://doi.org/10.3389/fncir.2015.00078">https://doi.org/10.3389/fncir.2015.00078</a>.
</div>
<div id="ref-Martinelli1997" class="csl-entry">
Martinelli, Sylvia D., Clive G. Brown, and Richard Durbin. 1997. <span>“Gene Expression and Development Databases forC. Elegans.”</span> <em>Seminars in Cell &amp; Developmental Biology</em> 8 (October): 459–67. <a href="https://doi.org/10.1006/scdb.1997.0171">https://doi.org/10.1006/scdb.1997.0171</a>.
</div>
<div id="ref-CiolliMattioli2019" class="csl-entry">
Mattioli, Camilla Ciolli, Aviv Rom, Vedran Franke, Koshi Imami, Gerard Arrey, Mandy Terne, Andrew Woehler, Altuna Akalin, Igor Ulitsky, and Marina Chekulaeva. 2019. <span>“Alternative 3’ UTRs Direct Localization of Functionally Diverse Protein Isoforms in Neuronal Compartments.”</span> <em>Nucleic Acids Research</em> 47 (March): 2560–73. <a href="https://doi.org/10.1093/nar/gky1270">https://doi.org/10.1093/nar/gky1270</a>.
</div>
<div id="ref-Maynard2020" class="csl-entry">
Maynard, Kristen R, Madhavi Tippani, Yoichiro Takahashi, BaDoi N Phan, Thomas M Hyde, Andrew E Jaffe, and Keri Martinowich. 2020. <span>“Dotdotdot: An Automated Approach to Quantify Multiplex Single Molecule Fluorescent in Situ Hybridization (smFISH) Images in Complex Tissues.”</span> <em>Nucleic Acids Research</em> 48 (June): e66–66. <a href="https://doi.org/10.1093/nar/gkaa312">https://doi.org/10.1093/nar/gkaa312</a>.
</div>
<div id="ref-McCray2021" class="csl-entry">
McCray, Tara, Julian V Pacheco, Candice C Loitz, Jason Garcia, Bethany Baumann, Michael J Schlicht, Klara Valyi-Nagy, Michael R Abern, and Larisa Nonn. 2021. <span>“Vitamin d Sufficiency Enhances Differentiation of Patient-Derived Prostate Epithelial Organoids.”</span> <em>iScience</em> 24: 101974. https://doi.org/<a href="https://doi.org/10.1016/j.isci.2020.101974">https://doi.org/10.1016/j.isci.2020.101974</a>.
</div>
<div id="ref-McKee2005" class="csl-entry">
McKee, Adrienne E., Emmanuel Minet, Charlene Stern, Shervin Riahi, Charles D. Stiles, and Pamela A. Silver. 2005. <span>“A Genome-Wide in Situ Hybridization Map of RNA-Binding Proteins Reveals Anatomically Restricted Expression in the Developing Mouse Brain.”</span> <em>BMC Developmental Biology</em> 5 (July): 14. <a href="https://doi.org/10.1186/1471-213X-5-14">https://doi.org/10.1186/1471-213X-5-14</a>.
</div>
<div id="ref-Medaglia2017" class="csl-entry">
Medaglia, Chiara, Amir Giladi, Liat Stoler-Barak, Marco De Giovanni, Tomer Meir Salame, Adi Biram, Eyal David, et al. 2017. <span>“Spatial Reconstruction of Immune Niches by Combining Photoactivatable Reporters and scRNA-Seq.”</span> <em>Science</em> 358 (December): 1622–26. <a href="https://doi.org/10.1126/science.aao4277">https://doi.org/10.1126/science.aao4277</a>.
</div>
<div id="ref-Meier-Ruge1976" class="csl-entry">
Meier-Ruge, W., W. Bielser, E. Remy, F. Hillenkamp, R. Nitsche, and R. Unsold. 1976. <span>“The Laser in the Lowry Technique for Microdissection of Freeze-Dried Tissue Slices.”</span> <em>The Histochemical Journal</em> 8 (July): 387–401. <a href="https://doi.org/10.1007/BF01003828">https://doi.org/10.1007/BF01003828</a>.
</div>
<div id="ref-Melsted2021" class="csl-entry">
Melsted, Páll, A Sina Booeshaghi, Lauren Liu, Fan Gao, Lambda Lu, Kyung Hoi (Joseph) Min, Eduardo da Veiga Beltrame, Kristján Eldjárn Hjörleifsson, Jase Gehring, and Lior Pachter. 2021. <span>“Modular, Efficient and Constant-Memory Single-Cell RNA-Seq Preprocessing.”</span> <em>Nature Biotechnology</em>. <a href="https://doi.org/10.1038/s41587-021-00870-2">https://doi.org/10.1038/s41587-021-00870-2</a>.
</div>
<div id="ref-Merritt2020" class="csl-entry">
Merritt, Christopher R., Giang T. Ong, Sarah E. Church, Kristi Barker, Patrick Danaher, Gary Geiss, Margaret Hoang, et al. 2020. <span>“Multiplex Digital Spatial Profiling of Proteins and RNA in Fixed Tissue.”</span> <em>Nature Biotechnology 2020 38:5</em> 38 (May): 586–99. <a href="https://doi.org/10.1038/s41587-020-0472-9">https://doi.org/10.1038/s41587-020-0472-9</a>.
</div>
<div id="ref-Middleton2019" class="csl-entry">
Middleton, Sarah A, James Eberwine, and Junhyong Kim. 2019. <span>“Comprehensive Catalog of Dendritically Localized mRNA Isoforms from Sub-Cellular Sequencing of Single Mouse Neurons.”</span> <em>BMC Biology</em> 17: 5. <a href="https://doi.org/10.1186/s12915-019-0630-z">https://doi.org/10.1186/s12915-019-0630-z</a>.
</div>
<div id="ref-Miller2021a" class="csl-entry">
Miller, Brendan F, Lyla Atta, Arpan Sahoo, Feiyang Huang, and Jean Fan. 2021. <span>“Reference-Free Cell-Type Deconvolution of Pixel-Resolution Spatially Resolved Transcriptomics Data.”</span> <em>bioRxiv</em>, January, 2021.06.15.448381. <a href="https://doi.org/10.1101/2021.06.15.448381">https://doi.org/10.1101/2021.06.15.448381</a>.
</div>
<div id="ref-Miller2021" class="csl-entry">
Miller, Brendan F, Dhananjay Bambah-Mukku, Catherine Dulac, Xiaowei Zhuang, and Jean Fan. 2021. <span>“Characterizing Spatial Gene Expression Heterogeneity in Spatially Resolved Single-Cell Transcriptomic Data with Nonuniform Cellular Densities.”</span> <em>Genome Research</em>, May. <a href="https://doi.org/10.1101/gr.271288.120">https://doi.org/10.1101/gr.271288.120</a>.
</div>
<div id="ref-Miller2014" class="csl-entry">
Miller, Jeremy A., Song-Lin Ding, Susan M. Sunkin, Kimberly A. Smith, Lydia Ng, Aaron Szafer, Amanda Ebbert, et al. 2014. <span>“Transcriptional Landscape of the Prenatal Human Brain.”</span> <em>Nature</em> 508 (April): 199–206. <a href="https://doi.org/10.1038/nature13185">https://doi.org/10.1038/nature13185</a>.
</div>
<div id="ref-Mirzabekov1994" class="csl-entry">
Mirzabekov, Andrei D. 1994. <span>“DNA Sequencing by Hybridization &amp;#X2014; a Megasequencing Method and a Diagnostic Tool?”</span> <em>Trends in Biotechnology</em> 12 (January): 27–32. <a href="https://doi.org/10.1016/0167-7799(94)90008-6">https://doi.org/10.1016/0167-7799(94)90008-6</a>.
</div>
<div id="ref-Moffitt2016" class="csl-entry">
Moffitt, J. R., and X. Zhuang. 2016. <span>“RNA Imaging with Multiplexed Error-Robust Fluorescence in Situ Hybridization (MERFISH).”</span> <em>Methods in Enzymology</em>. <a href="https://doi.org/10.1016/bs.mie.2016.03.020">https://doi.org/10.1016/bs.mie.2016.03.020</a>.
</div>
<div id="ref-Moffitt2018" class="csl-entry">
Moffitt, Jeffrey R., Dhananjay Bambah-Mukku, Stephen W. Eichhorn, Eric Vaughn, Karthik Shekhar, Julio D. Perez, Nimrod D. Rubinstein, et al. 2018. <span>“Molecular, Spatial, and Functional Single-Cell Profiling of the Hypothalamic Preoptic Region.”</span> <em>Science</em>. <a href="https://doi.org/10.1126/science.aau5324">https://doi.org/10.1126/science.aau5324</a>.
</div>
<div id="ref-Moffitt2016a" class="csl-entry">
Moffitt, Jeffrey R., Junjie Hao, Guiping Wang, Kok Hao Chen, Hazen P. Babcock, and Xiaowei Zhuang. 2016. <span>“High-Throughput Single-Cell Gene-Expression Profiling with Multiplexed Error-Robust Fluorescence in Situ Hybridization.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1612826113">https://doi.org/10.1073/pnas.1612826113</a>.
</div>
<div id="ref-Moncada2020" class="csl-entry">
Moncada, Reuben, Dalia Barkley, Florian Wagner, Marta Chiodin, Joseph C Devlin, Maayan Baron, Cristina H Hajdu, Diane M Simeone, and Itai Yanai. 2020. <span>“Integrating Microarray-Based Spatial Transcriptomics and Single-Cell RNA-Seq Reveals Tissue Architecture in Pancreatic Ductal Adenocarcinomas.”</span> <em>Nature Biotechnology</em> 38: 333–42. <a href="https://doi.org/10.1038/s41587-019-0392-8">https://doi.org/10.1038/s41587-019-0392-8</a>.
</div>
<div id="ref-Moor2017a" class="csl-entry">
Moor, Andreas E., Matan Golan, Efi E. Massasa, Doron Lemze, Tomer Weizman, Rom Shenhav, Shaked Baydatch, et al. 2017. <span>“Global mRNA Polarization Regulates Translation Efficiency in the Intestinal Epithelium.”</span> <em>Science</em> 357 (September): 1299–1303. <a href="https://doi.org/10.1126/science.aan2399">https://doi.org/10.1126/science.aan2399</a>.
</div>
<div id="ref-Moor2018" class="csl-entry">
Moor, Andreas E., Yotam Harnik, Shani Ben-Moshe, Efi E. Massasa, Milena Rozenberg, Raya Eilam, Keren Bahar Halpern, and Shalev Itzkovitz. 2018. <span>“Spatial Reconstruction of Single Enterocytes Uncovers Broad Zonation Along the Intestinal Villus Axis.”</span> <em>Cell</em> 175 (November): 1156–1167.e15. <a href="https://doi.org/10.1016/j.cell.2018.08.063">https://doi.org/10.1016/j.cell.2018.08.063</a>.
</div>
<div id="ref-Moor2017" class="csl-entry">
Moor, Andreas E., and Shalev Itzkovitz. 2017. <span>“Spatial Transcriptomics: Paving the Way for Tissue-Level Systems Biology.”</span> <em>Current Opinion in Biotechnology</em> 46 (August): 126–33. <a href="https://doi.org/10.1016/j.copbio.2017.02.004">https://doi.org/10.1016/j.copbio.2017.02.004</a>.
</div>
<div id="ref-Mori2019" class="csl-entry">
Mori, Tomoya, Haruka Takaoka, Junko Yamane, Cantas Alev, and Wataru Fujibuchi. 2019. <span>“Novel Computational Model of Gastrula Morphogenesis to Identify Spatial Discriminator Genes by Self-Organizing Map (SOM) Clustering.”</span> <em>Scientific Reports</em> 9 (December): 1–10. <a href="https://doi.org/10.1038/s41598-019-49031-1">https://doi.org/10.1038/s41598-019-49031-1</a>.
</div>
<div id="ref-Mori2017" class="csl-entry">
Mori, Tomoya, Junko Yamane, Kenta Kobayashi, Nobuko Taniyama, Takanori Tano, and Wataru Fujibuchi. 2017. <span>“Development of 3D Tissue Reconstruction Method from Single-Cell RNA-Seq Data.”</span> <em>Genomics and Computational Biology; Vol 3 No 1 (2017)</em>. <a href="https://doi.org/10.18547/gcb.2017.vol3.iss1.e53">https://doi.org/10.18547/gcb.2017.vol3.iss1.e53</a>.
</div>
<div id="ref-Moris2020" class="csl-entry">
Moris, Naomi, Kerim Anlas, Susanne C. van den Brink, Anna Alemany, Julia Schröder, Sabitri Ghimire, Tina Balayo, Alexander van Oudenaarden, and Alfonso Martinez Arias. 2020. <span>“An in Vitro Model of Early Anteroposterior Organization During Human Development.”</span> <em>Nature</em> 582 (June): 410–15. <a href="https://doi.org/10.1038/s41586-020-2383-9">https://doi.org/10.1038/s41586-020-2383-9</a>.
</div>
<div id="ref-Morton2014" class="csl-entry">
Morton, Matthew L., Xiaodong Bai, Callie R. Merry, Philip A. Linden, Ahmad M. Khalil, Rom S. Leidner, and Cheryl L. Thompson. 2014. <span>“Identification of mRNAs and lincRNAs Associated with Lung Cancer Progression Using Next-Generation RNA Sequencing from Laser Micro-Dissected Archival FFPE Tissue Specimens.”</span> <em>Lung Cancer</em> 85 (July): 31–39. <a href="https://doi.org/10.1016/j.lungcan.2014.03.020">https://doi.org/10.1016/j.lungcan.2014.03.020</a>.
</div>
<div id="ref-Myers2000" class="csl-entry">
Myers, Eugene W. 2000. <span>“A Whole-Genome Assembly of Drosophila.”</span> <em>Science</em> 287 (March): 2196–2204. <a href="https://doi.org/10.1126/science.287.5461.2196">https://doi.org/10.1126/science.287.5461.2196</a>.
</div>
<div id="ref-Nagalakshmi2008" class="csl-entry">
Nagalakshmi, Ugrappa, Zhong Wang, Karl Waern, Chong Shou, Debasish Raha, Mark Gerstein, and Michael Snyder. 2008. <span>“The Transcriptional Landscape of the Yeast Genome Defined by RNA Sequencing.”</span> <em>Science</em> 320 (June): 1344–49. <a href="https://doi.org/10.1126/science.1158441">https://doi.org/10.1126/science.1158441</a>.
</div>
<div id="ref-Nagarajan2020" class="csl-entry">
Nagarajan, Maxwell B, Augusto M Tentori, Wen Cai Zhang, Frank J Slack, and Patrick S Doyle. 2020. <span>“Spatially Resolved and Multiplexed MicroRNA Quantification from Tissue Using Nanoliter Well Arrays.”</span> <em>Microsystems &amp; Nanoengineering</em> 6: 51. <a href="https://doi.org/10.1038/s41378-020-0169-8">https://doi.org/10.1038/s41378-020-0169-8</a>.
</div>
<div id="ref-Nakamura2004" class="csl-entry">
Nakamura, Toru, Yoichi Furukawa, Hidewaki Nakagawa, Tatsuhiko Tsunoda, Hiroaki Ohigashi, Kohei Murata, Osamu Ishikawa, et al. 2004. <span>“Genome-Wide cDNA Microarray Analysis of Gene Expression Profiles in Pancreatic Cancers Using Populations of Tumor Cells and Normal Ductal Epithelial Cells Selected for Purity by Laser Microdissection.”</span> <em>Oncogene</em> 23 (March): 2385–2400. <a href="https://doi.org/10.1038/sj.onc.1207392">https://doi.org/10.1038/sj.onc.1207392</a>.
</div>
<div id="ref-Nakayama2005" class="csl-entry">
Nakayama, Naomi, Juana M. Arroyo, Joseph Simorowski, Bruce May, Robert Martienssen, and Vivian F. Irish. 2005. <span>“Gene Trap Lines Define Domains of Gene Regulation in Arabidopsis Petals and Stamens.”</span> <em>The Plant Cell</em> 17 (September): 2486–506. <a href="https://doi.org/10.1105/tpc.105.033985">https://doi.org/10.1105/tpc.105.033985</a>.
</div>
<div id="ref-Navarro2017" class="csl-entry">
Navarro, José Fernández, Joel Sjöstrand, Fredrik Salmén, Joakim Lundeberg, and Patrik L. Ståhl. 2017. <span>“ST Pipeline: An Automated Pipeline for Spatial Mapping of Unique Transcripts.”</span> <em>Bioinformatics (Oxford, England)</em>. <a href="https://doi.org/10.1093/bioinformatics/btx211">https://doi.org/10.1093/bioinformatics/btx211</a>.
</div>
<div id="ref-Nederlof1990" class="csl-entry">
Nederlof, P. M., S. van der Flier, J. Wiegant, A. K. Raap, H. J. Tanke, J. S. Ploem, and M. van der Ploeg. 1990. <span>“Multiple Fluorescence in Situ Hybridization.”</span> <em>Cytometry</em> 11: 126–31. <a href="https://doi.org/10.1002/cyto.990110115">https://doi.org/10.1002/cyto.990110115</a>.
</div>
<div id="ref-Newman2015" class="csl-entry">
Newman, Aaron M, Chih Long Liu, Michael R Green, Andrew J Gentles, Weiguo Feng, Yue Xu, Chuong D Hoang, Maximilian Diehn, and Ash A Alizadeh. 2015. <span>“Robust Enumeration of Cell Subsets from Tissue Expression Profiles.”</span> <em>Nature Methods</em> 12: 453–57. <a href="https://doi.org/10.1038/nmeth.3337">https://doi.org/10.1038/nmeth.3337</a>.
</div>
<div id="ref-Nichterwitz2016" class="csl-entry">
Nichterwitz, Susanne, Geng Chen, Julio Aguila Benitez, Marlene Yilmaz, Helena Storvall, Ming Cao, Rickard Sandberg, Qiaolin Deng, and Eva Hedlund. 2016. <span>“Laser Capture Microscopy Coupled with Smart-Seq2 for Precise Spatial Transcriptomic Profiling.”</span> <em>Nature Communications</em> 7 (November): 12139. <a href="https://doi.org/10.1038/ncomms12139">https://doi.org/10.1038/ncomms12139</a>.
</div>
<div id="ref-Nilsson1994" class="csl-entry">
Nilsson, Mats, Helena Malmgren, Martina Samiotaki, Marek Kwiatkowski, B. Chowdhary, and Ulf Landegren. 1994. <span>“Padlock Probes: Circularizing Oligonucleotides for Localized DNA Detection.”</span> <em>Science</em> 265 (September): 2085–88. <a href="https://doi.org/10.1126/science.7522346">https://doi.org/10.1126/science.7522346</a>.
</div>
<div id="ref-Nitzan2019" class="csl-entry">
Nitzan, Mor, Nikos Karaiskos, Nir Friedman, and Nikolaus Rajewsky. 2019. <span>“Gene Expression Cartography.”</span> <em>Nature</em> 576 (December): 132–37. <a href="https://doi.org/10.1038/s41586-019-1773-3">https://doi.org/10.1038/s41586-019-1773-3</a>.
</div>
<div id="ref-Noto2017" class="csl-entry">
Noto, Torben, Derrick Barnagian, and Jason B. Castro. 2017. <span>“Genome-Scale Investigation of Olfactory System Spatial Heterogeneity.”</span> Edited by Hiroaki Matsunami. <em>PLOS ONE</em> 12 (May): e0178087. <a href="https://doi.org/10.1371/journal.pone.0178087">https://doi.org/10.1371/journal.pone.0178087</a>.
</div>
<div id="ref-OKane1987" class="csl-entry">
O’Kane, C. J., and W. J. Gehring. 1987. <span>“Detection in Situ of Genomic Regulatory Elements in Drosophila.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 84 (December): 9123–27. <a href="https://doi.org/10.1073/pnas.84.24.9123">https://doi.org/10.1073/pnas.84.24.9123</a>.
</div>
<div id="ref-Ohyama2000" class="csl-entry">
Ohyama, H., X. Zhang, Y. Kohno, I. Alevizos, M. Posner, D. T. Wong, and R. Todd. 2000. <span>“Laser Capture Microdissection-Generated Target Sample for High-Density Oligonucleotide Array Hybridization.”</span> <em>BioTechniques</em> 29 (September): 530–36. <a href="https://doi.org/10.2144/00293st05">https://doi.org/10.2144/00293st05</a>.
</div>
<div id="ref-Oka2015" class="csl-entry">
Oka, Yuma, and Thomas N Sato. 2015. <span>“Whole-Mount Single Molecule FISH Method for Zebrafish Embryo.”</span> <em>Scientific Reports</em> 5 (February): 8571. <a href="https://doi.org/10.1038/srep08571">https://doi.org/10.1038/srep08571</a>.
</div>
<div id="ref-Okamura-Oho2012" class="csl-entry">
Okamura-Oho, Yuko, Kazuro Shimokawa, Satoko Takemoto, Asami Hirakiyama, Sakiko Nakamura, Yuki Tsujimura, Masaomi Nishimura, et al. 2012. <span>“Transcriptome Tomography for Brain Analysis in the Web-Accessible Anatomical Space.”</span> Edited by Satoru Hayasaka. <em>PLoS ONE</em> 7 (September): e45373. <a href="https://doi.org/10.1371/journal.pone.0045373">https://doi.org/10.1371/journal.pone.0045373</a>.
</div>
<div id="ref-Okochi2021" class="csl-entry">
Okochi, Yasushi, Shunta Sakaguchi, Ken Nakae, Takefumi Kondo, and Honda Naoki. 2021. <span>“Model-Based Prediction of Spatial Gene Expression via Generative Linear Mapping.”</span> <em>Nature Communications</em> 12: 3731. <a href="https://doi.org/10.1038/s41467-021-24014-x">https://doi.org/10.1038/s41467-021-24014-x</a>.
</div>
<div id="ref-Ortiz2020" class="csl-entry">
Ortiz, Cantin, Jose Fernandez Navarro, Aleksandra Jurek, Antje Märtin, Joakim Lundeberg, and Konstantinos Meletis. 2020. <span>“Molecular Atlas of the Adult Mouse Brain.”</span> <em>Science Advances</em> 6 (June): eabb3446. <a href="https://doi.org/10.1126/sciadv.abb3446">https://doi.org/10.1126/sciadv.abb3446</a>.
</div>
<div id="ref-Palla2022" class="csl-entry">
Palla, Giovanni, Hannah Spitzer, Michal Klein, David Fischer, Anna Christina Schaar, Louis Benedikt Kuemmerle, Sergei Rybakov, et al. 2022. <span>“Squidpy: A Scalable Framework for Spatial Omics Analysis.”</span> <em>Nat. Methods</em> 19 (February): 171–78.
</div>
<div id="ref-Pan2006" class="csl-entry">
Pan, Jia-Yu, André Guilherme, Ribeiro Balan, Eric P. Xing, Agma Juci Machado Traina, and Christos Faloutsos. 2006. <span>“Automatic Mining of Fruit Fly Embryo Images.”</span> In <em>Proceedings of the 12th ACM SIGKDD International Conference on Knowledge Discovery and Data Mining - KDD ’06</em>, 2006:693. ACM Press. <a href="https://doi.org/10.1145/1150402.1150489">https://doi.org/10.1145/1150402.1150489</a>.
</div>
<div id="ref-Park2019" class="csl-entry">
Park, Jeongbin, Wonyl Choi, Sebastian Tiesmeyer, Brian Long, Lars E. Borm, Emma Garren, Thuc Nghi Nguyen, et al. 2019. <span>“Segmentation-Free Inference of Cell Types from in Situ Transcriptomics Data.”</span> <em>bioRxiv</em>. <a href="https://doi.org/10.1101/800748">https://doi.org/10.1101/800748</a>.
</div>
<div id="ref-Park2021" class="csl-entry">
Park, Jiwoon, Jonathan Foox, Tyler Hether, David Danko, Sarah Warren, Youngmi Kim, Jason Reeves, et al. 2021. <span>“Systemic Tissue and Cellular Disruption from SARS-CoV-2 Infection Revealed in COVID-19 Autopsies and Spatial Omics Tissue Maps.”</span> <em>bioRxiv</em>, January, 2021.03.08.434433. <a href="https://doi.org/10.1101/2021.03.08.434433">https://doi.org/10.1101/2021.03.08.434433</a>.
</div>
<div id="ref-Partel2019" class="csl-entry">
Partel, Gabriele, Markus Hilscher, Giorgia Milli, Leslie Solorzano, Anna Klemm, Mats Nilsson, and Carolina Wählby. 2019. <span>“Identification of Spatial Compartments in Tissue from in Situ Sequencing Data.”</span> <em>bioRxiv</em>, September, 765842. <a href="https://doi.org/10.1101/765842">https://doi.org/10.1101/765842</a>.
</div>
<div id="ref-Partel2020" class="csl-entry">
Partel, Gabriele, and Carolina Wählby. 2020. <span>“Spage2vec: Unsupervised Detection of Spatial Gene Expression Constellations.”</span> <em>bioRxiv</em>, February, 2020.02.12.945345. <a href="https://doi.org/10.1101/2020.02.12.945345">https://doi.org/10.1101/2020.02.12.945345</a>.
</div>
<div id="ref-Parthasarathy2012" class="csl-entry">
Parthasarathy, Raghuveer. 2012. <span>“Rapid, Accurate Particle Tracking by Calculation of Radial Symmetry Centers.”</span> <em>Nature Methods</em> 9: 724–26. <a href="https://doi.org/10.1038/nmeth.2071">https://doi.org/10.1038/nmeth.2071</a>.
</div>
<div id="ref-Patrick2021" class="csl-entry">
Patrick, Ellis, Nicolas P Canete, Sourish S Iyengar, Andrew N Harman, Greg T Sutherland, and Pengyi Yang. 2021. <span>“Spatial Analysis for Highly Multiplexed Imaging Data to Identify Tissue Microenvironments.”</span> <em>bioRxiv</em>, January, 2021.08.16.456469. <a href="https://doi.org/10.1101/2021.08.16.456469">https://doi.org/10.1101/2021.08.16.456469</a>.
</div>
<div id="ref-Patrushev2018" class="csl-entry">
Patrushev, Ilya, Christina James-Zorn, Aldo Ciau-Uitz, Roger Patient, and Michael J. Gilchrist. 2018. <span>“New Methods for Computational Decomposition of Whole-Mount in Situ Images Enable Effective Curation of a Large, Highly Redundant Collection of Xenopus Images.”</span> <em>PLoS Computational Biology</em> 14 (August): e1006077. <a href="https://doi.org/10.1371/journal.pcbi.1006077">https://doi.org/10.1371/journal.pcbi.1006077</a>.
</div>
<div id="ref-Peng2016" class="csl-entry">
Peng, Guangdun, Shengbao Suo, Jun Chen, Weiyang Chen, Chang Liu, Fang Yu, Ran Wang, et al. 2016. <span>“Spatial Transcriptome for the Molecular Annotation of Lineage Fates and Cell Identity in Mid-Gastrula Mouse Embryo.”</span> <em>Developmental Cell</em> 36 (March): 681–97. <a href="https://doi.org/10.1016/j.devcel.2016.02.020">https://doi.org/10.1016/j.devcel.2016.02.020</a>.
</div>
<div id="ref-Peng2006" class="csl-entry">
Peng, Hanchuan, Fuhui Long, M. B. Eisen, and E. W. Myers. 2006. <span>“Clustering Gene Expression Patterns of Fly Embryos.”</span> In <em>3rd IEEE International Symposium on Biomedical Imaging: Macro to Nano, 2006.</em>, 2006:1144–47. IEEE. <a href="https://doi.org/10.1109/ISBI.2006.1625125">https://doi.org/10.1109/ISBI.2006.1625125</a>.
</div>
<div id="ref-Pennington" class="csl-entry">
Pennington, Jeffrey, Richard Socher, and Christopher D Manning. n.d. <span>“GloVe: Global Vectors for Word Representation.”</span>
</div>
<div id="ref-Perez-Martin2017" class="csl-entry">
Pérez-Martín, Fernando, Fernando J. Yuste-Lisbona, Benito Pineda, María Pilar Angarita-Díaz, Begoña García-Sogo, Teresa Antón, Sibilla Sánchez, et al. 2017. <span>“A Collection of Enhancer Trap Insertional Mutants for Functional Genomics in Tomato.”</span> <em>Plant Biotechnology Journal</em> 15 (November): 1439–52. <a href="https://doi.org/10.1111/pbi.12728">https://doi.org/10.1111/pbi.12728</a>.
</div>
<div id="ref-Perkel2019" class="csl-entry">
Perkel, Jeffrey M. 2019. <span>“Starfish Enterprise: Finding RNA Patterns in Single Cells.”</span> <em>Nature</em> 572 (August): 549–51. <a href="https://doi.org/10.1038/D41586-019-02477-9">https://doi.org/10.1038/D41586-019-02477-9</a>.
</div>
<div id="ref-Pettit2014" class="csl-entry">
Pettit, Jean-Baptiste, Raju Tomer, Kaia Achim, Sylvia Richardson, Lamiae Azizi, and John Marioni. 2014. <span>“Identifying Cell Types from Spatially Referenced Single-Cell Expression Datasets.”</span> Edited by Quaid Morris. <em>PLoS Computational Biology</em> 10 (September): e1003824. <a href="https://doi.org/10.1371/journal.pcbi.1003824">https://doi.org/10.1371/journal.pcbi.1003824</a>.
</div>
<div id="ref-Petukhov2020" class="csl-entry">
Petukhov, Viktor, Ruslan A Soldatov, Konstantin Khodosevich, and Peter V Kharchenko. 2020. <span>“Bayesian Segmentation of Spatially Resolved Transcriptomics Data.”</span> <em>bioRxiv</em>, January, 2020.10.05.326777. <a href="https://doi.org/10.1101/2020.10.05.326777">https://doi.org/10.1101/2020.10.05.326777</a>.
</div>
<div id="ref-Pham2020a" class="csl-entry">
Pham, Duy, Xiao Tan, Jun Xu, Laura F Grice, Pui Yeng Lam, Arti Raghubar, Jana Vukovic, Marc J Ruitenberg, and Quan Nguyen. 2020. <span>“stLearn: Integrating Spatial Location, Tissue Morphology and Gene Expression to Find Cell Types, Cell-Cell Interactions and Spatial Trajectories Within Undissociated Tissues.”</span> <em>bioRxiv</em>, January, 2020.05.31.125658. <a href="https://doi.org/10.1101/2020.05.31.125658">https://doi.org/10.1101/2020.05.31.125658</a>.
</div>
<div id="ref-Player2001" class="csl-entry">
Player, Audrey N., Lu Ping Shen, Daryn Kenny, Vincent P. Antao, and Janice A. Kolberg. 2001. <span>“Single-Copy Gene Detection Using Branched DNA (bDNA) in Situ Hybridization.”</span> <em>Journal of Histochemistry and Cytochemistry</em> 49 (May): 603–11. <a href="https://doi.org/10.1177/002215540104900507">https://doi.org/10.1177/002215540104900507</a>.
</div>
<div id="ref-Plouhinec2017" class="csl-entry">
Plouhinec, Jean-Louis, Sofía Medina-Ruiz, Caroline Borday, Elsa Bernard, Jean-Philippe Vert, Michael B. Eisen, Richard M. Harland, and Anne H. Monsoro-Burq. 2017. <span>“A Molecular Atlas of the Developing Ectoderm Defines Neural, Neural Crest, Placode, and Nonneural Progenitor Identity in Vertebrates.”</span> Edited by James Briscoe. <em>PLOS Biology</em> 15 (October): e2004045. <a href="https://doi.org/10.1371/journal.pbio.2004045">https://doi.org/10.1371/journal.pbio.2004045</a>.
</div>
<div id="ref-Pruteanu-Malinici2011" class="csl-entry">
Pruteanu-Malinici, Iulian, Daniel L. Mace, and Uwe Ohler. 2011. <span>“Automatic Annotation of Spatial Expression Patterns via Sparse Bayesian Factor Models.”</span> Edited by Joel S. Bader. <em>PLoS Computational Biology</em> 7 (July): e1002098. <a href="https://doi.org/10.1371/journal.pcbi.1002098">https://doi.org/10.1371/journal.pcbi.1002098</a>.
</div>
<div id="ref-Puchalski2018" class="csl-entry">
Puchalski, Ralph B, Nameeta Shah, Jeremy Miller, Rachel Dalley, Steve R Nomura, Jae-Guen Yoon, Kimberly A Smith, et al. 2018. <span>“An Anatomic Transcriptional Atlas of Human Glioblastoma.”</span> <em>Science</em> 360 (May): 660 LP–663. <a href="https://doi.org/10.1126/science.aaf2666">https://doi.org/10.1126/science.aaf2666</a>.
</div>
<div id="ref-Puniyani2013" class="csl-entry">
Puniyani, Kriti, and Eric P. Xing. 2013. <span>“GINI: From ISH Images to Gene Interaction Networks.”</span> Edited by Gal Chechik. <em>PLoS Computational Biology</em> 9 (October): e1003227. <a href="https://doi.org/10.1371/journal.pcbi.1003227">https://doi.org/10.1371/journal.pcbi.1003227</a>.
</div>
<div id="ref-Qian2020" class="csl-entry">
Qian, Xiaoyan, Kenneth D. Harris, Thomas Hauling, Dimitris Nicoloutsopoulos, Ana B. Muñoz-Manchado, Nathan Skene, Jens Hjerling-Leffler, and Mats Nilsson. 2020. <span>“Probabilistic Cell Typing Enables Fine Mapping of Closely Related Cell Types in Situ.”</span> <em>Nature Methods</em> 17 (January): 101–6. <a href="https://doi.org/10.1038/s41592-019-0631-4">https://doi.org/10.1038/s41592-019-0631-4</a>.
</div>
<div id="ref-Raj2008" class="csl-entry">
Raj, Arjun, Patrick van den Bogaard, Scott A. Rifkin, Alexander van Oudenaarden, and Sanjay Tyagi. 2008. <span>“Imaging Individual mRNA Molecules Using Multiple Singly Labeled Probes.”</span> <em>Nature Methods</em> 5 (October): 877–79. <a href="https://doi.org/10.1038/nmeth.1253">https://doi.org/10.1038/nmeth.1253</a>.
</div>
<div id="ref-Ramskold2012" class="csl-entry">
Ramsköld, Daniel, Shujun Luo, Yu-Chieh Wang, Robin Li, Qiaolin Deng, Omid R Faridani, Gregory A Daniels, et al. 2012. <span>“Full-Length mRNA-Seq from Single-Cell Levels of RNA and Individual Circulating Tumor Cells.”</span> <em>Nature Biotechnology</em> 30 (August): 777–82. <a href="https://doi.org/10.1038/nbt.2282">https://doi.org/10.1038/nbt.2282</a>.
</div>
<div id="ref-Regev2017" class="csl-entry">
Regev, Aviv, Sarah A Teichmann, Eric S Lander, Ido Amit, Christophe Benoist, Ewan Birney, Bernd Bodenmiller, et al. 2017. <span>“The Human Cell Atlas.”</span> Edited by Thomas R Gingeras. <em>eLife</em> 6: e27041. <a href="https://doi.org/10.7554/eLife.27041">https://doi.org/10.7554/eLife.27041</a>.
</div>
<div id="ref-Reid2011" class="csl-entry">
Reid, Carlton. 2011. <span>“19th Century Cyclists Paved the Way for Modern Motorists’ Roads.”</span>
</div>
<div id="ref-Ren2020" class="csl-entry">
Ren, Xianwen, Guojie Zhong, Qiming Zhang, Lei Zhang, Yujie Sun, and Zemin Zhang. 2020. <span>“Reconstruction of Cell Spatial Organization Based on Ligand-Receptor Mediated Self-Assembly.”</span> <em>bioRxiv</em>, January, 2020.02.13.948521. <a href="https://doi.org/10.1101/2020.02.13.948521">https://doi.org/10.1101/2020.02.13.948521</a>.
</div>
<div id="ref-Righelli2022" class="csl-entry">
Righelli, Dario, Lukas M. Weber, Helena L. Crowell, Brenda Pardo, Leonardo Collado-Torres, Shila Ghazanfar, Aaron T. L. Lun, Stephanie C. Hicks, and Davide Risso. 2022. <span>“SpatialExperiment: Infrastructure for Spatially-Resolved Transcriptomics Data in r Using Bioconductor.”</span> <em>Bioinformatics</em> 38 (May): 3128–31. <a href="https://doi.org/10.1093/BIOINFORMATICS/BTAC299">https://doi.org/10.1093/BIOINFORMATICS/BTAC299</a>.
</div>
<div id="ref-Ringwald1994" class="csl-entry">
Ringwald, Martin, Richard Baldock, Jonathan Bard, Matthew Kaufman, Janan T. Eppig, Joel E. Richardson, Joseph H. Nadeau, and Duncan Davidson. 1994. <span>“A Database for Mouse Development.”</span> <em>Science</em> 265 (September): 2033–34. <a href="https://doi.org/10.1126/science.8091224">https://doi.org/10.1126/science.8091224</a>.
</div>
<div id="ref-Ringwald1997" class="csl-entry">
Ringwald, Martin, Geoffrey L Davis, Alex G Smith, Laura E Trepanier, Dale A Begley, Joel E Richardson, and Janan T Eppig. 1997. <span>“The Mouse Gene Expression Database GXD.”</span> <em>Seminars in Cell &amp; Developmental Biology</em> 8: 489–97. https://doi.org/<a href="https://doi.org/10.1006/scdb.1997.0177">https://doi.org/10.1006/scdb.1997.0177</a>.
</div>
<div id="ref-Ringwald1999" class="csl-entry">
Ringwald, Martin, Mary E Mangan, Janan T Eppig, James A Kadin, and Joel E Richardson. 1999. <span>“GXD: A Gene Expression Database for the Laboratory Mouse.”</span> <em>Nucleic Acids Research</em> 27 (January): 106–12. <a href="https://doi.org/10.1093/nar/27.1.106">https://doi.org/10.1093/nar/27.1.106</a>.
</div>
<div id="ref-Rizvi2017" class="csl-entry">
Rizvi, Abbas H, Pablo G Camara, Elena K Kandror, Thomas J Roberts, Ira Schieren, Tom Maniatis, and Raul Rabadan. 2017. <span>“Single-Cell Topological RNA-Seq Analysis Reveals Insights into Cellular Differentiation and Development.”</span> <em>Nature Biotechnology</em> 35: 551–60. <a href="https://doi.org/10.1038/nbt.3854">https://doi.org/10.1038/nbt.3854</a>.
</div>
<div id="ref-Roberts2021" class="csl-entry">
Roberts, Kenny, Alexander Aivazidis, Vitalii Kleshchevnikov, Tong Li, Robin Fropf, Michael Rhodes, Joseph M Beechem, Martin Hemberg, and Omer Ali Bayraktar. 2021. <span>“Transcriptome-Wide Spatial RNA Profiling Maps the Cellular Architecture of the Developing Human Neocortex.”</span> <em>bioRxiv</em>, January, 2021.03.20.436265. <a href="https://doi.org/10.1101/2021.03.20.436265">https://doi.org/10.1101/2021.03.20.436265</a>.
</div>
<div id="ref-Roberts2019" class="csl-entry">
Roberts, Margaret E., Brandon M. Stewart, and Dustin Tingley. 2019. <span>“Stm: An r Package for Structural Topic Models.”</span> <em>Journal of Statistical Software</em> 91. <a href="https://doi.org/10.18637/jss.v091.i02">https://doi.org/10.18637/jss.v091.i02</a>.
</div>
<div id="ref-Robinson2021" class="csl-entry">
Robinson, Karen R. 2021. <span>“Comparing the Spanish Flu and COVID-19 Pandemics: Lessons to Carry Forward.”</span> <em>Nursing Forum</em> 56 (April): 350–57. https://doi.org/<a href="https://doi.org/10.1111/nuf.12534">https://doi.org/10.1111/nuf.12534</a>.
</div>
<div id="ref-Rodelsperger2020" class="csl-entry">
Rödelsperger, Christian, Annabel Ebbing, Devansh Raj Sharma, Misako Okumura, Ralf J Sommer, and Hendrik C Korswagen. 2020. <span>“Spatial Transcriptomics of Nematodes Identifies Sperm Cells as a Source of Genomic Novelty and Rapid Evolution.”</span> <em>Molecular Biology and Evolution</em>, August. <a href="https://doi.org/10.1093/molbev/msaa207">https://doi.org/10.1093/molbev/msaa207</a>.
</div>
<div id="ref-Rodriques2019" class="csl-entry">
Rodriques, Samuel G., Robert R. Stickels, Aleksandrina Goeva, Carly A. Martin, Evan Murray, Charles R. Vanderburg, Joshua Welch, Linlin M. Chen, Fei Chen, and Evan Z. Macosko. 2019. <span>“Slide-Seq: A Scalable Technology for Measuring Genome-Wide Expression at High Spatial Resolution.”</span> <em>Science</em>. <a href="https://doi.org/10.1126/science.aaw1219">https://doi.org/10.1126/science.aaw1219</a>.
</div>
<div id="ref-Rosen1993" class="csl-entry">
Rosen, Barry, and Rosa S. P. Beddington. 1993. <span>“Whole-Mount in Situ Hybridization in the Mouse Embryo: Gene Expression in Three Dimensions.”</span> <em>Trends in Genetics</em> 9 (May): 162–67. <a href="https://doi.org/10.1016/0168-9525(93)90162-B">https://doi.org/10.1016/0168-9525(93)90162-B</a>.
</div>
<div id="ref-Saelens2019" class="csl-entry">
Saelens, Wouter, Robrecht Cannoodt, Helena Todorov, and Yvan Saeys. 2019. <span>“A Comparison of Single-Cell Trajectory Inference Methods.”</span> <em>Nature Biotechnology</em> 37: 547–54. <a href="https://doi.org/10.1038/s41587-019-0071-9">https://doi.org/10.1038/s41587-019-0071-9</a>.
</div>
<div id="ref-Salgado2008" class="csl-entry">
Salgado, David, Gregory Gimenez, François Coulier, and Christophe Marcelle. 2008. <span>“COMPARE, a Multi-Organism System for Cross-Species Data Comparison and Transfer of Information.”</span> <em>Bioinformatics</em> 24 (February): 447–49. <a href="https://doi.org/10.1093/bioinformatics/btm599">https://doi.org/10.1093/bioinformatics/btm599</a>.
</div>
<div id="ref-Samacoits2018a" class="csl-entry">
Samacoits, Aubin, Racha Chouaib, Adham Safieddine, Abdel-Meneem Traboulsi, Wei Ouyang, Christophe Zimmer, Marion Peter, Edouard Bertrand, Thomas Walter, and Florian Mueller. 2018. <span>“A Computational Framework to Study Sub-Cellular RNA Localization.”</span> <em>Nature Communications</em> 9 (December): 4584. <a href="https://doi.org/10.1038/s41467-018-06868-w">https://doi.org/10.1038/s41467-018-06868-w</a>.
</div>
<div id="ref-Satija2015" class="csl-entry">
Satija, Rahul, Jeffrey A. Farrell, David Gennert, Alexander F. Schier, and Aviv Regev. 2015. <span>“Spatial Reconstruction of Single-Cell Gene Expression Data.”</span> <em>Nature Biotechnology</em> 33 (May): 495–502. <a href="https://doi.org/10.1038/nbt.3192">https://doi.org/10.1038/nbt.3192</a>.
</div>
<div id="ref-Satou2001" class="csl-entry">
Satou, Yutaka, Naohito Takatori, Lixy Yamada, Yasuaki Mochizuki, Makoto Hamaguchi, Hisayoshi Ishikawa, Shota Chiba, et al. 2001. <span>“Gene Expression Profiles in &amp;Lt;em&amp;gt;ciona Intestinalis&amp;lt;/Em&amp;gt; Tailbud Embryos.”</span> <em>Development</em> 128 (August): 2893 LP–2904. <a href="http://dev.biologists.org/content/128/15/2893.abstract">http://dev.biologists.org/content/128/15/2893.abstract</a>.
</div>
<div id="ref-Saviano2020" class="csl-entry">
Saviano, Antonio, Neil C. Henderson, and Thomas F. Baumert. 2020. <span>“Single-Cell Genomics and Spatial Transcriptomics: Discovery of Novel Cell States and Cellular Interactions in Liver Physiology and Disease Biology.”</span> <em>Journal of Hepatology</em> 73 (November): 1219–30. <a href="https://doi.org/10.1016/j.jhep.2020.06.004">https://doi.org/10.1016/j.jhep.2020.06.004</a>.
</div>
<div id="ref-Schede2020" class="csl-entry">
Schede, Halima Hannah, Christian Gabriel Schneider, Johanna Stergiadou, Lars E Borm, Anurag Ranjak, Tracy M Yamawaki, Fabrice P A David, et al. 2020. <span>“Spatial Tissue Profiling by Imaging-Free Molecular Tomography.”</span> <em>bioRxiv</em>, August, 2020.08.04.235655. <a href="https://doi.org/10.1101/2020.08.04.235655">https://doi.org/10.1101/2020.08.04.235655</a>.
</div>
<div id="ref-Schwaber2019" class="csl-entry">
Schwaber, Jessica, Stacey Andersen, and Lars Nielsen. 2019. <span>“Shedding Light: The Importance of Reverse Transcription Efficiency Standards in Data Interpretation.”</span> <em>Biomolecular Detection and Quantification</em> 17 (March): 100077. <a href="https://doi.org/10.1016/j.bdq.2018.12.002">https://doi.org/10.1016/j.bdq.2018.12.002</a>.
</div>
<div id="ref-Seydoux1994" class="csl-entry">
Seydoux, G, and A Fire. 1994. <span>“Soma-Germline Asymmetry in the Distributions of Embryonic RNAs in Caenorhabditis Elegans.”</span> <em>Development</em> 120 (October): 2823 LP–2834. <a href="http://dev.biologists.org/content/120/10/2823.abstract">http://dev.biologists.org/content/120/10/2823.abstract</a>.
</div>
<div id="ref-Sgroi1999" class="csl-entry">
Sgroi, Dennis C, Sarena Teng, Greg Robinson, Rebbecca LeVangie, James R Hudson, and Abdel G Elkahloun. 1999. <span>“&amp;Lt;em&amp;gt;in Vivo&amp;lt;/Em&amp;gt; Gene Expression Profile Analysis of Human Breast Cancer Progression.”</span> <em>Cancer Research</em> 59 (November): 5656 LP–5661. <a href="http://cancerres.aacrjournals.org/content/59/22/5656.abstract">http://cancerres.aacrjournals.org/content/59/22/5656.abstract</a>.
</div>
<div id="ref-Shah2016" class="csl-entry">
Shah, Sheel, Eric Lubeck, Wen Zhou, and Long Cai. 2016. <span>“In Situ Transcription Profiling of Single Cells Reveals Spatial Organization of Cells in the Mouse Hippocampus.”</span> <em>Neuron</em>. <a href="https://doi.org/10.1016/j.neuron.2016.10.001">https://doi.org/10.1016/j.neuron.2016.10.001</a>.
</div>
<div id="ref-Shah2018" class="csl-entry">
Shah, Sheel, Yodai Takei, Wen Zhou, Eric Lubeck, Jina Yun, Chee Huat Linus Eng, Noushin Koulena, et al. 2018. <span>“Dynamics and Spatial Genomics of the Nascent Transcriptome by Intron seqFISH.”</span> <em>Cell</em>. <a href="https://doi.org/10.1016/j.cell.2018.05.035">https://doi.org/10.1016/j.cell.2018.05.035</a>.
</div>
<div id="ref-Sharma2021" class="csl-entry">
Sharma, Aadya, Dibyashree Ghosh, Neha Divekar, Manisha Gore, Saikat Gochhait, and S S Shireshi. 2021. <span>“Comparing the Socio-Economic Implications of the 1918 Spanish Flu and the COVID-19 Pandemic in India: A Systematic Review of Literature.”</span> <em>International Social Science Journal</em>, March, 10.1111/issj.12266. <a href="https://doi.org/10.1111/issj.12266">https://doi.org/10.1111/issj.12266</a>.
</div>
<div id="ref-Sharma2020" class="csl-entry">
Sharma, Ankur, Justine Jia Wen Seow, Charles-Antoine Dutertre, Rhea Pai, Camille Blériot, Archita Mishra, Regina Men Men Wong, et al. 2020. <span>“Onco-Fetal Reprogramming of Endothelial Cells Drives Immunosuppressive Macrophages in Hepatocellular Carcinoma.”</span> <em>Cell</em> 183 (October): 377–394.e21. <a href="https://doi.org/10.1016/j.cell.2020.08.040">https://doi.org/10.1016/j.cell.2020.08.040</a>.
</div>
<div id="ref-Sharma2020a" class="csl-entry">
Sharma, Gaurav, Carlo Colantuoni, Loyal A Goff, Elana J Fertig, and Genevieve Stein-O’Brien. 2020. <span>“projectR: An r/Bioconductor Package for Transfer Learning via PCA, NMF, Correlation and Clustering.”</span> <em>Bioinformatics</em> 36 (June): 3592–93. <a href="https://doi.org/10.1093/bioinformatics/btaa183">https://doi.org/10.1093/bioinformatics/btaa183</a>.
</div>
<div id="ref-Shcherbatyy2014" class="csl-entry">
Shcherbatyy, Volodymyr, James Carson, Murat Yaylaoglu, Katharina Jäckle, Frauke Grabbe, Maren Brockmeyer, Halenur Yavuz, and Gregor Eichele. 2015. <span>“A Digital Atlas of Ion Channel Expression Patterns in the Two-Week-Old Rat Brain.”</span> <em>Neuroinformatics</em> 13 (January): 111–25. <a href="https://doi.org/10.1007/s12021-014-9247-0">https://doi.org/10.1007/s12021-014-9247-0</a>.
</div>
<div id="ref-Shendure2005" class="csl-entry">
Shendure, Jay. 2005. <span>“Accurate Multiplex Polony Sequencing of an Evolved Bacterial Genome.”</span> <em>Science</em> 309 (September): 1728–32. <a href="https://doi.org/10.1126/science.1117389">https://doi.org/10.1126/science.1117389</a>.
</div>
<div id="ref-Shi2020" class="csl-entry">
Shi, Hao, Qiaojuan Shi, Benjamin Grodner, Joan Sesing Lenz, Warren R Zipfel, Ilana Lauren Brito, and Iwijn De Vlaminck. 2020. <span>“Highly Multiplexed Spatial Mapping of Microbial Communities.”</span> <em>Nature</em> 588: 676–81. <a href="https://doi.org/10.1038/s41586-020-2983-4">https://doi.org/10.1038/s41586-020-2983-4</a>.
</div>
<div id="ref-Siegert2009" class="csl-entry">
Siegert, Sandra, Brigitte Gross Scherf, Karina Del Punta, Nick Didkovsky, Nathaniel Heintz, and Botond Roska. 2009. <span>“Genetic Address Book for Retinal Cell Types.”</span> <em>Nature Neuroscience</em> 12 (September): 1197–1204. <a href="https://doi.org/10.1038/nn.2370">https://doi.org/10.1038/nn.2370</a>.
</div>
<div id="ref-Singer1982" class="csl-entry">
Singer, R. H., and D. C. Ward. 1982. <span>“Actin Gene Expression Visualized in Chicken Muscle Tissue Culture by Using in Situ Hybridization with a Biotinated Nucleotide Analog.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em> 79 (December): 7331–35. <a href="https://doi.org/10.1073/pnas.79.23.7331">https://doi.org/10.1073/pnas.79.23.7331</a>.
</div>
<div id="ref-Singh2003" class="csl-entry">
Singh, Ram P., Vanessa M. Brown, Abhijit Chaudhari, Arshad H. Khan, Alex Ossadtchi, Daniel M. Sforza, A.Ken Meadors, Simon R. Cherry, Richard M. Leahy, and Desmond J. Smith. 2003. <span>“High-Resolution Voxelation Mapping of Human and Rodent Brain Gene Expression.”</span> <em>Journal of Neuroscience Methods</em> 125 (May): 93–101. <a href="https://doi.org/10.1016/S0165-0270(03)00045-1">https://doi.org/10.1016/S0165-0270(03)00045-1</a>.
</div>
<div id="ref-Sisecioglu2015" class="csl-entry">
Şişecioğlu, Melda, Harun Budak, Lars Geffers, Murat Çankaya, Mehmet Çiftci, Christina Thaller, Gregor Eichele, Ömer İrfan Küfrevioğlu, and Hasan Özdemir. 2015. <span>“A Compendium of Expression Patterns of Cholesterol Biosynthetic Enzymes in the Mouse Embryo.”</span> <em>Journal of Lipid Research</em> 56 (August): 1551–59. <a href="https://doi.org/10.1194/jlr.M059634">https://doi.org/10.1194/jlr.M059634</a>.
</div>
<div id="ref-Skarnes1995" class="csl-entry">
Skarnes, William C., Julie E. Moss, Stella M. Hurtley, and Rosa S. P. Beddington. 1995. <span>“Capturing Genes Encoding Membrane and Secreted Proteins Important for Mouse Development.”</span> <em>Proceedings of the National Academy of Sciences</em> 92 (July): 6592–96. <a href="https://doi.org/10.1073/pnas.92.14.6592">https://doi.org/10.1073/pnas.92.14.6592</a>.
</div>
<div id="ref-Skarnes2011" class="csl-entry">
Skarnes, William C., Barry Rosen, Anthony P. West, Manousos Koutsourakis, Wendy Bushell, Vivek Iyer, Alejandro O. Mujica, et al. 2011. <span>“A Conditional Knockout Resource for the Genome-Wide Study of Mouse Gene Function.”</span> <em>Nature</em> 474 (June): 337–42. <a href="https://doi.org/10.1038/nature10163">https://doi.org/10.1038/nature10163</a>.
</div>
<div id="ref-Smith2019a" class="csl-entry">
Smith, Constance M, Terry F. Hayamizu, Jacqueline H. Finger, Susan M. Bello, Ingeborg J. McCright, Jingxia Xu, Richard M. Baldarelli, et al. 2019. <span>“The Mouse Gene Expression Database (GXD): 2019 Update.”</span> <em>Nucleic Acids Research</em> 47 (January): D774–79. <a href="https://doi.org/10.1093/nar/gky922">https://doi.org/10.1093/nar/gky922</a>.
</div>
<div id="ref-Sountoulidis2020" class="csl-entry">
Sountoulidis, Alexandros, Andreas Liontos, Hong Phuong Nguyen, Alexandra B Firsova, Athanasios Fysikopoulos, Xiaoyan Qian, Werner Seeger, Erik Sundström, Mats Nilsson, and Christos Samakovlis. 2020. <span>“SCRINSHOT, a Spatial Method for Single-Cell Resolution Mapping of Cell States in Tissue Sections.”</span> <em>bioRxiv</em>, February, 2020.02.07.938571. <a href="https://doi.org/10.1101/2020.02.07.938571">https://doi.org/10.1101/2020.02.07.938571</a>.
</div>
<div id="ref-ncl" class="csl-entry">
<span>“SpaRTAN.”</span> n.d. <a href="https://www.ncl.ac.uk/gcf/spatiallyresolvedtranscriptomics/">https://www.ncl.ac.uk/gcf/spatiallyresolvedtranscriptomics/</a>.
</div>
<div id="ref-Spradling1982" class="csl-entry">
Spradling, A., and G. Rubin. 1982. <span>“Transposition of Cloned p Elements into Drosophila Germ Line Chromosomes.”</span> <em>Science</em> 218 (October): 341–47. <a href="https://doi.org/10.1126/science.6289435">https://doi.org/10.1126/science.6289435</a>.
</div>
<div id="ref-Sprague2003" class="csl-entry">
Sprague, Judy. 2003. <span>“The Zebrafish Information Network (ZFIN): The Zebrafish Model Organism Database.”</span> <em>Nucleic Acids Research</em> 31 (January): 241–43. <a href="https://doi.org/10.1093/nar/gkg027">https://doi.org/10.1093/nar/gkg027</a>.
</div>
<div id="ref-Srivastava2019" class="csl-entry">
Srivastava, Avi, Laraib Malik, Tom Smith, Ian Sudbery, and Rob Patro. 2019. <span>“Alevin Efficiently Estimates Accurate Gene Abundances from dscRNA-Seq Data.”</span> <em>Genome Biology</em> 20 (December): 65. <a href="https://doi.org/10.1186/s13059-019-1670-y">https://doi.org/10.1186/s13059-019-1670-y</a>.
</div>
<div id="ref-Srivatsan2021" class="csl-entry">
Srivatsan, Sanjay R, Mary C Regier, Eliza Barkan, Jennifer M Franks, Jonathan S Packer, Parker Grosjean, Madeleine Duran, et al. 2021. <span>“Embryo-Scale, Single-Cell Spatial Transcriptomics.”</span> <em>Science</em> 373: 111–17. <a href="https://doi.org/10.1126/science.abb9536">https://doi.org/10.1126/science.abb9536</a>.
</div>
<div id="ref-Stahl2016a" class="csl-entry">
Ståhl, Patrik L., Fredrik Salmén, Sanja Vickovic, Anna Lundmark, José Fernández Navarro, Jens Magnusson, Stefania Giacomello, et al. 2016. <span>“Visualization and Analysis of Gene Expression in Tissue Sections by Spatial Transcriptomics.”</span> <em>Science</em> 353 (July): 78–82. <a href="https://doi.org/10.1126/science.aaf2403">https://doi.org/10.1126/science.aaf2403</a>.
</div>
<div id="ref-Stanford1998" class="csl-entry">
Stanford, William L., Georgina Caruana, Katherine A. Vallis, Maneesha Inamdar, Michihiro Hidaka, Victoria L. Bautch, and Alan Bernstein. 1998. <span>“Expression Trapping: Identification of Novel Genes Expressed in Hematopoietic and Endothelial Lineages by Gene Trapping in ES Cells.”</span> <em>Blood</em> 92 (December): 4622–31. <a href="https://doi.org/10.1182/blood.V92.12.4622">https://doi.org/10.1182/blood.V92.12.4622</a>.
</div>
<div id="ref-Stanford2001" class="csl-entry">
Stanford, William L., Jason B. Cohn, and Sabine P. Cordes. 2001. <span>“Gene-Trap Mutagenesis: Past, Present and Beyond.”</span> <em>Nature Reviews Genetics</em> 2 (October): 756–68. <a href="https://doi.org/10.1038/35093548">https://doi.org/10.1038/35093548</a>.
</div>
<div id="ref-Stapleton2002" class="csl-entry">
Stapleton, Mark. 2002. <span>“The Drosophila Gene Collection: Identification of Putative Full-Length cDNAs for 70.”</span> <em>Genome Research</em> 12 (August): 1294–1300. <a href="https://doi.org/10.1101/gr.269102">https://doi.org/10.1101/gr.269102</a>.
</div>
<div id="ref-Stickels2020" class="csl-entry">
Stickels, Robert, Evan Murray, Pawan Kumar, Jilong Li, Jamie Marshall, Daniela Di Bella, Paola Arlotta, Evan Macosko, and Fei Chen. 2020. <span>“Sensitive Spatial Genome Wide Expression Profiling at Cellular Resolution.”</span> <em>bioRxiv</em>, March, 2020.03.12.989806. <a href="https://doi.org/10.1101/2020.03.12.989806">https://doi.org/10.1101/2020.03.12.989806</a>.
</div>
<div id="ref-Stoeger2015" class="csl-entry">
Stoeger, Thomas, Nico Battich, Markus D. Herrmann, Yauhen Yakimovich, and Lucas Pelkmans. 2015. <span>“Computer Vision for Image-Based Transcriptomics.”</span> <em>Methods</em> 85 (September): 44–53. <a href="https://doi.org/10.1016/j.ymeth.2015.05.016">https://doi.org/10.1016/j.ymeth.2015.05.016</a>.
</div>
<div id="ref-Strell2019" class="csl-entry">
Strell, Carina, Markus M. Hilscher, Navya Laxman, Jessica Svedlund, Chenglin Wu, Chika Yokota, and Mats Nilsson. 2019. <span>“Placing RNA in Context and Space – Methods for Spatially Resolved Transcriptomics.”</span> <em>FEBS Journal</em>. <a href="https://doi.org/10.1111/febs.14435">https://doi.org/10.1111/febs.14435</a>.
</div>
<div id="ref-Stringer2020" class="csl-entry">
Stringer, Carsen, Tim Wang, Michalis Michaelos, and Marius Pachitariu. 2020. <span>“Cellpose: A Generalist Algorithm for Cellular Segmentation.”</span> <em>Nature Methods 2020 18:1</em> 18 (December): 100–106. <a href="https://doi.org/10.1038/s41592-020-01018-x">https://doi.org/10.1038/s41592-020-01018-x</a>.
</div>
<div id="ref-Stuart2019a" class="csl-entry">
Stuart, Tim, Andrew Butler, Paul Hoffman, Christoph Hafemeister, Efthymia Papalexi, William M. Mauck, Yuhan Hao, Marlon Stoeckius, Peter Smibert, and Rahul Satija. 2019. <span>“Comprehensive Integration of Single-Cell Data.”</span> <em>Cell</em> 177 (June): 1888–1902.e21. <a href="https://doi.org/10.1016/j.cell.2019.05.031">https://doi.org/10.1016/j.cell.2019.05.031</a>.
</div>
<div id="ref-Su2020" class="csl-entry">
Su, Jing, and Qianqian Song. 2020. <span>“DSTG: Deconvoluting Spatial Transcriptomics Data Through Graph-Based Artificial Intelligence.”</span> <em>bioRxiv</em>, January, 2020.10.20.347195. <a href="https://doi.org/10.1101/2020.10.20.347195">https://doi.org/10.1101/2020.10.20.347195</a>.
</div>
<div id="ref-Sulston1992" class="csl-entry">
Sulston, J., Z. Du, K. Thomas, R. Wilson, L. Hillier, R. Staden, N. Halloran, et al. 1992. <span>“The c. Elegans Genome Sequencing Project: A Beginning.”</span> <em>Nature</em> 356: 37–41. <a href="https://doi.org/10.1038/356037a0">https://doi.org/10.1038/356037a0</a>.
</div>
<div id="ref-Sun2021" class="csl-entry">
Sun, Dongqing, Yihan Xiao, Zhaoyang Liu, Taiwen Li, Qiu Wu, and Chenfei Wang. 2021. <span>“STRIDE: Accurately Decomposing and Integrating Spatial Transcriptomics Using Single-Cell RNA Sequencing.”</span> <em>bioRxiv</em>, January, 2021.09.08.459458. <a href="https://doi.org/10.1101/2021.09.08.459458">https://doi.org/10.1101/2021.09.08.459458</a>.
</div>
<div id="ref-Sun2013" class="csl-entry">
Sun, Qian, Sherin Muckatira, Lei Yuan, Shuiwang Ji, Stuart Newfeld, Sudhir Kumar, and Jieping Ye. 2013. <span>“Image-Level and Group-Level Models for Drosophila Gene Expression Pattern Annotation.”</span> <em>BMC Bioinformatics</em> 14 (December): 350. <a href="https://doi.org/10.1186/1471-2105-14-350">https://doi.org/10.1186/1471-2105-14-350</a>.
</div>
<div id="ref-Sun2020" class="csl-entry">
Sun, Shiquan, Jiaqiang Zhu, and Xiang Zhou. 2020. <span>“Statistical Analysis of Spatial Expression Patterns for Spatially Resolved Transcriptomic Studies.”</span> <em>Nature Methods</em> 17 (February): 193–200. <a href="https://doi.org/10.1038/s41592-019-0701-7">https://doi.org/10.1038/s41592-019-0701-7</a>.
</div>
<div id="ref-Sun2020a" class="csl-entry">
Sun, Yu-Chi, Xiaoyin Chen, Stephan Fischer, Shaina Lu, Jesse Gillis, and Anthony M Zador. 2020. <span>“Integrating Barcoded Neuroanatomy with Spatial Transcriptional Profiling Reveals Cadherin Correlates of Projections Shared Across the Cortex.”</span> <em>bioRxiv</em>, January, 2020.08.25.266460. <a href="https://doi.org/10.1101/2020.08.25.266460">https://doi.org/10.1101/2020.08.25.266460</a>.
</div>
<div id="ref-Sundaresan1995" class="csl-entry">
Sundaresan, Venkatesan, Patricia Springer, Thomas Volpe, Samuel Haward, Jonathan D. G. Jones, Caroline Dean, Hong Ma, and Robert Martienssen. 1995. <span>“Patterns of Gene Action in Plant Development Revealed by Enhancer Trap and Gene Trap Transposable Elements.”</span> <em>Genes &amp; Development</em> 9 (July): 1797–1810. <a href="https://doi.org/10.1101/gad.9.14.1797">https://doi.org/10.1101/gad.9.14.1797</a>.
</div>
<div id="ref-Svensson2018" class="csl-entry">
Svensson, Valentine, Sarah A. Teichmann, and Oliver Stegle. 2018. <span>“SpatialDE: Identification of Spatially Variable Genes.”</span> <em>Nature Methods</em> 15 (April): 343–46. <a href="https://doi.org/10.1038/nmeth.4636">https://doi.org/10.1038/nmeth.4636</a>.
</div>
<div id="ref-Svensson2020" class="csl-entry">
Svensson, Valentine, Eduardo da Veiga Beltrame, and Lior Pachter. 2020. <span>“A Curated Database Reveals Trends in Single-Cell Transcriptomics.”</span> <em>Database</em> 2020 (January). <a href="https://doi.org/10.1093/database/baaa073">https://doi.org/10.1093/database/baaa073</a>.
</div>
<div id="ref-Takei2020" class="csl-entry">
Takei, Yodai, Jina Yun, Noah Ollikainen, Shiwei Zheng, Nico Pierson, Jonathan White, Sheel Shah, et al. 2020. <span>“Global Architecture of the Nucleus in Single Cells by DNA seqFISH+ and Multiplexed Immunofluorescence.”</span> <em>bioRxiv</em>, January, 2020.11.29.403055. <a href="https://doi.org/10.1101/2020.11.29.403055">https://doi.org/10.1101/2020.11.29.403055</a>.
</div>
<div id="ref-Tanevski2020" class="csl-entry">
Tanevski, Jovan, Thin Nguyen, Buu Truong, Nikos Karaiskos, Mehmet Eren Ahsen, Xinyu Zhang, Chang Shu, et al. 2020. <span>“Gene Selection for Optimal Prediction of Cell Position in Tissues from Single-Cell Transcriptomics Data.”</span> <em>Life Science Alliance</em> 3 (November): e202000867. <a href="https://doi.org/10.26508/lsa.202000867">https://doi.org/10.26508/lsa.202000867</a>.
</div>
<div id="ref-Tang2009" class="csl-entry">
Tang, Fuchou, Catalin Barbacioru, Yangzhou Wang, Ellen Nordman, Clarence Lee, Nanlan Xu, Xiaohui Wang, et al. 2009. <span>“mRNA-Seq Whole-Transcriptome Analysis of a Single Cell.”</span> <em>Nature Methods</em> 6: 377–82. <a href="https://doi.org/10.1038/nmeth.1315">https://doi.org/10.1038/nmeth.1315</a>.
</div>
<div id="ref-Tang2011" class="csl-entry">
Tang, Fuchou, Kaiqin Lao, and M Azim Surani. 2011. <span>“Development and Applications of Single-Cell Transcriptome Analysis.”</span> <em>Nature Methods</em> 8 (April): S6–11. <a href="https://doi.org/10.1038/nmeth.1557">https://doi.org/10.1038/nmeth.1557</a>.
</div>
<div id="ref-Tassy2010" class="csl-entry">
Tassy, Olivier, Delphine Dauga, Fabrice Daian, Daniel Sobral, François Robin, Pierre Khoueiry, David Salgado, et al. 2010. <span>“The ANISEED Database: Digital Representation, Formalization, and Elucidation of a Chordate Developmental Program.”</span> <em>Genome Research</em> 20 (October): 1459–68. <a href="https://doi.org/10.1101/gr.108175.110">https://doi.org/10.1101/gr.108175.110</a>.
</div>
<div id="ref-Tautz1989" class="csl-entry">
Tautz, Diethard, and Christine Pfeifle. 1989. <span>“A Non-Radioactive in Situ Hybridization Method for the Localization of Specific RNAs in Drosophila Embryos Reveals Translational Control of the Segmentation Gene Hunchback.”</span> <em>Chromosoma</em> 98 (August): 81–85. <a href="https://doi.org/10.1007/BF00291041">https://doi.org/10.1007/BF00291041</a>.
</div>
<div id="ref-Tecott1988" class="csl-entry">
Tecott, L H, J D Barchas, and J H Eberwine. 1988. <span>“In Situ Transcription: Specific Synthesis of Complementary DNA in Fixed Tissue Sections.”</span> <em>Science</em> 240 (June): 1661 LP–1664. <a href="https://doi.org/10.1126/science.2454508">https://doi.org/10.1126/science.2454508</a>.
</div>
<div id="ref-Thompson2010" class="csl-entry">
Thompson, Carol, Jonathan Wisor, Chang-Kyu Lee, Sayan Pathak, Dmitry Gerashchenko, Kimberly Smith, Shanna Fischer, et al. 2010. <span>“Molecular and Anatomical Signatures of Sleep Deprivation in the Mouse Brain.”</span> <em>Frontiers in Neuroscience</em>. <a href="https://www.frontiersin.org/article/10.3389/fnins.2010.00165">https://www.frontiersin.org/article/10.3389/fnins.2010.00165</a>.
</div>
<div id="ref-Thrane2018" class="csl-entry">
Thrane, Kim, Hanna Eriksson, Jonas Maaskola, Johan Hansson, and Joakim Lundeberg. 2018. <span>“Spatially Resolved Transcriptomics Enables Dissection of Genetic Heterogeneity in Stage III Cutaneous Malignant Melanoma.”</span> <em>Cancer Research</em>. <a href="https://doi.org/10.1158/0008-5472.CAN-18-0747">https://doi.org/10.1158/0008-5472.CAN-18-0747</a>.
</div>
<div id="ref-Thut2001" class="csl-entry">
Thut, Catherine J., Ryan B. Rountree, Michael Hwa, and David M. Kingsley. 2001. <span>“A Large-Scale in Situ Screen Provides Molecular Evidence for the Induction of Eye Anterior Segment Structures by the Developing Lens.”</span> <em>Developmental Biology</em> 231 (March): 63–76. <a href="https://doi.org/10.1006/dbio.2000.0140">https://doi.org/10.1006/dbio.2000.0140</a>.
</div>
<div id="ref-Tietjen2003" class="csl-entry">
Tietjen, Ian, Jason M Rihel, Yanxiang Cao, Georgy Koentges, Lisa Zakhary, and Catherine Dulac. 2003. <span>“Single-Cell Transcriptional Analysis of Neuronal Progenitors.”</span> <em>Neuron</em> 38: 161–75. https://doi.org/<a href="https://doi.org/10.1016/S0896-6273(03)00229-0">https://doi.org/10.1016/S0896-6273(03)00229-0</a>.
</div>
<div id="ref-Titlow2022" class="csl-entry">
Titlow, Joshua S, Maria Kiourlappou, Ana Palanca, Jeffrey Y Lee, Dalia S Gala, Darragh Ennis, Joyce J S Yu, et al. 2022. <span>“Systematic Analysis of YFP Gene Traps Reveals Common Discordance Between mRNA and Protein Across the Nervous System.”</span> <em>bioRxiv</em>, March, 2022.03.21.485142. <a href="https://doi.org/10.1101/2022.03.21.485142">https://doi.org/10.1101/2022.03.21.485142</a>.
</div>
<div id="ref-Tomancak2002" class="csl-entry">
Tomancak, Pavel, Amy Beaton, Richard Weiszmann, Elaine Kwan, Sheng Qiang Shu, Suzanna E. Lewis, Stephen Richards, et al. 2002. <span>“Systematic Determination of Patterns of Gene Expression During Drosophila Embryogenesis.”</span> <em>Genome Biology</em> 3 (December): research0088.1. <a href="https://doi.org/10.1186/gb-2002-3-12-research0088">https://doi.org/10.1186/gb-2002-3-12-research0088</a>.
</div>
<div id="ref-Tomancak2007" class="csl-entry">
Tomancak, Pavel, Benjamin P. Berman, Amy Beaton, Richard Weiszmann, Elaine Kwan, Volker Hartenstein, Susan E. Celniker, and Gerald M. Rubin. 2007. <span>“Global Analysis of Patterns of Gene Expression During Drosophila Embryogenesis.”</span> <em>Genome Biology</em> 8 (July): R145. <a href="https://doi.org/10.1186/gb-2007-8-7-r145">https://doi.org/10.1186/gb-2007-8-7-r145</a>.
</div>
<div id="ref-TPCL" class="csl-entry">
<span>“Translational Pathology Core Laboratory (TPCL).”</span> n.d. <a href="https://www.uclahealth.org/pathology/tpcl-services">https://www.uclahealth.org/pathology/tpcl-services</a>.
</div>
<div id="ref-Tripodo2020" class="csl-entry">
Tripodo, Claudio, Federica Zanardi, Fabio Iannelli, Saveria Mazzara, Mariella Vegliante, Gaia Morello, Arianna Di Napoli, et al. 2020. <span>“A Spatially Resolved Dark- Versus Light-Zone Microenvironment Signature Subdivides Germinal Center-Related Aggressive b Cell Lymphomas.”</span> <em>iScience</em> 23 (October). <a href="https://doi.org/10.1016/j.isci.2020.101562">https://doi.org/10.1016/j.isci.2020.101562</a>.
</div>
<div id="ref-Tuck2015" class="csl-entry">
Tuck, Elizabeth, Jeanne Estabel, Anika Oellrich, A. K. Maguire, Hibret A. Adissu, Luke Souter, Emma Siragher, et al. 2015. <span>“A Gene Expression Resource Generated by Genome-Wide lacZ Profiling in the Mouse.”</span> <em>Disease Models &amp; Mechanisms</em> 8 (November): 1467–78. <a href="https://doi.org/10.1242/dmm.021238">https://doi.org/10.1242/dmm.021238</a>.
</div>
<div id="ref-Tzur2018" class="csl-entry">
Tzur, Yonatan B., Eitan Winter, Jinmin Gao, Tamar Hashimshony, Itai Yanai, and Monica P. Colaiácovo. 2018. <span>“Spatiotemporal Gene Expression Analysis of the Caenorhabditis Elegans Germline Uncovers a Syncytial Expression Switch.”</span> <em>Genetics</em> 210 (October): 587–605. <a href="https://doi.org/10.1534/genetics.118.301315">https://doi.org/10.1534/genetics.118.301315</a>.
</div>
<div id="ref-Urdea1993" class="csl-entry">
Urdea, M. S. 1993. <span>“Synthesis and Characterization of Branched DNA (bDNA) for the Direct and Quantitative Detection of CMV, HBV, HCV, and HIV.”</span> <em>Clinical Chemistry</em> 39 (April): 725–26. <a href="https://doi.org/10.1093/clinchem/39.4.725">https://doi.org/10.1093/clinchem/39.4.725</a>.
</div>
<div id="ref-VanValen2016" class="csl-entry">
Valen, David A Van, Takamasa Kudo, Keara M Lane, Derek N Macklin, Nicolas T Quach, Mialy M DeFelice, Inbal Maayan, Yu Tanouchi, Euan A Ashley, and Markus W Covert. 2016. <span>“Deep Learning Automates the Quantitative Analysis of Individual Cells in Live-Cell Imaging Experiments.”</span> <em>PLOS Computational Biology</em> 12 (November): e1005177. <a href="https://doi.org/10.1371/journal.pcbi.1005177">https://doi.org/10.1371/journal.pcbi.1005177</a>.
</div>
<div id="ref-Valoczi2004" class="csl-entry">
Valoczi, A. 2004. <span>“Sensitive and Specific Detection of microRNAs by Northern Blot Analysis Using LNA-Modified Oligonucleotide Probes.”</span> <em>Nucleic Acids Research</em> 32 (December): e175–75. <a href="https://doi.org/10.1093/nar/gnh171">https://doi.org/10.1093/nar/gnh171</a>.
</div>
<div id="ref-Vandenbon2020" class="csl-entry">
Vandenbon, Alexis, and Diego Diez. 2020. <span>“A Clustering-Independent Method for Finding Differentially Expressed Genes in Single-Cell Transcriptome Data.”</span> <em>Nature Communications</em> 11: 4318. <a href="https://doi.org/10.1038/s41467-020-17900-3">https://doi.org/10.1038/s41467-020-17900-3</a>.
</div>
<div id="ref-Venter2001" class="csl-entry">
Venter, J. Craig, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, et al. 2001. <span>“The Sequence of the Human Genome.”</span> <em>Science</em> 291 (February): 1304–51. <a href="https://doi.org/10.1126/science.1058040">https://doi.org/10.1126/science.1058040</a>.
</div>
<div id="ref-Veritas" class="csl-entry">
<span>“Veritas Laser Capture Microdissection (LCM) and Laser Cutting System from Applied Biosystems.”</span> n.d. <a href="https://www.unthsc.edu/research/flow-cytometry-and-laser-capture-microdissection-core-facility/beckman-coulter-cytomics-fc500-flow-cytometry-analyzer/veritas-laser-capture">https://www.unthsc.edu/research/flow-cytometry-and-laser-capture-microdissection-core-facility/beckman-coulter-cytomics-fc500-flow-cytometry-analyzer/veritas-laser-capture</a>.
</div>
<div id="ref-Verma2020" class="csl-entry">
Verma, Archit, and Barbara Engelhardt. 2020. <span>“A Bayesian Nonparametric Semi-Supervised Model for Integration of Multiple Single-Cell Experiments.”</span> <em>bioRxiv</em>, January, 2020.01.14.906313. <a href="https://doi.org/10.1101/2020.01.14.906313">https://doi.org/10.1101/2020.01.14.906313</a>.
</div>
<div id="ref-Vickovic2019" class="csl-entry">
Vickovic, Sanja, Gökcen Eraslan, Fredrik Salmén, Johanna Klughammer, Linnea Stenbeck, Denis Schapiro, Tarmo Äijö, et al. 2019. <span>“High-Definition Spatial Transcriptomics for in Situ Tissue Profiling.”</span> <em>Nature Methods</em>. <a href="https://doi.org/10.1038/s41592-019-0548-y">https://doi.org/10.1038/s41592-019-0548-y</a>.
</div>
<div id="ref-Vickovic2020" class="csl-entry">
Vickovic, Sanja, Britta Lötstedt, Johanna Klughammer, Åsa Segerstolpe, Orit Rozenblatt-Rosen, and Aviv Regev. 2020. <span>“SM-Omics: An Automated Platform for High-Throughput Spatial Multi-Omics.”</span> <em>bioRxiv</em>, January, 2020.10.14.338418. <a href="https://doi.org/10.1101/2020.10.14.338418">https://doi.org/10.1101/2020.10.14.338418</a>.
</div>
<div id="ref-Victora2010" class="csl-entry">
Victora, Gabriel D, Tanja A Schwickert, David R Fooksman, Alice O Kamphorst, Michael Meyer-Hermann, Michael L Dustin, and Michel C Nussenzweig. 2010. <span>“Germinal Center Dynamics Revealed by Multiphoton Microscopy with a Photoactivatable Fluorescent Reporter.”</span> <em>Cell</em> 143 (November): 592–605. <a href="https://doi.org/10.1016/j.cell.2010.10.032">https://doi.org/10.1016/j.cell.2010.10.032</a>.
</div>
<div id="ref-Villacampa2020" class="csl-entry">
Villacampa, Eva Gracia, Ludvig Larsson, Linda Kvastad, Alma Andersson, Joseph Carlson, and Joakim Lundeberg. 2020. <span>“Genome-Wide Spatial Expression Profiling in FFPE Tissues.”</span> <em>bioRxiv</em>, July, 2020.07.24.219758. <a href="https://doi.org/10.1101/2020.07.24.219758">https://doi.org/10.1101/2020.07.24.219758</a>.
</div>
<div id="ref-Visel2013" class="csl-entry">
Visel, Axel, Leila Taher, Hani Girgis, Dalit May, Olga Golonzhka, Renee V. Hoch, Gabriel L. McKinsey, et al. 2013. <span>“A High-Resolution Enhancer Atlas of the Developing Telencephalon.”</span> <em>Cell</em> 152 (February): 895–908. <a href="https://doi.org/10.1016/j.cell.2012.12.041">https://doi.org/10.1016/j.cell.2012.12.041</a>.
</div>
<div id="ref-Vu2021" class="csl-entry">
Vu, Tam, Alexander Vallmitjana, Joshua Gu, Kieu La, Qi Xu, Jesus Flores, Jan Zimak, et al. 2021. <span>“Spatial Transcriptomics Using Combinatorial Fluorescence Spectral and Lifetime Encoding, Imaging and Analysis.”</span> <em>bioRxiv</em>, January, 2021.06.22.449468. <a href="https://doi.org/10.1101/2021.06.22.449468">https://doi.org/10.1101/2021.06.22.449468</a>.
</div>
<div id="ref-Waldhaus2015" class="csl-entry">
Waldhaus, Jörg, Robert Durruthy-Durruthy, and Stefan Heller. 2015. <span>“Quantitative High-Resolution Cellular Map of the Organ of Corti.”</span> <em>Cell Reports</em> 11 (June): 1385–99. <a href="https://doi.org/10.1016/j.celrep.2015.04.062">https://doi.org/10.1016/j.celrep.2015.04.062</a>.
</div>
<div id="ref-Wang2019" class="csl-entry">
Wang, Chong, Tian Lu, George Emanuel, Hazen P. Babcock, and Xiaowei Zhuang. 2019. <span>“Imaging-Based Pooled CRISPR Screening Reveals Regulators of lncRNA Localization.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1903808116">https://doi.org/10.1073/pnas.1903808116</a>.
</div>
<div id="ref-Wang2012" class="csl-entry">
Wang, Fay, John Flanagan, Nan Su, Li-Chong Wang, Son Bui, Allissa Nielson, Xingyong Wu, Hong-Thuy Vo, Xiao-Jun Ma, and Yuling Luo. 2012. <span>“RNAscope.”</span> <em>The Journal of Molecular Diagnostics</em> 14 (January): 22–29. <a href="https://doi.org/10.1016/j.jmoldx.2011.08.002">https://doi.org/10.1016/j.jmoldx.2011.08.002</a>.
</div>
<div id="ref-Wang2020" class="csl-entry">
Wang, Guiping, Cheen-Euong Ang, Jean Fan, Andrew Wang, Jeffrey R Moffitt, and Xiaowei Zhuang. 2020. <span>“Spatial Organization of the Transcriptome in Individual Neurons.”</span> <em>bioRxiv</em>, December, 2020.12.07.414060.
</div>
<div id="ref-Wang2018" class="csl-entry">
Wang, Guiping, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2018. <span>“Multiplexed Imaging of High-Density Libraries of RNAs with MERFISH and Expansion Microscopy.”</span> <em>Scientific Reports</em>. <a href="https://doi.org/10.1038/s41598-018-22297-7">https://doi.org/10.1038/s41598-018-22297-7</a>.
</div>
<div id="ref-Wang2022" class="csl-entry">
Wang, Mingyue, Qinan Hu, Tianhang Lv, Yuhang Wang, Qing Lan, Rong Xiang, Zhencheng Tu, et al. 2022. <span>“High-Resolution 3D Spatiotemporal Transcriptomic Maps of Developing Drosophila Embryos and Larvae.”</span> <em>Developmental Cell</em> 57 (May): 1271–1283.e4. <a href="https://doi.org/10.1016/J.DEVCEL.2022.04.006">https://doi.org/10.1016/J.DEVCEL.2022.04.006</a>.
</div>
<div id="ref-Wang2018b" class="csl-entry">
Wang, Xiao, William E. Allen, Matthew A. Wright, Emily L. Sylwestrak, Nikolay Samusik, Sam Vesuna, Kathryn Evans, et al. 2018. <span>“Three-Dimensional Intact-Tissue Sequencing of Single-Cell Transcriptional States.”</span> <em>Science</em> 361 (July): eaat5691. <a href="https://doi.org/10.1126/science.aat5691">https://doi.org/10.1126/science.aat5691</a>.
</div>
<div id="ref-Wang2021" class="csl-entry">
Wang, Yuhan, Mark Eddison, Greg Fleishman, Martin Weigert, Shengjin Xu, Tim Wang, Konrad Rokicki, et al. 2021. <span>“EASI-FISH for Thick Tissue Defines Lateral Hypothalamus Spatio-Molecular Organization.”</span> <em>Cell</em> 184 (December): 6361–6377.e24.
</div>
<div id="ref-Waterston2002" class="csl-entry">
Waterston, Robert H., Kerstin Lindblad-Toh, Ewan Birney, Jane Rogers, Josep F. Abril, Pankaj Agarwal, Richa Agarwala, et al. 2002. <span>“Initial Sequencing and Comparative Analysis of the Mouse Genome.”</span> <em>Nature</em> 420 (December): 520–62. <a href="https://doi.org/10.1038/nature01262">https://doi.org/10.1038/nature01262</a>.
</div>
<div id="ref-Waylen2020" class="csl-entry">
Waylen, Lisa N, Hieu T Nim, Luciano G Martelotto, and Mirana Ramialison. 2020. <span>“From Whole-Mount to Single-Cell Spatial Assessment of Gene Expression in 3D.”</span> <em>Communications Biology</em> 3: 602. <a href="https://doi.org/10.1038/s42003-020-01341-1">https://doi.org/10.1038/s42003-020-01341-1</a>.
</div>
<div id="ref-Weinstein2019" class="csl-entry">
Weinstein, Joshua A., Aviv Regev, and Feng Zhang. 2019. <span>“DNA Microscopy: Optics-Free Spatio-Genetic Imaging by a Stand-Alone Chemical Reaction.”</span> <em>Cell</em> 178 (June): 229–241.e16. <a href="https://doi.org/10.1016/j.cell.2019.05.019">https://doi.org/10.1016/j.cell.2019.05.019</a>.
</div>
<div id="ref-Welch2019" class="csl-entry">
Welch, Joshua D., Velina Kozareva, Ashley Ferreira, Charles Vanderburg, Carly Martin, and Evan Z. Macosko. 2019. <span>“Single-Cell Multi-Omic Integration Compares and Contrasts Features of Brain Cell Identity.”</span> <em>Cell</em> 177 (June): 1873–1887.e17. <a href="https://doi.org/10.1016/j.cell.2019.05.006">https://doi.org/10.1016/j.cell.2019.05.006</a>.
</div>
<div id="ref-West2015" class="csl-entry">
West, David B., Ravi K. Pasumarthi, Brian Baridon, Esi Djan, Amanda Trainor, Stephen M. Griffey, Eric K. Engelhard, et al. 2015. <span>“A lacZ Reporter Gene Expression Atlas for 313 Adult KOMP Mutant Mouse Lines.”</span> <em>Genome Research</em> 25 (April): 598–607. <a href="https://doi.org/10.1101/gr.184184.114">https://doi.org/10.1101/gr.184184.114</a>.
</div>
<div id="ref-Westerfield1997" class="csl-entry">
Westerfield, Monte, Eckehard Doerry, Arthur E. Kirkpatrick, Wolfgang Driever, and Sarah A. Douglas. 1997. <span>“An on-Line Database for Zebrafish Development and Genetics Research.”</span> <em>Seminars in Cell &amp; Developmental Biology</em> 8 (October): 477–88. <a href="https://doi.org/10.1006/scdb.1997.0173">https://doi.org/10.1006/scdb.1997.0173</a>.
</div>
<div id="ref-White2013" class="csl-entry">
White, Jacqueline K., Anna-Karin Gerdin, Natasha A. Karp, Ed Ryder, Marija Buljan, James N. Bussell, Jennifer Salisbury, et al. 2013. <span>“Genome-Wide Generation and Systematic Phenotyping of Knockout Mice Reveals New Roles for Many Genes.”</span> <em>Cell</em> 154 (July): 452–64. <a href="https://doi.org/10.1016/j.cell.2013.06.022">https://doi.org/10.1016/j.cell.2013.06.022</a>.
</div>
<div id="ref-Wienholds2005" class="csl-entry">
Wienholds, Erno. 2005. <span>“MicroRNA Expression in Zebrafish Embryonic Development.”</span> <em>Science</em> 309 (July): 310–11. <a href="https://doi.org/10.1126/science.1114519">https://doi.org/10.1126/science.1114519</a>.
</div>
<div id="ref-Wilson1989" class="csl-entry">
Wilson, Clive, Rebecca Kurth Pearson, Hugo J Bellen, C J O’Kane, Ueli Grossniklaus, and Walter J Gehring. 1989. <span>“P-Element-Mediated Enhancer Detection: An Efficient Method for Isolating and Characterizing Developmentally Regulated Genes in Drosophila.”</span> <em>Genes &amp; Development</em> 3 (September): 1301–13. <a href="https://doi.org/10.1101/gad.3.9.1301">https://doi.org/10.1101/gad.3.9.1301</a>.
</div>
<div id="ref-Wu2016" class="csl-entry">
Wu, Siqi, Antony Joseph, Ann S. Hammonds, Susan E. Celniker, Bin Yu, and Erwin Frise. 2016. <span>“Stability-Driven Nonnegative Matrix Factorization to Interpret Spatial Gene Expression and Build Local Gene Networks.”</span> <em>Proceedings of the National Academy of Sciences</em> 113 (April): 4290–95. <a href="https://doi.org/10.1073/pnas.1521171113">https://doi.org/10.1073/pnas.1521171113</a>.
</div>
<div id="ref-Wu2021" class="csl-entry">
Wu, Sunny Z, Ghamdan Al-Eryani, Daniel Lee Roden, Simon Junankar, Kate Harvey, Alma Andersson, Aatish Thennavan, et al. 2021. <span>“A Single-Cell and Spatially Resolved Atlas of Human Breast Cancers.”</span> <em>Nature Genetics</em> 53: 1334–47. <a href="https://doi.org/10.1038/s41588-021-00911-1">https://doi.org/10.1038/s41588-021-00911-1</a>.
</div>
<div id="ref-Wurst1995a" class="csl-entry">
Wurst, W, J Rossant, V Prideaux, M Kownacka, A Joyner, D P Hill, F Guillemot, S Gasca, D Cado, and A Auerbach. 1995. <span>“A Large-Scale Gene-Trap Screen for Insertional Mutations in Developmentally Regulated Genes in Mice.”</span> <em>Genetics</em> 139. <a href="https://www.genetics.org/content/139/2/889">https://www.genetics.org/content/139/2/889</a>.
</div>
<div id="ref-XDB3" class="csl-entry">
<span>“XDB3.”</span> 2004. <a href="http://xenopus.nibb.ac.jp/">http://xenopus.nibb.ac.jp/</a>.
</div>
<div id="ref-Xia2019c" class="csl-entry">
Xia, Chenglong, Hazen P. Babcock, Jeffrey R. Moffitt, and Xiaowei Zhuang. 2019. <span>“Multiplexed Detection of RNA Using MERFISH and Branched DNA Amplification.”</span> <em>Scientific Reports</em> 9 (December): 1–13. <a href="https://doi.org/10.1038/s41598-019-43943-8">https://doi.org/10.1038/s41598-019-43943-8</a>.
</div>
<div id="ref-Xia2019a" class="csl-entry">
Xia, Chenglong, Jean Fan, George Emanuel, Junjie Hao, and Xiaowei Zhuang. 2019. <span>“Spatial Transcriptome Profiling by MERFISH Reveals Subcellular RNA Compartmentalization and Cell Cycle-Dependent Gene Expression.”</span> <em>Proceedings of the National Academy of Sciences of the United States of America</em>. <a href="https://doi.org/10.1073/pnas.1912459116">https://doi.org/10.1073/pnas.1912459116</a>.
</div>
<div id="ref-Xu2020" class="csl-entry">
Xu, Xiangmin, Todd C Holmes, Min-Hua Luo, Kevin T Beier, Gregory D Horwitz, Fei Zhao, Wenbo Zeng, May Hui, Bert L Semler, and Rozanne M Sandri-Goldin. 2020. <span>“Viral Vectors for Neural Circuit Mapping and Recent Advances in Trans-Synaptic Anterograde Tracers.”</span> <em>Neuron</em> 107: 1029–47. https://doi.org/<a href="https://doi.org/10.1016/j.neuron.2020.07.010">https://doi.org/10.1016/j.neuron.2020.07.010</a>.
</div>
<div id="ref-Yan2023" class="csl-entry">
Yan, Lulu, and Xiaoqiang Sun. 2023. <span>“Benchmarking and Integration of Methods for Deconvoluting Spatial Transcriptomic Data.”</span> <em>Bioinformatics</em> 39 (January). <a href="https://doi.org/10.1093/BIOINFORMATICS/BTAC805">https://doi.org/10.1093/BIOINFORMATICS/BTAC805</a>.
</div>
<div id="ref-Yang2021a" class="csl-entry">
Yang, Tao, Nicole Alessandri-Haber, Wen Fury, Michael Schaner, Robert Breese, Michael LaCroix-Fralish, Jinrang Kim, et al. 2021. <span>“AdRoit: An Accurate and Robust Method to Infer Complex Transcriptome Composition.”</span> <em>bioRxiv</em>, January, 2020.12.14.422697. <a href="https://doi.org/10.1101/2020.12.14.422697">https://doi.org/10.1101/2020.12.14.422697</a>.
</div>
<div id="ref-Yang1997" class="csl-entry">
Yang, Xiangdong W., Peter Model, and Nathaniel Heintz. 1997. <span>“Homologous Recombination Based Modification in Esherichia Coli and Germline Transmission in Transgenic Mice of a Bacterial Artificial Chromsome.”</span> <em>Nature Biotechnology</em> 15 (September): 859–65. <a href="https://doi.org/10.1038/nbt0997-859">https://doi.org/10.1038/nbt0997-859</a>.
</div>
<div id="ref-Yang2020" class="csl-entry">
Yang, Xiaoyu, Seth Bergenholtz, Lenka Maliskova, Mark-Phillip Pebworth, Arnold R Kriegstein, Yun Li, and Yin Shen. 2020. <span>“SMART-q: An Integrative Pipeline Quantifying Cell Type-Specific RNA Transcription.”</span> <em>PLOS ONE</em> 15 (April): e0228760. <a href="https://doi.org/10.1371/journal.pone.0228760">https://doi.org/10.1371/journal.pone.0228760</a>.
</div>
<div id="ref-Yang2019" class="csl-entry">
Yang, Yang, Qingwei Fang, and Hong-Bin Shen. 2019. <span>“Predicting Gene Regulatory Interactions Based on Spatial Gene Expression Data and Deep Learning.”</span> Edited by Smita Krishnaswamy. <em>PLOS Computational Biology</em> 15 (September): e1007324. <a href="https://doi.org/10.1371/journal.pcbi.1007324">https://doi.org/10.1371/journal.pcbi.1007324</a>.
</div>
<div id="ref-Yang2021" class="csl-entry">
Yang, Yi, Xingjie Shi, Wei Liu, Qiuzhong Zhou, Mai Chan Lau, Jeffrey Chun Tatt Lim, Lei Sun, Joe Yeong, and Jin Liu. 2021. <span>“SC-MEB: Spatial Clustering with Hidden Markov Random Field Using Empirical Bayes.”</span> <em>bioRxiv</em>, January, 2021.06.05.447181. <a href="https://doi.org/10.1101/2021.06.05.447181">https://doi.org/10.1101/2021.06.05.447181</a>.
</div>
<div id="ref-Yaylaoglu2005" class="csl-entry">
Yaylaoglu, Murat Burak, Andrew Titmus, Axel Visel, Gonzalo Alvarez-Bolado, Christina Thaller, and Gregor Eichele. 2005. <span>“Comprehensive Expression Atlas of Fibroblast Growth Factors and Their Receptors Generated by a Novel Robotic in Situ Hybridization Platform.”</span> <em>Developmental Dynamics</em> 234 (October): 371–86. <a href="https://doi.org/10.1002/dvdy.20441">https://doi.org/10.1002/dvdy.20441</a>.
</div>
<div id="ref-Yoda2017" class="csl-entry">
Yoda, Takuya, Masahito Hosokawa, Kiyofumi Takahashi, Chikako Sakanashi, Haruko Takeyama, and Hideki Kambara. 2017. <span>“Site-Specific Gene Expression Analysis Using an Automated Tissue Micro-Dissection Punching System.”</span> <em>Scientific Reports</em> 7 (December): 4325. <a href="https://doi.org/10.1038/s41598-017-04616-6">https://doi.org/10.1038/s41598-017-04616-6</a>.
</div>
<div id="ref-Yokoyama2009" class="csl-entry">
Yokoyama, Shigetoshi, Yoshiaki Ito, Hiroe Ueno-Kudoh, Hirohito Shimizu, Kenta Uchibe, Sonia Albini, Kazuhiko Mitsuoka, et al. 2009. <span>“A Systems Approach Reveals That the Myogenesis Genome Network Is Regulated by the Transcriptional Repressor RP58.”</span> <em>Developmental Cell</em> 17 (December): 836–48. <a href="https://doi.org/10.1016/j.devcel.2009.10.011">https://doi.org/10.1016/j.devcel.2009.10.011</a>.
</div>
<div id="ref-Yoshikawa2006" class="csl-entry">
Yoshikawa, Toshiyuki, Yulan Piao, Jinhui Zhong, Ryo Matoba, Mark G. Carter, Yuxia Wang, Ilya Goldberg, and Minoru S. H. Ko. 2006. <span>“High-Throughput Screen for Genes Predominantly Expressed in the ICM of Mouse Blastocysts by Whole Mount in Situ Hybridization.”</span> <em>Gene Expression Patterns</em> 6 (January): 213–24. <a href="https://doi.org/10.1016/j.modgep.2005.06.003">https://doi.org/10.1016/j.modgep.2005.06.003</a>.
</div>
<div id="ref-Yuan2012" class="csl-entry">
Yuan, Lei, Alexander Woodard, Shuiwang Ji, Yuan Jiang, Zhi-Hua Zhou, Sudhir Kumar, and Jieping Ye. 2012. <span>“Learning Sparse Representations for Fruit-Fly Gene Expression Pattern Image Annotation and Retrieval.”</span> <em>BMC Bioinformatics</em> 13 (May): 107. <a href="https://doi.org/10.1186/1471-2105-13-107">https://doi.org/10.1186/1471-2105-13-107</a>.
</div>
<div id="ref-Yuan2019" class="csl-entry">
Yuan, Ye, and Ziv Bar-Joseph. 2019. <span>“GCNG: Graph Convolutional Networks for Inferring Cell-Cell Interactions.”</span> <em>bioRxiv</em>, January, 2019.12.23.887133. <a href="https://doi.org/10.1101/2019.12.23.887133">https://doi.org/10.1101/2019.12.23.887133</a>.
</div>
<div id="ref-Zappia2018" class="csl-entry">
Zappia, Luke, Belinda Phipson, and Alicia Oshlack. 2018. <span>“Exploring the Single-Cell RNA-Seq Analysis Landscape with the scRNA-Tools Database.”</span> <em>PLOS Computational Biology</em> 14 (June): e1006245. <a href="https://doi.org/10.1371/journal.pcbi.1006245">https://doi.org/10.1371/journal.pcbi.1006245</a>.
</div>
<div id="ref-Zechel2014" class="csl-entry">
Zechel, Sabrina, Pawel Zajac, Peter Lönnerberg, Carlos F. Ibáñez, and Sten Linnarsson. 2014. <span>“Topographical Transcriptome Mapping of the Mouse Medial Ganglionic Eminence by Spatially Resolved RNA-Seq.”</span> <em>Genome Biology</em> 15 (October): 486. <a href="https://doi.org/10.1186/s13059-014-0486-z">https://doi.org/10.1186/s13059-014-0486-z</a>.
</div>
<div id="ref-Zeisel2018" class="csl-entry">
Zeisel, Amit, Hannah Hochgerner, Peter Lönnerberg, Anna Johnsson, Fatima Memic, Job van der Zwan, Martin Häring, et al. 2018. <span>“Molecular Architecture of the Mouse Nervous System.”</span> <em>Cell</em> 174 (August): 999–1014.e22. <a href="https://doi.org/10.1016/j.cell.2018.06.021">https://doi.org/10.1016/j.cell.2018.06.021</a>.
</div>
<div id="ref-Zeng2015" class="csl-entry">
Zeng, Tao, Rongjian Li, Ravi Mukkamala, Jieping Ye, and Shuiwang Ji. 2015. <span>“Deep Convolutional Neural Networks for Annotating Gene Expression Patterns in the Mouse Brain.”</span> <em>BMC Bioinformatics</em> 16 (December): 147. <a href="https://doi.org/10.1186/s12859-015-0553-9">https://doi.org/10.1186/s12859-015-0553-9</a>.
</div>
<div id="ref-Zhang2018" class="csl-entry">
Zhang, Ke, Wanwan Feng, and Peng Wang. 2018. <span>“Identification of Spatially Variable Genes with Graph Cuts.”</span> <em>bioRxiv</em>, January, 491472. <a href="https://doi.org/10.1101/491472">https://doi.org/10.1101/491472</a>.
</div>
<div id="ref-Zhang2020" class="csl-entry">
Zhang, Meng, Stephen W Eichhorn, Brian Zingg, Zizhen Yao, Hongkui Zeng, Hongwei Dong, and Xiaowei Zhuang. 2020. <span>“Molecular, Spatial and Projection Diversity of Neurons in Primary Motor Cortex Revealed by in Situ Single-Cell Transcriptomics.”</span> <em>bioRxiv</em>, January, 2020.06.04.105700. <a href="https://doi.org/10.1101/2020.06.04.105700">https://doi.org/10.1101/2020.06.04.105700</a>.
</div>
<div id="ref-Zhang2013" class="csl-entry">
Zhang, Wenlu, Daming Feng, Rongjian Li, Andrey Chernikov, Nikos Chrisochoides, Christopher Osgood, Charlotte Konikoff, Stuart Newfeld, Sudhir Kumar, and Shuiwang Ji. 2013. <span>“A Mesh Generation and Machine Learning Framework for Drosophila Gene Expression Pattern Image Analysis.”</span> <em>BMC Bioinformatics</em> 14 (December): 372. <a href="https://doi.org/10.1186/1471-2105-14-372">https://doi.org/10.1186/1471-2105-14-372</a>.
</div>
<div id="ref-Zhao2021" class="csl-entry">
Zhao, Edward, Matthew R Stone, Xing Ren, Jamie Guenthoer, Kimberly S Smythe, Thomas Pulliam, Stephen R Williams, et al. 2021. <span>“Spatial Transcriptomics at Subspot Resolution with BayesSpace.”</span> <em>Nat. Biotechnol.</em> 39 (November): 1375–84.
</div>
<div id="ref-Zhao2013" class="csl-entry">
Zhao, Feng, Licheng Jiao, and Hanqiang Liu. 2013. <span>“Kernel Generalized Fuzzy c-Means Clustering with Spatial Information for Image Segmentation.”</span> <em>Digit. Signal Process.</em> 23 (January): 184–99. <a href="https://doi.org/10.1016/j.dsp.2012.09.016">https://doi.org/10.1016/j.dsp.2012.09.016</a>.
</div>
<div id="ref-Zhao2012" class="csl-entry">
Zhao, Tuo, Han Liu, Kathryn Roeder, John Lafferty, and Larry Wasserman. 2012. <span>“The Huge Package for High-Dimensional Undirected Graph Estimation in r.”</span> <em>J. Mach. Learn. Res.</em> 13 (April): 1059–62.
</div>
<div id="ref-Zheng2017" class="csl-entry">
Zheng, Grace X. Y., Jessica M. Terry, Phillip Belgrader, Paul Ryvkin, Zachary W. Bent, Ryan Wilson, Solongo B. Ziraldo, et al. 2017. <span>“Massively Parallel Digital Transcriptional Profiling of Single Cells.”</span> <em>Nature Communications</em> 8 (April): 14049. <a href="https://doi.org/10.1038/ncomms14049">https://doi.org/10.1038/ncomms14049</a>.
</div>
<div id="ref-Zhou2020" class="csl-entry">
Zhou, Chuan, Ru Huang, Xiaoming Zhou, and Da Xing. 2020. <span>“Sensitive and Specific microRNA Detection by RNA Dependent DNA Ligation and Rolling Circle Optical Signal Amplification.”</span> <em>Talanta</em> 216: 120954. https://doi.org/<a href="https://doi.org/10.1016/j.talanta.2020.120954">https://doi.org/10.1016/j.talanta.2020.120954</a>.
</div>
<div id="ref-Zhou2007a" class="csl-entry">
Zhou, Jie, and Hanchuan Peng. 2007. <span>“Automatic Recognition and Annotation of Gene Expression Patterns of Fly Embryos.”</span> <em>Bioinformatics</em> 23 (March): 589–96. <a href="https://doi.org/10.1093/bioinformatics/btl680">https://doi.org/10.1093/bioinformatics/btl680</a>.
</div>
<div id="ref-Zhou2019" class="csl-entry">
Zhou, Wen, Mary A. Yui, Brian A. Williams, Jina Yun, Barbara J. Wold, Long Cai, and Ellen V. Rothenberg. 2019. <span>“Single-Cell Analysis Reveals Regulatory Gene Expression Dynamics Leading to Lineage Commitment in Early t Cell Development.”</span> <em>Cell Systems</em> 9 (October): 321–337.e9. <a href="https://doi.org/10.1016/j.cels.2019.09.008">https://doi.org/10.1016/j.cels.2019.09.008</a>.
</div>
<div id="ref-Zhu2023" class="csl-entry">
Zhu, Jiaqiang, Lulu Shang, and Xiang Zhou. 2023. <span>“SRTsim: Spatial Pattern Preserving Simulations for Spatially Resolved Transcriptomics.”</span> <em>Genome Biology</em> 24 (December): 1–30. <a href="https://doi.org/10.1186/S13059-023-02879-Z/FIGURES/7">https://doi.org/10.1186/S13059-023-02879-Z/FIGURES/7</a>.
</div>
<div id="ref-Zhu2020" class="csl-entry">
Zhu, Junjie, and Chiara Sabatti. 2020. <span>“Integrative Spatial Single-Cell Analysis with Graph-Based Feature Learning.”</span> <em>bioRxiv</em>, January, 2020.08.12.248971. <a href="https://doi.org/10.1101/2020.08.12.248971">https://doi.org/10.1101/2020.08.12.248971</a>.
</div>
<div id="ref-Zhu2018" class="csl-entry">
Zhu, Qian, Sheel Shah, Ruben Dries, Long Cai, and Guo Cheng Yuan. 2018. <span>“Identification of Spatially Associated Subpopulations by Combining scRNAseq and Sequential Fluorescence in Situ Hybridization Data.”</span> <em>Nature Biotechnology</em> 36 (December): 1183–90. <a href="https://doi.org/10.1038/nbt.4260">https://doi.org/10.1038/nbt.4260</a>.
</div>
<div id="ref-Zhu2001" class="csl-entry">
Zhu, Y Y, E M Machleder, A Chenchik, R Li, and P D Siebert. 2001. <span>“Reverse Transcriptase Template Switching: A SMART™ Approach for Full-Length cDNA Library Construction.”</span> <em>BioTechniques</em> 30 (April): 892–97. <a href="https://doi.org/10.2144/01304pf02">https://doi.org/10.2144/01304pf02</a>.
</div>
<div id="ref-Zuo2021" class="csl-entry">
Zuo, Qianying, Ayca Nazli Mogol, Yu-Jeh Liu, Ashlie Santaliz Casiano, Christine Chien, Jenny Drnevich, Ozan Berk Imir, et al. 2021. <span>“Targeting Metabolic Adaptations in the Breast Cancer–Liver Metastatic Niche Using Dietary Approaches to Improve Endocrine Therapy Efficacy.”</span> <em>bioRxiv</em>, January, 2021.09.07.458711. <a href="https://doi.org/10.1101/2021.09.07.458711">https://doi.org/10.1101/2021.09.07.458711</a>.
</div>
</div>
</div>






  <div class="chapter-nav">
<div class="prev"><a href="future.html"><span class="header-section-number">8</span> From the past to the present to the future</a></div>
<div class="empty"></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav"><li><a class="nav-link" href="#references">References</a></li></ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/pachterlab/LP_2021/blob/main/supplement/09-references.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/pachterlab/LP_2021/edit/main/supplement/09-references.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Museum of Spatial Transcriptomics</strong>" was written by Lambda Moses, Lior Pachter. It was last built on 2023-09-15.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
