# Data analysis in the prequel era {#prequel-analysis}

```{block2, note-text2, type='rmdtip', include=TRUE}
Many machine learning and statistics methods are mentioned in this chapter. The names of these methods are linked to articles explaining them for those who are unfamiliar. Some of them are math heavy.
```

```{r include=FALSE}
knitr::opts_chunk$set(echo = FALSE, fig.keep = "all",
                      message = FALSE, warning = FALSE,
                      fig.align = "center")
library(museumst)
library(tidyverse)
library(sf)
library(lubridate)
theme_set(theme_bw())
```

From the earliest days of enhancer and gene traps to the (WM)ISH atlases, identifying genes with spatially and temporally variable expression patterns, comparing and classifying the patterns, identifying new marker genes of cell types and developmental stages, and using gene expression to redefine cell types have been among the goals of the studies [@OKane1987; @Gossler1989; @Wurst1995a; @Sundaresan1995; @Gawantka1998; @Tomancak2002; @Lein2007]. In the prequel era, these were typically done manually, which, with the growing size of atlases in the 2000s, was time consuming and potentially inconsistent between curators. Thus, computational methods were developed to analyze images from the (WM)ISH atlases. This chapter reviews data analysis methods designed for (WM)ISH atlases and does not involve scRNA-seq data; methods involving both (WM)ISH and scRNA-seq are reviewed in Chapter \@ref(current-analysis) for the current era because scRNA-seq is at present a popular and rapidly growing field, too in vogue to be considered "prequel". If our collection is representative, then the rise of prequel data analysis methods arrived much later than that of data collection (Figure \@ref(fig:prequel-poly)).

```{r}
all_prequel <- read_metadata(sheet_use = c("Prequel", "Prequel analysis"), 
                             update = TRUE)
all_prequel <- all_prequel %>% 
  mutate(Type = case_when(sheet == "Prequel analysis" ~ "analysis",
                          TRUE ~ "data"))
```

```{r prequel-poly, fig.width=7, fig.height=4, fig.cap="Comparing trends in data collection and data analysis in the prequel era. Bin width is 365 days. The x-shaped points show the number of publications from the last bin, which is not yet full."}
era_freqpoly(all_prequel, Type, preprints = TRUE, binwidth = 365) +
    scale_x_date(date_breaks = "2 years", date_labels = "%Y") +
    scale_y_continuous(expand = expansion(c(0,0.05)))
```

Except for one study on *Platynereis dumereilii* in 2014 [@Pettit2014], on *Xenopus tropicalis* in 2018 [@Patrushev2018], one on post mortem human brain in 2021 [@Abed-Esfahani2021], all data analysis methods in our collection were designed for either *Drosophila melanogaster* or *Mus musculus* (Figure \@ref(fig:pa-species)). There seem to have been two waves; the first for *Drosophila*, peaking in the late 2000s, mostly concerning the BDGP in situ atlas, and the second for mice, peaking in early 2010s, mostly concerning ABA (Figure \@ref(fig:pa-species)). The apparent rise since 2019 is in part driven by deep learning methods to annotate gene expression patterns or infer gene interactions. Given the small number of publications in this category and potential incompleteness of the curation, the trends should be taken with a grain of salt.

```{r}
prequel_analysis <- read_metadata("Prequel analysis", update = TRUE)
```

```{r}
p4b_annot <- tibble(facets = c("Drosophila melanogaster", "Mus musculus"),
                    xint = ymd(c("2002-12-23", "2006-12-06")),
                    x =  xint,
                    label = c("BDGP", "ABA"))
```

```{r pa-species, fig.width=6, fig.height=6, fig.cap="Gray histogram in the background is overall histogram of prequel data analysis literature. Number of publications in each time bin for each species is highlighted in the facets."}
pubs_per_year(prequel_analysis, facet_by = "species", fill_by = "species",
                      n_top = 2, preprints = TRUE) +
    geom_vline(data = p4b_annot, aes(xintercept = xint), color = "gray70") +
    geom_text(data = p4b_annot, aes(label = label, x = x, y = 4)) +
    theme(legend.position = "top")
```

## Gene patterns {#prequel-gene-patterns}
The most common goal of these data analysis methods was to annotate and compare gene expression patterns, especially to automate annotation of the BDGP atlas (Figure \@ref(fig:pa-category)). It seems reasonable to focus on 4 phases in this category: first, in early to mid 2000s, after image registration, the images were binarized into "expressed" and "not expressed" regions, and the shapes of the expressed regions were summarized and compared. Metrics to summarize the shapes included [moment invariant](https://towardsdatascience.com/introduction-to-the-invariant-moment-and-its-application-to-the-feature-extraction-ee991f39ec) [@Jayaraman2001; @Gurunathan2004], Hamming distance [@Kumar2002], and a weighted score involving [L1 distance](https://iq.opengenus.org/manhattan-distance/) between column or row histograms of two images [@Liu2007]. These unsupervised methods enabled clustering of patterns and querying genes with similar patterns to a given gene.

```{r}
categories <- unnest_cat(prequel_analysis, category, other_cols = "species")
```

```{r pa-category, fig.width=6, fig.height=6, fig.cap="Number of publications in each time bin for each category of data analysis is highlighted in the facets."}
pubs_per_year(categories, facet_by = "category", n_top = 5, 
              fill_by = "species", sort_by = "count", preprints = TRUE) +
    theme(legend.position = "top")
```

Second, from the mid 2000s to mid 2010s, many supervised and unsupervised methods for gene expression pattern annotation or comparison were developed. In supervised methods, extensive feature engineering more sophisticated than binarization was performed on registered images for image annotation with machine learning classification. These methods were trained with existing BDGP annotations and developed to automatically annotate the BDGP expression patterns with controlled vocabulary (CV) of anatomical regions where genes were expressed. In BDGP, a gene gets annotated with a CV if the gene was deemed expressed in the anatomical region and developmental stage denoted by the CV, so the annotation typically contained a list of CVs.

The feature engineering can be based on the wavelet transform [@Zhou2007a] and Fourier coefficients [@Heffel2008a], but a particularly popular feature engineering method was [scale-invariant feature transform (SIFT)](https://docs.opencv.org/master/da/df5/tutorial_py_sift_intro.html) [@Lowe2004; @Ji2008; @Li2009; @Ji2009]. A method published in 2009 that used SIFT followed by [bag of words](https://machinelearningmastery.com/gentle-introduction-bag-words-model/) where "word" is a [k means](https://medium.com/analytics-vidhya/k-means-clustering-explained-419ee66d095e) cluster (code book) was quite influential [@Ji2009]; several later methods were inspired by this method, with improved code books [@Sun2013; @Ji2009a; @Yuan2012; @Liscovitch2013a]. The most common classifier that take in the features to predict annotations is [support vector machine (SVM)](https://towardsdatascience.com/support-vector-machine-introduction-to-machine-learning-algorithms-934a444fca47) [@Sun2013; @Yuan2012] or multi-label variants of it [@Ji2008; @Ji2009].

Unsupervised methods rely on clustering algorithms after images are registered on a common mesh, such as [affinity propagation clustering](https://towardsdatascience.com/unsupervised-machine-learning-affinity-propagation-algorithm-explained-d1fef85f22c8) [@Frise2010] and co-clustering (rows and columns of matrix are clustered simultaneously) [@Jagalur2007; @Zhang2013].

Third, another notable type of the feature engineering is dimension reduction. In 2006, some methods applied dimension reduction methods such as [principal component analysis (PCA)](https://liorpachter.wordpress.com/2014/05/26/what-is-principal-component-analysis/) and [independent component analysis (ICA)](http://wwwf.imperial.ac.uk/~nsjones/TalkSlides/HyvarinenSlides.pdf) to the registered images to find "eigen" patterns [@Pan2006; @Peng2006]. Instead of PCA or ICA, the dimension reduction can also be sparse Bayesian factor analysis [@Pruteanu-Malinici2011], sparse dictionary learning [@Li2017], and [non-negative matrix factorization (NMF)](http://www.cs.cmu.edu/~11755/lectures/Lee_Seung_NMF.pdf) [@Noto2017; @Wu2016]. The dimension reduction can be used for unsupervised clustering of genes [@Pan2006; @Peng2006; @Pruteanu-Malinici2011], as well as supervised classification methods such as SVM and [logistic regression](https://towardsdatascience.com/logistic-regression-detailed-overview-46c4da4303bc) to annotate gene expression patterns with controlled vocabulary [@Pruteanu-Malinici2011; @Wu2016]. Notably, in NMF, both the matrix for basis patterns and the coefficient matrix for the genes tend to exhibit block structures; the blocks in the gene coefficient matrix has been used to cluster genes [@Noto2017].

Fourth, since 2015, [convolutional neural networks (CNNs)](https://towardsdatascience.com/a-comprehensive-guide-to-convolutional-neural-networks-the-eli5-way-3bd2b1164a53) have been adopted to analyze gene expression patterns. Typically, a pre-trained model, such as ResNet50, OverFeat, or Alexnet is used. With some modifications or retraining of the original model, the model can be used to extract features for gene pattern annotation with logistic regression [@Zeng2015], classifying new patterns [@Andonian2019, @Long2021], and predicting interactions between genes [@Yang2019].

## Spatial regions {#prequel-spatial-regions}
Closely related to classifying gene expression patterns are these questions: What are the implications of gene expression patterns to traditional anatomical regions as in the CV? Can we discover novel anatomical regions from gene expression? How well do expression-based regions correspond to the traditional regions? A few studies, which we call "spatial region", tried to answer these questions in the ABA (Figure \@ref(fig:pa-category)). Clusters of expression patterns of cell type specific genes [@Ko2013], or the most localized genes [@Grange2014], principal components of the patterns [@Bohland2010a], or patterns of coexpression modules were compared to traditional anatomy [@Grange2014]. At least in the mouse brain, with the principal components, these clusters may correspond to traditional anatomy quite well [@Bohland2010a]. However, when cell types are taken into account in clustering, gene expression seems to be able to refine traditional anatomy [@Ko2013; @Grange2014].

A clustering strategy for identifying spatial regions that takes the spatial neighborhood into account is [Markov random field (MRF)](https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-867-machine-learning-fall-2006/lecture-notes/lec23.pdf). In MRFs, nearby voxels can be made to be more likely to share a label, which can be cell type or histological region, and the probability of a voxel taking each of the labels only depends on labels of neighboring voxels. MRFs were used to delineate spatial regions in a 3D FISH atlas of the developing [*Platynereis dumereilii*](https://platynereis.github.io/) brain [@Pettit2014], with 86 high quality genes. The images in the atlas were aligned into a 3D model and broken into voxels 3 $\mu$m per side, which is smaller than a typical single cell; the spatial neighborhood graph is the 3D square grid of the voxels. As FISH is not very quantitative, the gene expression was manually binarized. Expression of each gene at each voxel is modeled with a [Bernoulli distribution](https://mathworld.wolfram.com/BernoulliDistribution.html), and the 86 genes are assumed to be independent. Cluster label assignment is modeled with [Potts model](https://en.wikipedia.org/wiki/Potts_model), a type of MRF in which only neighboring voxels with the same label contribute to the probability distribution of the labels, thus favoring neighbors with the same label. The parameters, such as interaction strength between neighboring voxels for the Potts model and the probability parameter of the Bernoulli distributions are estimated with [expectation maximization (EM)](http://ai.stanford.edu/~chuongdo/papers/em_tutorial.pdf).

## Gene interactions {#prequel-gene-interactions}
While not single cell resolution, (WM)ISH atlases provide transcriptomes within the tissue at a resolution far higher than that of typical bulk RNA-seq and bulk microarray, thus opening the way to studying coexperssion and interaction between genes within the tissue. There are a few methods that aim to decide whether two genes interact according to (WM)ISH images, some dating published long before the popularization of scRNA-seq. Already in 2002, an early method that compares binarized gene expression patterns was used to identify interactions among genes by comparing patterns from wild type and mutant backgrounds [@Kumar2002]. 

However, as mutant lines are harder to obtain than wild type images, the simplest method is to set a threshold in [Pearson correlation](https://www.questionpro.com/blog/pearson-correlation-coefficient/) coefficient between two genes to decide an edge should be drawn on the gene coexpression graph [@Wu2016; @Campiteli2013]. 

Alternatively, a sparse [Markov network](http://ml.informatik.uni-freiburg.de/former/_media/teaching/ws1314/gm/11-markov_logic_networks.handout.pdf) whose nodes are genes and edges are presence of interaction can be learnt from expression profiles in each voxel [@Puniyani2013], or a CNN can be trained on known interactions and predict new interactions based on gene expression patterns [@Yang2019]. There are other types of analyses, such as inferring gene function from expression pattern, identifying spatially variable genes, and gene expression imputation at locations. The latter two are still important topics in current era data analysis.

## Decline {#decline}
What contributed to the decline of the golden age of prequel data analysis? Partly a lack of usage of the methods developed, which was exacerbated by the decline of the golden age of (WM)ISH atlases in the 2010s so there were fewer new atlases where the methods can be applied (Figure \@ref(fig:hist1)). While many methods to automate gene expression pattern annotation for BDGP were developed before 2013, for the 2013 BDGP update that added images of 708 transcription factors, the BDGP annotated the new images with human curators instead of the automated methods [@Hammonds2013]. Nor did BDGP use the new methods to compare and classify the new gene expression patterns; instead, the curator assigned CV annotations were used for analysis [@Hammonds2013; @Tomancak2007]. BDGP did not have a major update after 2013; as existing images have already been annotated, there is no need to automate annotations.

There are additional possible reasons why these methods were not used: First, it is unclear from the publications of the methods where the software implementation can be obtained. Second, a reason why most prequel analysis methods were developed for either BDGP or ABA is that since one gene is stained for in one embryo/section at a time, the images must be registered and standardized for different genes to be comparable; BDGP, through FlyExpress [@Kumar2017], and ABA, provide images that have already been registered and standardized, while many other atlases, such as GEISHA, do not. Due to challenges in image registration in other organisms, the automated gene expression pattern analysis methods can't be applied. Third, lack of usage of these methods can also be due to insufficient accuracy; from 2009 to 2013, the [area under the curve (AUC)](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5) of the automated annotations is typically around 0.8 and rarely exceeded 0.9 [@Ji2009; @Pruteanu-Malinici2011; @Yuan2012; @Sun2013], which means when using such tools to annotate new images, extensive human review would still be required.

## Geography of prequel data analysis {#geo-prequel-analysis}

```{r}
city_gc <- geocode_inst_city(prequel_analysis)
pubs_on_map2 <- partial(pubs_on_map, city_gc = city_gc)
```

If our collection is representative, then contribution to prequel data analysis concentrates in a few institutions (Figure \@ref(fig:pa-map)), not all of which are elite.

```{r pa-map, fig.width=10, fig.height=5, fig.cap="Number of publications per city for prequel data analysis.", out.width="100%"}
pubs_on_map2(prequel_analysis)
```

```{r pa-map-us, fig.width=9, fig.height=6, fig.cap="Number of publications per city for prequel data analysis in the US.", out.width="100%"}
pubs_on_map2(prequel_analysis, zoom = "usa")
```

When broken down by species, it seems that distinct institutions contributed to data analysis of *Drosophila* and mouse data. UC Berkeley and Lawrence Berkeley National Laboratory (LBL) are responsible for BDGP, and Allen is responsible for ABA. However, among the top contributors are other institutions such as Arizona State University (ASU) and Old Dominion University (ODU) (Figure \@ref(fig:pa-species2)).

```{r pa-species2, fig.width=10, fig.height=15, fig.cap="Number of publications per city for prequel data analysis broken down by species of interest.", out.width="100%"}
pubs_on_map2(prequel_analysis, facet_by = "species", n_top = 2, ncol = 1, 
             n_label = 5)
```
